<html>
    <head>
        <meta charset="utf-8">
        
            <script src="lib/bindings/utils.js"></script>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/dist/vis-network.min.css" integrity="sha512-WgxfT5LWjfszlPHXRmBWHkV2eceiWTOBvrKCNbdgDYTHrT2AeLCGbF4sZlZw3UMN3WtL0tGUoIAKsu8mllg/XA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
            <script src="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/vis-network.min.js" integrity="sha512-LnvoEWDFrqGHlHmDD2101OrLcbsfkrzoSpvtSQtxK3RMnRV0eOkhhBN2dXHKRrUU8p2DGRTk35n4O8nWSVe1mQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
            
        
<center>
<h1></h1>
</center>

<!-- <link rel="stylesheet" href="../node_modules/vis/dist/vis.min.css" type="text/css" />
<script type="text/javascript" src="../node_modules/vis/dist/vis.js"> </script>-->
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
          crossorigin="anonymous"
        />
        <script
          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js"
          integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf"
          crossorigin="anonymous"
        ></script>


        <center>
          <h1></h1>
        </center>
        <style type="text/css">

             #mynetwork {
                 width: 100%;
                 height: 600px;
                 background-color: white;
                 border: 1px solid lightgray;
                 position: relative;
                 float: left;
             }

             
             #loadingBar {
                 position:absolute;
                 top:0px;
                 left:0px;
                 width: 100%;
                 height: 600px;
                 background-color:rgba(200,200,200,0.8);
                 -webkit-transition: all 0.5s ease;
                 -moz-transition: all 0.5s ease;
                 -ms-transition: all 0.5s ease;
                 -o-transition: all 0.5s ease;
                 transition: all 0.5s ease;
                 opacity:1;
             }

             #bar {
                 position:absolute;
                 top:0px;
                 left:0px;
                 width:20px;
                 height:20px;
                 margin:auto auto auto auto;
                 border-radius:11px;
                 border:2px solid rgba(30,30,30,0.05);
                 background: rgb(0, 173, 246); /* Old browsers */
                 box-shadow: 2px 0px 4px rgba(0,0,0,0.4);
             }

             #border {
                 position:absolute;
                 top:10px;
                 left:10px;
                 width:500px;
                 height:23px;
                 margin:auto auto auto auto;
                 box-shadow: 0px 0px 4px rgba(0,0,0,0.2);
                 border-radius:10px;
             }

             #text {
                 position:absolute;
                 top:8px;
                 left:530px;
                 width:30px;
                 height:50px;
                 margin:auto auto auto auto;
                 font-size:22px;
                 color: #000000;
             }

             div.outerBorder {
                 position:relative;
                 top:400px;
                 width:600px;
                 height:44px;
                 margin:auto auto auto auto;
                 border:8px solid rgba(0,0,0,0.1);
                 background: rgb(252,252,252); /* Old browsers */
                 background: -moz-linear-gradient(top,  rgba(252,252,252,1) 0%, rgba(237,237,237,1) 100%); /* FF3.6+ */
                 background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(252,252,252,1)), color-stop(100%,rgba(237,237,237,1))); /* Chrome,Safari4+ */
                 background: -webkit-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* Chrome10+,Safari5.1+ */
                 background: -o-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* Opera 11.10+ */
                 background: -ms-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* IE10+ */
                 background: linear-gradient(to bottom,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* W3C */
                 filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#fcfcfc', endColorstr='#ededed',GradientType=0 ); /* IE6-9 */
                 border-radius:72px;
                 box-shadow: 0px 0px 10px rgba(0,0,0,0.2);
             }
             

             

             
        </style>
    </head>


    <body>
        <div class="card" style="width: 100%">
            
            
            <div id="mynetwork" class="card-body"></div>
        </div>

        
            <div id="loadingBar">
              <div class="outerBorder">
                <div id="text">0%</div>
                <div id="border">
                  <div id="bar"></div>
                </div>
              </div>
            </div>
        
        

        <script type="text/javascript">

              // initialize global variables.
              var edges;
              var nodes;
              var allNodes;
              var allEdges;
              var nodeColors;
              var originalNodes;
              var network;
              var container;
              var options, data;
              var filter = {
                  item : '',
                  property : '',
                  value : []
              };

              

              

              // This method is responsible for drawing the graph, returns the drawn network
              function drawGraph() {
                  var container = document.getElementById('mynetwork');

                  

                  // parsing and collecting nodes and edges from the python
                  nodes = new vis.DataSet([{"betweenness_centrality": "336.1007831057829", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.24603174603174602", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n0", "label": "Bifidobacterium", "labels": ":Genus", "name": "Bifidobacterium", "pagerank_centrality": "0.8798910261840658", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 18, "synonym": "[\u0027\"Bifidibacterium\" RELATED synonym []\u0027, \u0027\"Tissieria\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11594590\u0027, \u0027PMID:17158978\u0027, \u0027PMID:1742200\u0027, \u0027PMID:20061504\u0027, \u0027PMID:8573484\u0027]"}, {"betweenness_centrality": "240.83263014763014", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20840336134453782", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n268", "label": "Bifidobacteriaceae", "labels": ":Family", "name": "Bifidobacteriaceae", "pagerank_centrality": "0.7421356537815125", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 16, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19244447\u0027]"}, {"betweenness_centrality": "327.73706293706294", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.24652087475149106", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n1", "label": "Segatella", "labels": ":Genus", "name": "Segatella", "pagerank_centrality": "0.7004457427452533", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 18, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "232.04658674658683", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20701168614357263", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n269", "label": "Prevotellaceae", "labels": ":Family", "name": "Prevotellaceae", "pagerank_centrality": "0.8048991524851027", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 15, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "429.19056024926147", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.29107981220657275", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "14.0", "font": {"size": 12}, "id": "n2", "label": "Streptococcus", "labels": ":Genus", "name": "Streptococcus", "pagerank_centrality": "1.413261928471924", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 20, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10555340\u0027, \u0027PMID:14657115\u0027, \u0027PMID:1720654\u0027, \u0027PMID:19620365\u0027, \u0027PMID:19880633\u0027, \u0027PMID:7537076\u0027, \u0027PMID:8995803\u0027]"}, {"betweenness_centrality": "846.4779036407153", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.32978723404255317", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "18.0", "description": "A trophic process during which a living entity acquires food and energy by consuming another organism or organism part, living or dead.", "font": {"size": 12}, "id": "n81", "iri": "http://purl.obolibrary.org/obo/ECOCORE_00000011", "label": "Heterotrophy", "labels": ":BiologicalProcess", "name": "Heterotrophy", "pagerank_centrality": "1.7438053272001854", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 31, "synonym": "", "xref": ""}, {"betweenness_centrality": "466.96011170404756", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.3147208121827411", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "18.0", "description": "The chemical reactions and pathways involving carbohydrates, any of a group of organic compounds based of the general formula Cx(H2O)y. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n291", "iri": "http://purl.obolibrary.org/obo/GO_0005975 ", "label": "Carbohydrate metabolic process", "labels": ":BiologicalProcess", "name": "Carbohydrate metabolic process", "pagerank_centrality": "1.7413181515657719", "providedBy": "MiDAS", "shape": "dot", "size": 21, "synonym": "carbohydrate metabolism", "xref": ""}, {"betweenness_centrality": "241.57325920163362", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.30170316301703165", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "14.0", "description": "The chemical reactions and pathways involving a short-chain fatty acid. A short-chain fatty acid has an aliphatic tail containing fewer than 6 carbons. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n290", "iri": "http://purl.obolibrary.org/obo/GO_0046459 ", "label": "Short-chain fatty acid metabolic process", "labels": ":BiologicalProcess", "name": "Short-chain fatty acid metabolic process", "pagerank_centrality": "1.3725780668159704", "providedBy": "MiDAS", "shape": "dot", "size": 16, "synonym": "short-chain fatty acid metabolism", "xref": ""}, {"betweenness_centrality": "220.31736859333623", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.23529411764705882", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n270", "label": "Streptococcaceae", "labels": ":Family", "name": "Streptococcaceae", "pagerank_centrality": "0.6289535273032705", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 15, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "527.0752696363102", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.31794871794871793", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "18.0", "description": "The anaerobic enzymatic conversion of organic compounds, especially carbohydrates, coupling the oxidation and reduction of NAD/H and the generation of adenosine triphosphate (ATP). During fermentation, organic compounds act as both the electron acceptor and donor. Organic molecules, such as sugars and amino acids, are utilized as the energy and carbon source. In the absence of, or the inability to utilize, external electron acceptors, organic molecules act as the electron acceptor to balance the redox of the cell. Common by-products include volatile fatty acids, alcohols, hydrogen and CO2.", "font": {"size": 12}, "id": "n82", "iri": "http://purl.obolibrary.org/obo/GO_0006113", "label": "Fermentation", "labels": ":BiologicalProcess", "name": "Fermentation", "pagerank_centrality": "1.7130264686220826", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 23, "synonym": "", "xref": "MetaCyc:FERMENTATION-PWY|Wikipedia:Fermentation_(biochemistry)"}, {"betweenness_centrality": "102.75842559831163", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.289044289044289", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "12.0", "description": "The chemical reactions and pathways involving amino acids, carboxylic acids containing one or more amino groups. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n292", "iri": "http://purl.obolibrary.org/obo/GO_0006520 ", "label": "Amino acid metabolic process", "labels": ":BiologicalProcess", "name": "Amino acid metabolic process", "pagerank_centrality": "1.1777811130064837", "providedBy": "MiDAS", "shape": "dot", "size": 12, "synonym": "cellular amino acid metabolic process|cellular amino acid metabolism|amino acid and derivative metabolism|cellular amino acid and derivative metabolic process", "xref": ""}, {"betweenness_centrality": "410.84830343576317", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2980769230769231", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n3", "label": "Blautia", "labels": ":Genus", "name": "Blautia", "pagerank_centrality": "1.2102024936465345", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 20, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:18676476\u0027]"}, {"betweenness_centrality": "270.33028741868605", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2450592885375494", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n271", "label": "Lachnospiraceae", "labels": ":Family", "name": "Lachnospiraceae", "pagerank_centrality": "0.9929847526647204", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 16, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "226.34707390706848", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2850574712643678", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "14.0", "font": {"size": 12}, "id": "n4", "label": "Dorea", "labels": ":Genus", "name": "Dorea", "pagerank_centrality": "1.364561753818243", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 15, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11931151\u0027]"}, {"betweenness_centrality": "21.84126984126984", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.21946902654867256", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n5", "label": "Mediterraneibacter", "labels": ":Genus", "name": "Mediterraneibacter", "pagerank_centrality": "0.5631520221410032", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 10, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "324.1020046507959", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.29314420803782504", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "10.0", "font": {"size": 12}, "id": "n6", "label": "Faecalibacterium", "labels": ":Genus", "name": "Faecalibacterium", "pagerank_centrality": "1.0948397454181475", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 18, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12508881\u0027]"}, {"betweenness_centrality": "155.75790865677746", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.23709369024856597", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n272", "label": "Oscillospiraceae", "labels": ":Family", "name": "Oscillospiraceae", "pagerank_centrality": "0.6297889284685073", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"Hungateiclostridiaceae\" RELATED synonym []\u0027, \u0027\"Ruminococcaceae\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13130057\u0027, \u0027PMID:30124399\u0027, \u0027PMID:31526446\u0027]"}, {"betweenness_centrality": "380.73363738498216", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2850574712643678", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "10.0", "font": {"size": 12}, "id": "n7", "label": "Eubacterium", "labels": ":Genus", "name": "Eubacterium", "pagerank_centrality": "1.0928727887073846", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 19, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:8863439\u0027]"}, {"betweenness_centrality": "202.25237619187962", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.23134328358208955", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n273", "label": "Erysipelotrichaceae", "labels": ":Family", "name": "Erysipelotrichaceae", "pagerank_centrality": "0.647086477669329", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 15, "synonym": "[\u0027\"Erysipelothrix group\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:14742484\u0027, \u0027PMID:27270136\u0027]"}, {"betweenness_centrality": "365.1955026827598", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2811791383219955", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "18.0", "font": {"size": 12}, "id": "n8", "label": "Paracoccus", "labels": ":Genus", "name": "Paracoccus", "pagerank_centrality": "1.8522633584603514", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 19, "synonym": "[\u0027\"Paracoccus\" EXACT scientific_name []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "12.554544039838154", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.23177570093457944", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "4.0", "description": "The process in which a multicellular organism, a unicellular organism or a group of unicellular organisms grow in a threadlike, filamentous shape. Excessive growth of organisms with a filamentous morphology has long been associated with the operational problem of bulking. In this condition, interfloc bridging by the filaments impedes adequate settling of the biomass, resulting in loss of solids with the treated effluent", "font": {"size": 12}, "id": "n293", "iri": "http://purl.obolibrary.org/obo/GO_0030447 ", "label": "Filamentous growth", "labels": ":BiologicalProcess", "name": "Filamentous growth", "pagerank_centrality": "0.5324940758228365", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "128.54666154033788", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.23177570093457944", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n274", "label": "Paracoccaceae", "labels": ":Family", "name": "Paracoccaceae", "pagerank_centrality": "0.6145590284474491", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"alpha-3 proteobacteria\" RELATED synonym []\u0027, \u0027\"Rhodobacter clade\" RELATED synonym []\u0027, \u0027\"Rhodobacteraceae\" RELATED synonym []\u0027, \u0027\"Rhodobacteraceae Garrity et al. 2006\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16403855\u0027, \u0027PMID:28106881\u0027]"}, {"betweenness_centrality": "6.222981490473751", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.2540983606557377", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "4.0", "description": "The chemical reactions and pathways resulting in the formation of a polyphosphate, the anion or salt of polyphosphoric acid. The PAOs are enriched for in wastewater treatment systems configured for enhanced biological phosphorus removal (EBPR). They are responsible for the bulk of biological phosphorus removal from wastewater. The principle of EBPR systems relies on the cycling of biomass between carbon rich anaerobic conditions and carbon deficient aerobic conditions. The PAOs accumulate excess levels of phosphorus as polyphosphate under aerobic conditions, which they utilize as an energy source for substrate assimilation under anaerobic conditions. Thus, wastage of polyphosphate rich biomass from the aerobic tank leads to net P removal from the system. Earlier definitions of the PAO included cycling of glycogen and polyhydroxyalkanoates (PHAs) in accordance with the model PAO, Candidatus Accumulibacter. However, the Tetrasphaera do not accumulate PHA or cycle glycogen in the same way. As such, in MiDAS, the cycling of excess polyphosphate under the dynamic conditions of EBPR is applied to define a PAO.", "font": {"size": 12}, "id": "n288", "iri": "http://purl.obolibrary.org/obo/GO_0006799 ", "label": "Polyphosphate biosynthetic process", "labels": ":BiologicalProcess", "name": "Polyphosphate biosynthetic process", "pagerank_centrality": "0.4947796926460722", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "polyphosphate anabolism|polyphosphate biosynthesis|polyphosphate formation|polyphosphate synthesis", "xref": ""}, {"betweenness_centrality": "213.55828506289038", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.23529411764705882", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n9", "label": "Neisseria", "labels": ":Genus", "name": "Neisseria", "pagerank_centrality": "0.6710876386501152", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 15, "synonym": "[\u0027\"\\\"Merismopedia\\\" Zopf 1885\" RELATED synonym []\u0027, \u0027\"Gonococcus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:7520730\u0027]"}, {"betweenness_centrality": "133.80828506289035", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20361247947454844", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n275", "label": "Neisseriaceae", "labels": ":Family", "name": "Neisseriaceae", "pagerank_centrality": "0.759045551099168", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"Simonsiellaceae\" RELATED synonym []\u0027, \u0027\"Vitreoscillaceae\" RELATED in_part []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13983561\u0027, \u0027PMID:16350128\u0027, \u0027PMID:23575986\u0027, \u0027PMID:37882538\u0027, \u0027PMID:8347509\u0027]"}, {"betweenness_centrality": "1038.6530648905996", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2767857142857143", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "20.0", "font": {"size": 12}, "id": "n10", "label": "Nitrosomonas", "labels": ":Genus", "name": "Nitrosomonas", "pagerank_centrality": "2.4792479654135016", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 36, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.2171628721541156", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "A trophic process during which a living entity generates food from abiotic sources or through consumption of living or dead material, and captures energy from preformed molecules. Autotrophs are able to fix CO2 as a carbon source for growth. Important examples in wastewater treatment include nitrite and ammonia oxidizers (NOB and AOB), anaerobic ammonia oxidizers (anammox). Mixotrophs can obtain carbon by fixing CO2 and or from organic sources (heterotrophy). The criterium for this MiDAS field is the potential for CO2 fixation i.e. organisms possessing the Calvin-cycle.", "font": {"size": 12}, "id": "n69", "iri": "http://purl.obolibrary.org/obo/ECOCORE_00000163", "label": "Chemotrophy", "labels": ":BiologicalProcess", "name": "Chemotrophy", "pagerank_centrality": "0.3591291234831877", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "55.83398998662155", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.2572614107883817", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "4.0", "description": "The reduction of nitrate to dinitrogen by four reactions; each intermediate is transformed to the next lower oxidation state; also part of cellular bioenergetics; the nitrogen compounds can serve as terminal acceptors for electron transport phosphorylation in place of oxygen. The denitrifiers are usually facultative-anaerobic heterotrophic organisms utilizing nitrate and/or nitrite as electron acceptors in the absence of oxygen. Denitrification is the final step of nitrogen removal from wastewater and generally occurs under anaerobic conditions. Nitrate is available from the oxidation of ammonia by the aerobic activities of the nitrifiers (AOB and NOB). Denitrification involves the sequential reduction of nitrate to inert dinitrogen gas, via several intermediates (nitrite, nitric oxide (gas) and nitrous oxide (gas)). Not all organisms involved in denitrification possess all the genes for reduction of nitrate all the way to dinitrogen gas. As such, for MiDAS profiles, the nitrite reducing organisms are selected to represent the denitrifiers; given denitrification is defined as \u201cthe dissimilatory transformation of nitrate or nitrite to a gas species concomitant with energy conservation\u201d and not all organisms capable of nitrate reduction can reduce nitrite.", "font": {"size": 12}, "id": "n65", "iri": "http://purl.obolibrary.org/obo/GO_0019333", "label": "Denitrification pathway", "labels": ":BiologicalProcess", "name": "Denitrification pathway", "pagerank_centrality": "0.5415841339380558", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 11, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.2171628721541156", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The nitrite oxidizing bacteria are chemolithotrophic organisms, which utilize nitrite as an energy source and. Nitrite oxidation to nitrate is coupled to the reduction of O2. Nitrite is obtained from the activities of the ammonia oxidizing bacteria.", "font": {"size": 12}, "id": "n286", "iri": "http://purl.obolibrary.org/obo/OMIT_0027217", "label": "Nitrification", "labels": ":BiologicalProcess", "name": "Nitrification", "pagerank_centrality": "0.3591291234831877", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "202.93834027397065", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2283609576427256", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n276", "label": "Nitrosomonadaceae", "labels": ":Family", "name": "Nitrosomonadaceae", "pagerank_centrality": "0.6682765939847275", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 15, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13130037\u0027, \u0027PMID:16403855\u0027]"}, {"betweenness_centrality": "77.02294600136707", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1646746347941567", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n11", "label": "Thiodictyon", "labels": ":Genus", "name": "Thiodictyon", "pagerank_centrality": "0.7610431108179928", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "41.74666666666667", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1546134663341646", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n277", "label": "Chromatiaceae", "labels": ":Family", "name": "Chromatiaceae", "pagerank_centrality": "0.8018411283695475", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 11, "synonym": "[\u0027\"purple sulfur bacteria\" EXACT OMO:0003003 []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:28612170\u0027, \u0027PMID:9828415\u0027]"}, {"betweenness_centrality": "1231.261527594532", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.3315508021390374", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "28.0", "font": {"size": 12}, "id": "n12", "label": "Acinetobacter", "labels": ":Genus", "name": "Acinetobacter", "pagerank_centrality": "2.8487110687171393", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 40, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10028249\u0027, \u0027PMID:7520730\u0027, \u0027PMID:8934907\u0027, \u0027PMID:9226915\u0027, \u0027PMID:9336926\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.24949698189134809", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The chemical reactions and pathways resulting in the formation of glycogen, a polydisperse, highly branched glucan composed of chains of D-glucose residues. The GAO are well adapted to the dynamic conditions of EBPR where they compete with the PAO for resources. Under anaerobic conditions they utilize aerobically stored glycogen for the energy and reducing equivalents required for carbon uptake and storage as PHAs. Anaerobically stored carbon is utilized under subsequent aerobic conditions for growth and replenishment of glycogen stores. As they do not cycle polyphosphate, but compete with the PAO for nutrient resources, their proliferation may be at the expense of P removal efficiency in EBPR. However, recent research indicates that GAO not are a real problem in full-scale plants only in lab-scale reactors.", "font": {"size": 12}, "id": "n289", "iri": "http://purl.obolibrary.org/obo/GO_0005978 ", "label": "Glycogen biosynthetic process", "labels": ":BiologicalProcess", "name": "Glycogen biosynthetic process", "pagerank_centrality": "0.32141474030642314", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "glycogen anabolism|glycogen biosynthesis|glycogen formation|glycogen synthesis", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.24949698189134809", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "A substance within a foam formulation that is used to maintain or improve its stability. Organisms with a hydrophobic surface have been associated with the stabilization of foams in both activated sludge and anaerobic digester systems. Excessive build-up of surface foams can result in carryover of solids from clarifiers in wastewater treatment systems and several operational problems in anaerobic digesters including biofouling of gas collection pipes and pumps.", "font": {"size": 12}, "id": "n294", "iri": "http://purl.obolibrary.org/obo/NCIT_C176634 ", "label": "Foam Stabilization", "labels": ":BiologicalProcess", "name": "Foam Stabilization", "pagerank_centrality": "0.32141474030642314", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "foam stabilizing agent", "xref": ""}, {"betweenness_centrality": "610.2728105732036", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.26898047722342733", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n278", "label": "Moraxellaceae", "labels": ":Family", "name": "Moraxellaceae", "pagerank_centrality": "1.0394134925554734", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 25, "synonym": "[\u0027\"Acinetobacteraceae\" RELATED synonym []\u0027, \u0027\"Branhamaceae\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:9542079\u0027]"}, {"betweenness_centrality": "197.0653007696744", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2938388625592417", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "14.0", "font": {"size": 12}, "id": "n13", "label": "Enhydrobacter", "labels": ":Genus", "name": "Enhydrobacter", "pagerank_centrality": "1.3687157579071623", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 14, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:22145860\u0027]"}, {"betweenness_centrality": "165.04437617937614", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.18618618618618618", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n15", "label": "Bifidobacteriales", "labels": ":Order", "name": "Bifidobacteriales", "pagerank_centrality": "0.8140898120834644", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 14, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19244447\u0027, \u0027PMID:27446019\u0027]"}, {"betweenness_centrality": "114.15948440448445", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.17342657342657342", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n221", "label": "Actinomycetes", "labels": ":Class", "name": "Actinomycetes", "pagerank_centrality": "0.8298115972566475", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"Actinobacteria Stackebrandt et al. 1997\" RELATED synonym []\u0027, \u0027\"Actinomycetia\" RELATED synonym []\u0027, \u0027\"high G+C Gram-positive bacteria\" EXACT genbank_common_name []\u0027, \u0027\"high G+C Gram-positive bacteria\" RELATED blast_name []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10028260\u0027, \u0027PMID:11155976\u0027, \u0027PMID:11321122\u0027, \u0027PMID:19244447\u0027, \u0027PMID:28840812\u0027, \u0027PMID:30186281\u0027]"}, {"betweenness_centrality": "155.52277722277725", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.18397626112759644", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n16", "label": "Bacteroidales", "labels": ":Order", "name": "Bacteroidales", "pagerank_centrality": "0.8479151323527702", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 13, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:28905708\u0027]"}, {"betweenness_centrality": "102.56563436563437", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1700960219478738", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n222", "label": "Bacteroidia", "labels": ":Class", "name": "Bacteroidia", "pagerank_centrality": "0.8472885232872919", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"Bacteroidetes\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "132.49393785740554", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20361247947454844", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n17", "label": "Lactobacillales", "labels": ":Order", "name": "Lactobacillales", "pagerank_centrality": "0.7308881253822399", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 13, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "67.89523655870418", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.18535127055306427", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n223", "label": "Bacilli", "labels": ":Class", "name": "Bacilli", "pagerank_centrality": "0.7456185921800391", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 11, "synonym": "[\u0027\"Bacillus/Lactobacillus/Streptococcus group\" RELATED synonym []\u0027, \u0027\"Firmibacteria\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "133.6053734481763", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.21124361158432708", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n18", "label": "Lachnospirales", "labels": ":Order", "name": "Lachnospirales", "pagerank_centrality": "0.6420483905457826", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 13, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "155.44439025119325", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.19558359621451105", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n224", "label": "Clostridia", "labels": ":Class", "name": "Clostridia", "pagerank_centrality": "1.0031630009016748", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"Natranaerobiia\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:26410691\u0027, \u0027PMID:31076745\u0027, \u0027PMID:32955149\u0027]"}, {"betweenness_centrality": "72.52699235086105", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20563847429519072", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n19", "label": "Eubacteriales", "labels": ":Order", "name": "Eubacteriales", "pagerank_centrality": "0.6986713271799944", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 11, "synonym": "[\u0027\"Clostridiales\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:24480908\u0027]"}, {"betweenness_centrality": "117.17380476330811", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20097244732576985", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n20", "label": "Erysipelotrichales", "labels": ":Order", "name": "Erysipelotrichales", "pagerank_centrality": "0.740275523082984", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "56.27515188110262", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1831610044313146", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n225", "label": "Erysipelotrichia", "labels": ":Class", "name": "Erysipelotrichia", "pagerank_centrality": "0.7495931523410573", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:23606477\u0027]"}, {"betweenness_centrality": "49.188480832157055", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20294599018003273", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n21", "label": "Rhodobacterales", "labels": ":Order", "name": "Rhodobacterales", "pagerank_centrality": "0.6891381439906831", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16403855\u0027]"}, {"betweenness_centrality": "148.35233964052486", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2042833607907743", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n226", "label": "Alphaproteobacteria", "labels": ":Class", "name": "Alphaproteobacteria", "pagerank_centrality": "0.9925976661091573", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"a-proteobacteria\" RELATED blast_name []\u0027, \u0027\"alpha proteobacteria\" RELATED synonym []\u0027, \u0027\"alpha subdivision\" RELATED synonym []\u0027, \u0027\"alpha subgroup\" RELATED synonym []\u0027, \u0027\"Alphabacteria\" RELATED synonym []\u0027, \u0027\"Proteobacteria alpha subdivision\" RELATED synonym []\u0027, \u0027\"Purple bacteria, alpha subdivision\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11541974\u0027, \u0027PMID:11837318\u0027, \u0027PMID:16166687\u0027, \u0027PMID:16403855\u0027, \u0027PMID:19060069\u0027]"}, {"betweenness_centrality": "86.53753607503607", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1887366818873668", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n22", "label": "Neisseriales", "labels": ":Order", "name": "Neisseriales", "pagerank_centrality": "0.7700544628399291", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:23575986\u0027]"}, {"betweenness_centrality": "169.72478114791156", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1884498480243161", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n227", "label": "Betaproteobacteria", "labels": ":Class", "name": "Betaproteobacteria", "pagerank_centrality": "1.0619951208648033", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 14, "synonym": "[\u0027\"b-proteobacteria\" RELATED blast_name []\u0027, \u0027\"beta proteobacteria\" RELATED synonym []\u0027, \u0027\"beta subdivision\" RELATED synonym []\u0027, \u0027\"beta subgroup\" RELATED synonym []\u0027, \u0027\"Proteobacteria beta subdivision\" RELATED synonym []\u0027, \u0027\"Purple bacteria, beta subdivision\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16403855\u0027, \u0027PMID:28581923\u0027]"}, {"betweenness_centrality": "123.12802281365319", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n23", "label": "Nitrosomonadales", "labels": ":Order", "name": "Nitrosomonadales", "pagerank_centrality": "0.7315260404705041", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"Ammonia oxidizing bacteria\" RELATED synonym []\u0027, \u0027\"Ammonia-oxidizing bacteria\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10028253\u0027, \u0027PMID:11760943\u0027, \u0027PMID:16403855\u0027, \u0027PMID:25017294\u0027, \u0027PMID:25701846\u0027, \u0027PMID:28581923\u0027]"}, {"betweenness_centrality": "97.76372066529954", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.17342657342657342", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n24", "label": "Chromatiales", "labels": ":Order", "name": "Chromatiales", "pagerank_centrality": "0.7808859466944127", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"Chromatiaceae/Ectothiorhodospiraceae group\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "293.216628372797", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20462046204620463", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n228", "label": "Gammaproteobacteria", "labels": ":Class", "name": "Gammaproteobacteria", "pagerank_centrality": "1.0367671607531481", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 17, "synonym": "[\u0027\"g-proteobacteria\" RELATED blast_name []\u0027, \u0027\"gamma proteobacteria\" RELATED synonym []\u0027, \u0027\"gamma subdivision\" RELATED synonym []\u0027, \u0027\"gamma subgroup\" RELATED synonym []\u0027, \u0027\"Proteobacteria gamma subdivision\" RELATED synonym []\u0027, \u0027\"Pseudomonadia\" RELATED synonym []\u0027, \u0027\"Purple bacteria, gamma subdivision\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:23334881\u0027]"}, {"betweenness_centrality": "279.4697075742973", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.22752293577981653", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n25", "label": "Moraxellales", "labels": ":Order", "name": "Moraxellales", "pagerank_centrality": "0.6612234118597824", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 17, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "148.7982984467335", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.22627737226277372", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n26", "label": "Hyphomicrobiales", "labels": ":Order", "name": "Hyphomicrobiales", "pagerank_centrality": "0.6488009351555782", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"alpha-2 proteobacteria\" RELATED in_part []\u0027, \u0027\"rhizobacteria\" RELATED synonym []\u0027, \u0027\"Rhizobiaceae group\" RELATED synonym []\u0027, \u0027\"Rhizobiales\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:1854635\u0027]"}, {"betweenness_centrality": "111.92387834387837", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.17174515235457063", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n35", "label": "Actinomycetota", "labels": ":Phylum", "name": "Actinomycetota", "pagerank_centrality": "0.7936264819847738", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"Actinobacteraeota\" RELATED synonym []\u0027, \u0027\"Actinobacteria\" RELATED synonym []\u0027, \u0027\"actinobacteria\" RELATED blast_name []\u0027, \u0027\"Actinobacteriota\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11837318\u0027, \u0027PMID:16280504\u0027, \u0027PMID:29458499\u0027]"}, {"betweenness_centrality": "414.0834908236273", "border-color": "#cf8f44ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1870286576168929", "color": {"background": "#e9ccaaff", "border": "#cf8f44ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n14", "label": "Bacteria", "labels": ":Kingdom", "name": "Bacteria", "pagerank_centrality": "1.3881510771567105", "propertyValue": "[\u0027has_rank NCBITaxon:superkingdom\u0027]", "shape": "dot", "size": 20, "synonym": "[\u0027\"bacteria\" RELATED blast_name []\u0027, \u0027\"eubacteria\" EXACT genbank_common_name []\u0027, \u0027\"Monera\" RELATED in_part []\u0027, \u0027\"Procaryotae\" RELATED in_part []\u0027, \u0027\"Prokaryota\" RELATED in_part []\u0027, \u0027\"Prokaryotae\" RELATED in_part []\u0027, \u0027\"prokaryote\" RELATED in_part []\u0027, \u0027\"prokaryotes\" RELATED in_part []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10425795\u0027, \u0027PMID:10425796\u0027, \u0027PMID:10425797\u0027, \u0027PMID:10490293\u0027, \u0027PMID:10843050\u0027, \u0027PMID:10939651\u0027, \u0027PMID:10939673\u0027, \u0027PMID:10939677\u0027, \u0027PMID:11211268\u0027, \u0027PMID:11321083\u0027, \u0027PMID:11321113\u0027, \u0027PMID:11411719\u0027, \u0027PMID:11540071\u0027, \u0027PMID:11542017\u0027, \u0027PMID:11542087\u0027, \u0027PMID:11760965\u0027, \u0027PMID:12054223\u0027, \u0027PMID:2112744\u0027, \u0027PMID:270744\u0027, \u0027PMID:32628106\u0027, \u0027PMID:36748408\u0027, \u0027PMID:7520741\u0027, \u0027PMID:8123559\u0027, \u0027PMID:8186100\u0027, \u0027PMID:8590690\u0027, \u0027PMID:9103655\u0027, \u0027PMID:9336922\u0027]"}, {"betweenness_centrality": "97.35000000000002", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.16824966078697423", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n36", "label": "Bacteroidota", "labels": ":Phylum", "name": "Bacteroidota", "pagerank_centrality": "0.8009494021766413", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"\\\"Bacteroidota\\\" Whitman et al. 2018\" RELATED synonym []\u0027, \u0027\"Bacteroides-Cytophaga-Flexibacter group\" EXACT OMO:0003003 []\u0027, \u0027\"Bacteroidetes\" RELATED synonym []\u0027, \u0027\"BCF group\" EXACT equivalent_name []\u0027, \u0027\"CFB group\" EXACT equivalent_name []\u0027, \u0027\"CFB group bacteria\" RELATED blast_name []\u0027, \u0027\"Cytophaga-Flexibacter-Bacteroides phylum\" EXACT OMO:0003003 []\u0027, \u0027\"Sphingobacteria\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11541229\u0027, \u0027PMID:11542017\u0027, \u0027PMID:28066339\u0027]"}, {"betweenness_centrality": "248.41510440900709", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.18507462686567164", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n37", "label": "Bacillota", "labels": ":Phylum", "name": "Bacillota", "pagerank_centrality": "1.357590285217961", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 16, "synonym": "[\u0027\"\\\"Firmicutes\\\" corrig. Gibbons and Murray 1978 (Approved Lists 1980)\" RELATED synonym []\u0027, \u0027\"Bacillaeota\" RELATED synonym []\u0027, \u0027\"Bacillota corrig. Gibbons and Murray 2021\" RELATED synonym []\u0027, \u0027\"Bacillus/Clostridium group\" RELATED synonym []\u0027, \u0027\"clostridial firmicutes\" RELATED synonym []\u0027, \u0027\"Clostridium group firmicutes\" RELATED synonym []\u0027, \u0027\"Firmicuteota\" RELATED synonym []\u0027, \u0027\"Firmicutes\" EXACT equivalent_name []\u0027, \u0027\"firmicutes\" RELATED blast_name []\u0027, \u0027\"Low G+C firmicutes\" RELATED synonym []\u0027, \u0027\"low G+C Gram-positive bacteria\" EXACT OMO:0003003 []\u0027, \u0027\"low GC Gram+\" EXACT OMO:0003003 []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10555317\u0027, \u0027PMID:11034484\u0027, \u0027PMID:11542017\u0027, \u0027PMID:15143038\u0027, \u0027PMID:25403554\u0027, \u0027PMID:37409558\u0027]"}, {"betweenness_centrality": "384.1589269617612", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.19195046439628483", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n38", "label": "Pseudomonadota", "labels": ":Phylum", "name": "Pseudomonadota", "pagerank_centrality": "1.3136075894569015", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 19, "synonym": "[\u0027\"\\\"Alphaproteobacteriota\\\" Whitman et al. 2018\" RELATED synonym []\u0027, \u0027\"Alphaproteobacteriota\" RELATED synonym []\u0027, \u0027\"Proteobacteria\" RELATED synonym []\u0027, \u0027\"proteobacteria\" RELATED blast_name []\u0027, \u0027\"Proteobacteriota\" RELATED synonym []\u0027, \u0027\"purple photosynthetic bacteria and relatives\" EXACT OMO:0003003 []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11321122\u0027, \u0027PMID:11837318\u0027]"}, {"betweenness_centrality": "1568.0915879158688", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.30024213075060535", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "48.0", "font": {"size": 12}, "id": "n39", "label": "Bifidobacterium adolescentis", "labels": ":Species", "name": "Bifidobacterium adolescentis", "pagerank_centrality": "6.337300727974077", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 49, "synonym": "[\u0027\"Bifidobacterium stercoris\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:20081020\u0027, \u0027PMID:24187022\u0027]"}, {"betweenness_centrality": "580.9626664606461", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.29453681710213775", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "22.0", "font": {"size": 12}, "id": "n40", "label": "Bifidobacterium longum", "labels": ":Species", "name": "Bifidobacterium longum", "pagerank_centrality": "2.527395127096061", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 24, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12508852\u0027, \u0027PMID:8347514\u0027]"}, {"betweenness_centrality": "382.96393217893217", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.27927927927927926", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n43", "label": "Blautia producta", "labels": ":Species", "name": "Blautia producta", "pagerank_centrality": "1.4159042671889546", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 19, "synonym": "[\u0027\"Peptostreptococcus productus\" RELATED synonym []\u0027, \u0027\"Ruminococcus productus\" RELATED synonym []\u0027, \u0027\"Streptococcus productus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:18676476\u0027, \u0027PMID:8123556\u0027, \u0027PMID:8590686\u0027]"}, {"betweenness_centrality": "585.2649981984382", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2987951807228916", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "14.0", "font": {"size": 12}, "id": "n47", "label": "Eubacterium biformis", "labels": ":Species", "name": "Eubacterium biformis", "pagerank_centrality": "1.5017560546575859", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 24, "synonym": "[\u0027\"Bacteroides biformis\" RELATED synonym []\u0027, \u0027\"Eubacterium biforme\" RELATED synonym []\u0027, \u0027\"Pseudobacterium biforme\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:25180093\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#256491c9", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ActivityAndBehavior", "closeness_centrality": "0.23134328358208955", "color": {"background": "#77b3ddc9", "border": "#256491c9"}, "degree_centrality": "2.0", "description": "an anaerobe that cannot grow in the presence of oxygen", "font": {"size": 12}, "id": "n284", "iri": "", "label": "obligate anaerobe", "labels": ":ActivityAndBehavior", "name": "obligate anaerobe", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "54.03781229949961", "border-color": "#256491c9", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ActivityAndBehavior", "closeness_centrality": "0.27312775330396477", "color": {"background": "#77b3ddc9", "border": "#256491c9"}, "degree_centrality": "6.0", "description": "an organism that does not require oxygen for growth", "font": {"size": 12}, "id": "n280", "iri": "", "label": "anaerobe", "labels": ":ActivityAndBehavior", "name": "anaerobe", "pagerank_centrality": "0.7768663320458286", "providedBy": "KG-Microbe", "shape": "dot", "size": 11, "synonym": "anaerobic organism", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23134328358208955", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n153", "iri": "", "label": "BIFIDOBACTERIUM MEDIUM (DSMZ Medium 58)", "labels": ":ChemicalEntity", "name": "BIFIDOBACTERIUM MEDIUM (DSMZ Medium 58)", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "238.4799673666547", "border-color": "#e75d24ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "PhenotypicFeature", "closeness_centrality": "0.30317848410757947", "color": {"background": "#f3ab8eff", "border": "#e75d24ff"}, "degree_centrality": "8.0", "description": "", "font": {"size": 12}, "id": "n27", "iri": "", "label": "bacillus", "labels": ":PhenotypicFeature", "name": "bacillus", "pagerank_centrality": "0.9500009163453295", "providedBy": "KG-Microbe", "shape": "dot", "size": 15, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#e75d24ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "PhenotypicFeature", "closeness_centrality": "0.23134328358208955", "color": {"background": "#f3ab8eff", "border": "#e75d24ff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n34", "iri": "", "label": "irregular", "labels": ":PhenotypicFeature", "name": "irregular", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "251.70324125908482", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.29594272076372313", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "6.0", "description": "An environmental system which has its properties and dynamics determined by feces.", "font": {"size": 12}, "id": "n251", "iri": "http://purl.obolibrary.org/obo/ENVO_01001029", "label": "fecal environment", "labels": ":EnvironmentalFeature", "name": "fecal environment", "pagerank_centrality": "0.7737742508015693", "providedBy": "KG-Microbe", "shape": "dot", "size": 16, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.23134328358208955", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "2.0", "description": "Anatomical structure that is an individual member of a species and consists of more than one cell.", "font": {"size": 12}, "id": "n259", "iri": "http://purl.obolibrary.org/obo/UBERON_0000468", "label": "multicellular organism", "labels": ":EnvironmentalFeature", "name": "multicellular organism", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "Koerper|body|multi-cellular organism|organism|whole body|whole organism", "xref": ""}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "A glycosylglucose consisting of two D-glucopyranose units connected by an alpha-(1-\u003e4)-linkage.", "font": {"size": 12}, "id": "n101", "iri": "http://purl.obolibrary.org/obo/CHEBI_17306", "label": "maltose", "labels": ":ChemicalEntity", "name": "maltose", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "1-alpha-D-Glucopyranosyl-4-alpha-D-glucopyranose|4-(alpha-D-glucopyranosido)-alpha-glucopyranose|4-(alpha-D-glucosido)-D-glucose|4-O-alpha-D-glucopyranosyl-D-glucopyranose|4-O-alpha-D-glucopyranosyl-D-glucose|Cextromaltose|D-(+)-maltose|D-maltose|Malt sugar|Maltose|Malzzucker|alpha-D-Glcp-(1-\u003e4)-D-Glcp|alpha-D-Glucopyranosyl-(1-\u003e4)-D-glucopyranose|alpha-D-glucopyranosyl-(1-\u003e4)-D-glucopyranose|alpha-D-glucopyranosyl-(1-\u003e4)-D-glucose|alpha-malt sugar|maltobiose", "xref": "Beilstein:1292747|CAS:69-79-4|DrugBank:DB03323|KEGG:C00208|KEGG:D00044|KEGG:G00275|KNApSAcK:C00001140|PMID:16332759|PMID:17723085|PMID:22094343|PMID:22185612|PMID:22246222|PMID:22252265|PMID:22411612|PMID:22424089|PMID:22451670|PMID:22469630|PMID:22529943|PMID:22573161|PMID:22669197|Reaxys:1292747|Wikipedia:Maltose"}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "An aldohexose used as a source of energy and metabolic intermediate.", "font": {"size": 12}, "id": "n98", "iri": "http://purl.obolibrary.org/obo/CHEBI_17234", "label": "glucose", "labels": ":ChemicalEntity", "name": "glucose", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "DL-glucose|Glc|Glucose|Glukose|gluco-hexose|glucose", "xref": "CAS:50-99-7|KEGG:C00293|Wikipedia:Glucose"}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "A glycosylglucose consisting of two glucose units linked via a beta(1-\u003e4) bond.", "font": {"size": 12}, "id": "n94", "iri": "http://purl.obolibrary.org/obo/CHEBI_17057", "label": "cellobiose", "labels": ":ChemicalEntity", "name": "cellobiose", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "1-beta-D-Glucopyranosyl-4-D-glucopyranose|4-(beta-D-glucosido)-D-glucose|4-O-beta-D-glucopyranosyl-D-glucopyranose|4-beta-D-glucopyranosyl-D-glucopyranose|Cellobiose|D-(+)-cellobiose|D-celiobiose|D-cellobiose|D-glucosyl-beta-(1-4)-D-glucose|beta-D-glucopyranosyl-(1-\u003e4)-D-glucopyranose|cellose", "xref": "Beilstein:1292744|CAS:528-50-7|DrugBank:DB02061|KEGG:C00185|KNApSAcK:C00001134|PMID:19669625|PMID:19968554|Wikipedia:Cellobiose"}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "A trisaccharide composed of alpha-D-galactopyranose, alpha-D-glucopyranose and beta-D-fructofuranose joined in sequence by 1-\u003e6 and 1\u003c-\u003e2 glycosidic linkages, respectively.", "font": {"size": 12}, "id": "n90", "iri": "http://purl.obolibrary.org/obo/CHEBI_16634", "label": "raffinose", "labels": ":ChemicalEntity", "name": "raffinose", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "6G-alpha-D-galactosylsucrose|Gossypose|Melitose|Melitriose|Raffinose|alpha-D-Galp-(1-\u003e6)-alpha-D-Glcp-(1\u003c-\u003e2)-beta-D-Fruf|alpha-D-galactopyranosyl-(1-\u003e6)-alpha-D-glucopyranosyl beta-D-fructofuranoside|beta-D-fructofuranosyl alpha-D-galactopyranosyl-(1-\u003e6)-alpha-D-glucopyranoside|raffinose|rafinose|raflinose", "xref": "CAS:512-69-6|HMDB:HMDB0003213|KEGG:C00492|KEGG:G00249|KNApSAcK:C00001145|MetaCyc:CPD-1099|PDBeChem:RAF|PMID:23317449|PMID:23879777|PMID:23882273|PMID:24001862|PMID:24354450|PMID:24360500|Reaxys:99543|Wikipedia:Raffinose"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23134328358208955", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A glycosylglucose formed by an alpha-(1-\u003e6)-linkage between D-galactose and D-glucose.", "font": {"size": 12}, "id": "n128", "iri": "http://purl.obolibrary.org/obo/CHEBI_28053", "label": "melibiose", "labels": ":ChemicalEntity", "name": "melibiose", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "(Gal)1 (Glc)1|6-O-(alpha-D-Galactopyranosyl)-D-glucopyranose|6-O-alpha-D-galactopyranosyl-D-glucopyranose|D-Melibiose|D-mellibiose|Gal-alpha(1,6)Glc|Melibiose|WURCS=2.0/2,2,1/[a2122h-1x_1-5][a2112h-1a_1-5]/1-2/a6-b1|alpha-D-Gal-(1-\u003e6)-D-Glc|alpha-D-Galp-(1-\u003e6)-D-Glcp|alpha-D-galactopyranosyl-(1-\u003e6)-D-glucopyranose|alpha-D-galactosyl-(1-\u003e6)-D-glucose|melibiose", "xref": "Beilstein:1292781|CAS:585-99-9|GlyGen:G09165XB|GlyTouCan:G09165XB|Gmelin:887780|HMDB:HMDB0000048|KEGG:C05402|KEGG:G01275|KNApSAcK:C00001142|MetaCyc:MELIBIOSE|PMID:12797744|PMID:17006649|PMID:17724458|PMID:19846069|PMID:19913595|PMID:27577255|PMID:27794607|PMID:28453942|PMID:3290105|PMID:718021|PMID:7524207|Reaxys:1292781|Wikipedia:Melibiose"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23134328358208955", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "Any aldopentose where the open-chain form has all the hydroxy groups on the same side in the Fischer projection. Occurrs in two enantiomeric forms, D- and L-ribose, of which only the former is found in nature.", "font": {"size": 12}, "id": "n138", "iri": "http://purl.obolibrary.org/obo/CHEBI_33942", "label": "ribose", "labels": ":ChemicalEntity", "name": "ribose", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "Rib|ribo-pentose|ribose", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23134328358208955", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A polydisperse, highly branched glucan composed of chains of D-glucopyranose residues in alpha(1-\u003e4) glycosidic linkage, joined together by alpha(1-\u003e6) glycosidic linkages. A small number of alpha(1-\u003e3) glycosidic linkages and some cumulative alpha(1-\u003e6) links also may occur. The branches in glycogen typically contain 8 to 12 glucose residues.", "font": {"size": 12}, "id": "n129", "iri": "http://purl.obolibrary.org/obo/CHEBI_28087", "label": "glycogen", "labels": ":ChemicalEntity", "name": "glycogen", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "Glycogen|WURCS=2.0/1,4,3/[a2122h-1a_1-5]/1-1-1-1/a4-b1_b4-c1_b6-d1|animal starch|liver starch", "xref": "CAS:9005-79-2|GlyGen:G99991IU|GlyTouCan:G99991IU|HMDB:HMDB0000757|KEGG:C00182|MetaCyc:CPD0-971|Wikipedia:Glycogen"}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "A glycosylglucose disaccharide, found most notably in milk, that consists of D-galactose and D-glucose fragments bonded through a beta-1-\u003e4 glycosidic linkage. The glucose fragment can be in either the alpha- or beta-pyranose form, whereas the galactose fragment can only have the beta-pyranose form.", "font": {"size": 12}, "id": "n103", "iri": "http://purl.obolibrary.org/obo/CHEBI_17716", "label": "lactose", "labels": ":ChemicalEntity", "name": "lactose", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "(+)-lactose|(Gal)1 (Glc)1|1-beta-D-Galactopyranosyl-4-D-glucopyranose|4-(beta-D-galactosido)-D-glucose|4-O-beta-D-galactopyranosyl-D-glucose|D-lactose|Galbeta1-4Glc|Lac|Laktobiose|Laktose|Milchzucker|Milk sugar|WURCS=2.0/2,2,1/[a2122h-1x_1-5][a2112h-1b_1-5]/1-2/a4-b1|beta-D-Gal-(1-\u003e4)-D-Glc|beta-D-Galp-(1-\u003e4)-D-Glcp|beta-D-galactopyranosyl-(1-\u003e4)-D-glucopyranose|beta-Gal1,4-Glc|lactobiose|lactose|milk sugar", "xref": "Beilstein:1292745|CAS:63-42-3|GlyGen:G74621DY|GlyTouCan:G74621DY|Gmelin:882872|KEGG:C00243|KEGG:D00046|KEGG:G10504|KNApSAcK:C00001136|PMID:1292745|PMID:17329833|PMID:18300214|PMID:19053747|PMID:19846069|PMID:19913595|PMID:20094999|PMID:20503067|PMID:20699559|PMID:20873837|PMID:20961532|PMID:21403918|PMID:2432147|PMID:2456994|PMID:28690131|PMID:28807538|PMID:6194884|PMID:7574700|Reaxys:1292745"}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "An aryl beta-D-glucoside that is salicyl alcohol in which the phenolic hydrogen has been replaced by a beta-D-glucosyl residue.", "font": {"size": 12}, "id": "n107", "iri": "http://purl.obolibrary.org/obo/CHEBI_17814", "label": "salicin", "labels": ":ChemicalEntity", "name": "salicin", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "2(hydroxymethyl)phenyl-beta-D-glucopyranoside|2-(hydroxymethyl)phenyl beta-D-glucopyranoside|2-(hydroxymethyl)phenyl-O-beta-D-glucopyranoside|2-(hydroxymethyl)phenyl-beta-D-glucopyranoside|D-(-)-salicin|Salicin|Salicoside|o-(hydroxymethyl)phenyl beta-D-glucopyranoside|salicin|salicyl alcohol glucoside|saligenin beta-D-glucopyranoside", "xref": "CAS:138-52-3|KEGG:C01451|KNApSAcK:C00002672|LINCS:LSM-3442|PMID:12221594|PMID:20883292|PMID:21226125|PMID:21449910|Reaxys:89593|Wikipedia:Salicin"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23134328358208955", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A disaccharide formed by a (1\u003c-\u003e1)-glycosidic bond between two units of D-glucose.", "font": {"size": 12}, "id": "n124", "iri": "http://purl.obolibrary.org/obo/CHEBI_27082", "label": "trehalose", "labels": ":ChemicalEntity", "name": "trehalose", "pagerank_centrality": "0.37269846673126045", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "An aldopentose, found in the embryos of most edible plants and used in medicine to test for malabsorption by administration in water to the patient.", "font": {"size": 12}, "id": "n111", "iri": "http://purl.obolibrary.org/obo/CHEBI_18222", "label": "xylose", "labels": ":ChemicalEntity", "name": "xylose", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "DL-xylose|Xyl|Xylose|xylo-pentose|xylose", "xref": "KEGG:C01394|Wikipedia:Xylose"}, {"betweenness_centrality": "21.5323718507155", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2678185745140389", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "", "font": {"size": 12}, "id": "n114", "iri": "http://purl.obolibrary.org/obo/CHEBI_22599", "label": "arabinose", "labels": ":ChemicalEntity", "name": "arabinose", "pagerank_centrality": "0.5831577909960495", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "Ara|arabino-pentose|arabinose", "xref": "CAS:147-81-9|HMDB:HMDB0029942|Wikipedia:Arabinose"}, {"betweenness_centrality": "99.06531385281383", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.26898047722342733", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "A hexitol produced by a variety of organisms including bacteria, fungi, lichens and plants.", "font": {"size": 12}, "id": "n134", "iri": "http://purl.obolibrary.org/obo/CHEBI_29864", "label": "mannitol", "labels": ":ChemicalEntity", "name": "mannitol", "pagerank_centrality": "0.5532554860076495", "providedBy": "KG-Microbe", "shape": "dot", "size": 12, "synonym": "mannitol", "xref": "PMID:24269997|PMID:24323504|PMID:24374122|PMID:24861101|Wikipedia:Mannitol"}, {"betweenness_centrality": "66.59893939393942", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.26439232409381663", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n44", "label": "Dorea formicigenerans", "labels": ":Species", "name": "Dorea formicigenerans", "pagerank_centrality": "0.8923400577123091", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 11, "synonym": "[\u0027\"Eubacterium formicigenerans\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11931151\u0027]"}, {"betweenness_centrality": "192.01231282158713", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2577962577962578", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n45", "label": "Mediterraneibacter gnavus", "labels": ":Species", "name": "Mediterraneibacter gnavus", "pagerank_centrality": "1.448615773627613", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 14, "synonym": "[\u0027\"Mediterraneibacter gnavus (Moore et al. 1976) Togo et al. 2023\" RELATED synonym []\u0027, \u0027\"Ruminococcus gnavus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:29855844\u0027]"}, {"betweenness_centrality": "103.60597644787276", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.2917647058823529", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "4.0", "description": "", "font": {"size": 12}, "id": "n262", "iri": "", "label": "host_animal_endotherm", "labels": ":EnvironmentalFeature", "name": "host_animal_endotherm", "pagerank_centrality": "0.5303950959217912", "providedBy": "KG-Microbe", "shape": "dot", "size": 12, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.22794117647058823", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n265", "iri": "", "label": "vagina", "labels": ":EnvironmentalFeature", "name": "vagina", "pagerank_centrality": "0.34370854104977916", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "104.26264450204079", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.28054298642533937", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "6.0", "description": "An environmental system determined by an intestine.", "font": {"size": 12}, "id": "n255", "iri": "http://purl.obolibrary.org/obo/ENVO_2100002", "label": "intestine environment", "labels": ":EnvironmentalFeature", "name": "intestine environment", "pagerank_centrality": "0.7165536991802286", "providedBy": "KG-Microbe", "shape": "dot", "size": 12, "synonym": "", "xref": ""}, {"betweenness_centrality": "1384.1085177206699", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.3139240506329114", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "34.0", "font": {"size": 12}, "id": "n41", "label": "Segatella copri", "labels": ":Species", "name": "Segatella copri", "pagerank_centrality": "4.2441032798001075", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 44, "synonym": "[\u0027\"Prevotella copri Hayashi et al. 2007\" RELATED synonym []\u0027, \u0027\"Segatella copri (Hayashi et al. 2007) Hitch et al. 2023\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "338.38264567676345", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.30617283950617286", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "14.0", "font": {"size": 12}, "id": "n46", "label": "Faecalibacterium prausnitzii", "labels": ":Species", "name": "Faecalibacterium prausnitzii", "pagerank_centrality": "1.5958584083262326", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 18, "synonym": "[\u0027\"Bacillus mucosus anaerobius\" RELATED synonym []\u0027, \u0027\"Bacteroides praussnitzii\" RELATED synonym []\u0027, \u0027\"Fusobacterium prausnitzii\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12508881\u0027, \u0027PMID:7981107\u0027, \u0027PMID:8573517\u0027, \u0027PMID:8863411\u0027]"}, {"betweenness_centrality": "366.10536020996574", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.29107981220657275", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n49", "label": "Neisseria cinerea", "labels": ":Species", "name": "Neisseria cinerea", "pagerank_centrality": "1.4235862276570754", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 19, "synonym": "[\u0027\"Micrococcus cinereus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "1596.95418145494", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.34831460674157305", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "34.0", "font": {"size": 12}, "id": "n55", "label": "Acinetobacter lwoffii", "labels": ":Species", "name": "Acinetobacter lwoffii", "pagerank_centrality": "3.7639037563072493", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 50, "synonym": "[\u0027\"Acinetobacter genomosp. 9\" RELATED synonym []\u0027, \u0027\"Acinetobacter genomospecies 9\" RELATED synonym []\u0027, \u0027\"Acinetobacter mesopotamicus\" RELATED synonym []\u0027, \u0027\"Moraxella lwoffi\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:2799067\u0027, \u0027PMID:33136203\u0027, \u0027PMID:4203395\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23938223938223938", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A glycosyl glycoside formed by glucose and fructose units joined by an acetal oxygen bridge from hemiacetal of glucose to the hemiketal of the fructose.", "font": {"size": 12}, "id": "n109", "iri": "http://purl.obolibrary.org/obo/CHEBI_17992", "label": "sucrose", "labels": ":ChemicalEntity", "name": "sucrose", "pagerank_centrality": "0.3604593242647892", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "1-alpha-D-Glucopyranosyl-2-beta-D-fructofuranoside|Cane sugar|SUCROSE|Saccharose|Sacharose|Sucrose|White sugar|beta-D-Fruf-(2\u003c-\u003e1)-alpha-D-Glcp|beta-D-fructofuranosyl alpha-D-glucopyranoside|sacarosa|sucrose|table sugar", "xref": "Beilstein:90825|CAS:57-50-1|DrugBank:DB02772|Drug_Central:4610|Gmelin:97695|HMDB:HMDB0000258|KEGG:C00089|KEGG:D00025|KEGG:D06533|KEGG:G00370|KNApSAcK:C00001151|MetaCyc:SUCROSE|PDBeChem:SUC|PMID:11021636|PMID:11093712|PMID:11111003|PMID:12065720|PMID:12706980|PMID:13508893|PMID:15291457|PMID:15660210|PMID:15792978|PMID:15845855|PMID:16228482|PMID:16304615|PMID:16313996|PMID:16525719|PMID:16660545|PMID:16663947|PMID:16665852|PMID:17233733|PMID:17439666|PMID:17597061|PMID:18625236|PMID:19199566|PMID:19726178|PMID:21703290|PMID:21972845|PMID:22085755|PMID:22311778|PMID:22404833|PMID:22751876|Reaxys:1435311|Reaxys:90825|Wikipedia:Sucrose"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23938223938223938", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A deoxymannose sugar that is the 6-deoxy derivative of hexose.", "font": {"size": 12}, "id": "n121", "iri": "http://purl.obolibrary.org/obo/CHEBI_26546", "label": "rhamnose", "labels": ":ChemicalEntity", "name": "rhamnose", "pagerank_centrality": "0.3604593242647892", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "ramnose", "xref": "PMID:24211429|PMID:24831810"}, {"betweenness_centrality": "418.36904800852176", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2995169082125604", "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "20.0", "font": {"size": 12}, "id": "n42", "label": "Streptococcus minor", "labels": ":Species", "name": "Streptococcus minor", "pagerank_centrality": "2.142068803268611", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 20, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:15023958\u0027]"}, {"betweenness_centrality": "768.7958943347099", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.31313131313131315", "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "30.0", "font": {"size": 12}, "id": "n53", "label": "Acinetobacter guillouiae", "labels": ":Species", "name": "Acinetobacter guillouiae", "pagerank_centrality": "3.3906672930892627", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 29, "synonym": "[\u0027\"Acinetobacter genomic species 11\" RELATED synonym []\u0027, \u0027\"Acinetobacter genomosp. 11\" RELATED synonym []\u0027, \u0027\"Acinetobacter genomospecies 11\" RELATED synonym []\u0027, \u0027\"Acinetobacter genosp. 11\" RELATED synonym []\u0027, \u0027\"Acinetobacter genospecies 11\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19661501\u0027]"}, {"betweenness_centrality": "436.453053303479", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.3237597911227154", "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "26.0", "font": {"size": 12}, "id": "n54", "label": "Acinetobacter johnsonii", "labels": ":Species", "name": "Acinetobacter johnsonii", "pagerank_centrality": "2.6712636633836317", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 20, "synonym": "[\u0027\"Acinetobacter genomosp. 7\" RELATED synonym []\u0027, \u0027\"Acinetobacter genomospecies 7\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "275.0181751971227", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.3024390243902439", "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "16.0", "font": {"size": 12}, "id": "n58", "label": "Enhydrobacter aerosaccus", "labels": ":Species", "name": "Enhydrobacter aerosaccus", "pagerank_centrality": "1.7313753828578218", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 16, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:22145860\u0027]"}, {"betweenness_centrality": "40.03285233285235", "border-color": "#256491c9", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ActivityAndBehavior", "closeness_centrality": "0.2811791383219955", "color": {"background": "#77b3ddc9", "border": "#256491c9"}, "degree_centrality": "12.0", "description": "an organism that can survive and grow in an oxygenated environment", "font": {"size": 12}, "id": "n281", "iri": "", "label": "aerobe", "labels": ":ActivityAndBehavior", "name": "aerobe", "pagerank_centrality": "1.2571794262644558", "providedBy": "KG-Microbe", "shape": "dot", "size": 11, "synonym": "aerobic organism", "xref": ""}, {"betweenness_centrality": "10.976190476190476", "border-color": "#256491c9", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ActivityAndBehavior", "closeness_centrality": "0.2530612244897959", "color": {"background": "#77b3ddc9", "border": "#256491c9"}, "degree_centrality": "4.0", "description": "an aerobe that can grow at oxygen levels lower than what is present in the atmosphere", "font": {"size": 12}, "id": "n279", "iri": "", "label": "microaerophile", "labels": ":ActivityAndBehavior", "name": "microaerophile", "pagerank_centrality": "0.530720503974259", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "microaerophilic organism", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#256491c9", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ActivityAndBehavior", "closeness_centrality": "0.24603174603174602", "color": {"background": "#77b3ddc9", "border": "#256491c9"}, "degree_centrality": "4.0", "description": "an aerobe that requires oxygen to grow", "font": {"size": 12}, "id": "n283", "iri": "", "label": "obligate aerobe", "labels": ":ActivityAndBehavior", "name": "obligate aerobe", "pagerank_centrality": "0.5211734790819091", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#e75d24ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "PhenotypicFeature", "closeness_centrality": "0.24603174603174602", "color": {"background": "#f3ab8eff", "border": "#e75d24ff"}, "degree_centrality": "4.0", "description": "", "font": {"size": 12}, "id": "n29", "iri": "", "label": "coccobacillus", "labels": ":PhenotypicFeature", "name": "coccobacillus", "pagerank_centrality": "0.5211734790819091", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.21869488536155202", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The chemical reactions and pathways resulting in the formation of acetate, the anion of acetic acid. Acetogenesis is routinely applied to broadly describe the biological synthesis of acetate. However, this definition is somewhat misleading as several metabolic strategies result in acetate production. MiDAS therefore applies the strict microbiological definition. True acetogens employ the reductive acetyl-CoA pathway for the synthesis of acetyl-CoA that can be converted to acetate or utilized as a source of carbon for biomass production. In this pathway, hydrogen and CO2 are utilized as an electron donor and acceptor, respectively. Acetogens often utilize alternative donors and acceptors to supplement or replace the use of the pathway for energy or carbon supply depending on growth conditions. Commonly, the pathway operates in conjunction with fermentative pathways, operating to replace generated reducing equivalents in the reduction of CO2. Importantly, organisms possessing the reductive acetyl-CoA pathway can reverse the pathway to oxidize acetate to CO2\u00a0and H2 in concert with hydrogenotrophic methanogens. This is known as syntrophic acetate oxidation (SAO) as it requires the methanogens to remove H2 to make the pathway thermodynamically favorable.", "font": {"size": 12}, "id": "n72", "iri": "http://purl.obolibrary.org/obo/GO_0019413", "label": "Acetate biosynthetic process", "labels": ":BiologicalProcess", "name": "Acetate biosynthetic process", "pagerank_centrality": "0.3490990818745196", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 10, "synonym": "acetate anabolism|acetate biosynthesis|acetate formation|acetate synthesis", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2052980132450331", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n157", "iri": "", "label": "PYG MEDIUM (MODIFIED) (DSMZ Medium 104)", "labels": ":ChemicalEntity", "name": "PYG MEDIUM (MODIFIED) (DSMZ Medium 104)", "pagerank_centrality": "0.3536660736853662", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23484848484848486", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n154", "iri": "", "label": "CHOPPED MEAT MEDIUM WITH CARBOHYDRATES (DSMZ Medium 110)", "labels": ":ChemicalEntity", "name": "CHOPPED MEAT MEDIUM WITH CARBOHYDRATES (DSMZ Medium 110)", "pagerank_centrality": "0.34228813885406", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "68.68957153957155", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.28054298642533937", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n48", "label": "Paracoccus marcusii", "labels": ":Species", "name": "Paracoccus marcusii", "pagerank_centrality": "0.6831860915111615", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:9731296\u0027]"}, {"betweenness_centrality": "132.95147300068362", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.22103386809269163", "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n50", "label": "Nitrosomonas mobilis", "labels": ":Species", "name": "Nitrosomonas mobilis", "pagerank_centrality": "0.8785321875992748", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"Nitrosococcus mobilis\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:5979\u0027]"}, {"betweenness_centrality": "132.95147300068362", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.22103386809269163", "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n51", "label": "Nitrosomonas halophila", "labels": ":Species", "name": "Nitrosomonas halophila", "pagerank_centrality": "0.8785321875992748", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 13, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11511864\u0027]"}, {"betweenness_centrality": "164.94294600136692", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.18618618618618618", "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n52", "label": "Thiodictyon syntrophicum", "labels": ":Species", "name": "Thiodictyon syntrophicum", "pagerank_centrality": "0.966349880344243", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 14, "synonym": "[\u0027\"Thiodictyon syntrophicum\" EXACT equivalent_name []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:22386960\u0027, \u0027PMID:7857801\u0027, \u0027PMID:8123559\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2863741339491917", "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n56", "label": "Acinetobacter rhizosphaerae", "labels": ":Species", "name": "Acinetobacter rhizosphaerae", "pagerank_centrality": "0.8718523392834558", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 10, "synonym": "[\u0027\"\u0027Acinetobacter rhizosphaerae\u0027\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "8.178445165945165", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.27555555555555555", "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "10.0", "font": {"size": 12}, "id": "n57", "label": "Acinetobacter schindleri", "labels": ":Species", "name": "Acinetobacter schindleri", "pagerank_centrality": "1.0442087515497425", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 10, "synonym": "[\u0027\"Acinetobacter phenon 2\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11594623\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23892100192678228", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A monocarboxylic acid anion resulting from the removal of a proton from the carboxy group of acetic acid.", "font": {"size": 12}, "id": "n135", "iri": "http://purl.obolibrary.org/obo/CHEBI_30089", "label": "acetate", "labels": ":ChemicalEntity", "name": "acetate", "pagerank_centrality": "0.34061645980552", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "ACETATE ION|Azetat|CH3-COO(-)|Ethanoat|MeCO2 anion|acetate|acetic acid, ion(1-)|ethanoate", "xref": "Beilstein:1901470|CAS:71-50-1|DrugBank:DB03166|Gmelin:1379|KEGG:C00033|MetaCyc:ACET|PDBeChem:ACT|PMID:17190852|PMID:22211106|PMID:22371380|Reaxys:1901470|UM-BBD_compID:c0050|Wikipedia:Acetate"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23892100192678228", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A primary alcohol that is ethane in which one of the hydrogens is substituted by a hydroxy group.", "font": {"size": 12}, "id": "n88", "iri": "http://purl.obolibrary.org/obo/CHEBI_16236", "label": "ethanol", "labels": ":ChemicalEntity", "name": "ethanol", "pagerank_centrality": "0.34061645980552", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "1-hydroxyethane|Aethanol|Aethylalkohol|Alkohol|C2H5OH|Dehydrated ethanol|ETHANOL|EtOH|Ethanol|Ethyl alcohol|Methylcarbinol|[CH2Me(OH)]|[OEtH]|alcohol|alcohol etilico|alcool ethylique|etanol|ethanol|hydroxyethane|spiritus vini", "xref": "Beilstein:1718733|CAS:64-17-5|DrugBank:DB00898|Drug_Central:1076|Gmelin:787|HMDB:HMDB0000108|KEGG:C00469|KEGG:D00068|KEGG:D06542|KNApSAcK:C00019560|MetaCyc:ETOH|MolBase:858|MolBase:859|PDBeChem:EOH|PMID:11046114|PMID:11090978|PMID:11198720|PMID:11200745|PMID:11262320|PMID:11303910|PMID:11333032|PMID:11505026|PMID:11590970|PMID:11728426|PMID:11750186|PMID:11754521|PMID:11810019|PMID:11826039|PMID:11981228|PMID:12824058|PMID:12829422|PMID:12888778|PMID:12946583|PMID:14674846|PMID:15019421|PMID:15239123|PMID:15285839|PMID:15464411|PMID:15465973|PMID:15749123|PMID:15900217|PMID:15902919|PMID:16084479|PMID:16133132|PMID:16352430|PMID:16390872|PMID:16737463|PMID:16891664|PMID:16934862|PMID:17043811|PMID:17190852|PMID:17663926|PMID:17687877|PMID:18095657|PMID:18249266|PMID:18320157|PMID:18347649|PMID:18408978|PMID:18411066|PMID:18456322|PMID:18513832|PMID:18922656|PMID:18925476|PMID:19280886|PMID:19359288|PMID:19384566|PMID:19458312|PMID:19851413|PMID:19901811|PMID:21600756|PMID:21762181|PMID:21881875|PMID:21967628|PMID:22019193|PMID:22222864|PMID:22261437|PMID:22286266|PMID:22306018|PMID:22331491|PMID:22336593|PPDB:1373|Reaxys:1718733|UM-BBD_compID:c0038|Wikipedia:Ethanol"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23892100192678228", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n84", "iri": "http://purl.obolibrary.org/obo/CHEBI_132943", "label": "aspartate", "labels": ":ChemicalEntity", "name": "aspartate", "pagerank_centrality": "0.34061645980552", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.23892100192678228", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "A hydroxy monocarboxylic acid anion that is the conjugate base of lactic acid, arising from deprotonation of the carboxy group.", "font": {"size": 12}, "id": "n117", "iri": "http://purl.obolibrary.org/obo/CHEBI_24996", "label": "lactate", "labels": ":ChemicalEntity", "name": "lactate", "pagerank_centrality": "0.34061645980552", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "2-hydroxypropanoate|2-hydroxypropanoic acid, ion(1-)|2-hydroxypropionate|MeCH(OH)CO2 anion|b-lactate|beta-lactate|lactate", "xref": "Beilstein:3587719|CAS:113-21-3|Gmelin:240074|KEGG:C01432|MetaCyc:Lactate"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2672413793103448", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "4.0", "description": "", "font": {"size": 12}, "id": "n150", "iri": "", "label": "CASO AGAR (MERCK 105458) (DSMZ Medium 220)", "labels": ":ChemicalEntity", "name": "CASO AGAR (MERCK 105458) (DSMZ Medium 220)", "pagerank_centrality": "0.5098211391715126", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.2450592885375494", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "2.0", "description": "An environmental material primarily composed of dihydrogen oxide in its liquid form.", "font": {"size": 12}, "id": "n235", "iri": "http://purl.obolibrary.org/obo/ENVO_00002006", "label": "liquid water", "labels": ":EnvironmentalFeature", "name": "liquid water", "pagerank_centrality": "0.3231345842995007", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "water", "xref": "EcoLexicon:water|LTER:617|https://en.wikipedia.org/wiki/Water"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2588726513569937", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n145", "iri": "", "label": "R2A MEDIUM (DSMZ Medium 830)", "labels": ":ChemicalEntity", "name": "R2A MEDIUM (DSMZ Medium 830)", "pagerank_centrality": "0.336686554872012", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2588726513569937", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n144", "iri": "", "label": "TRYPTICASE SOY YEAST EXTRACT MEDIUM (DSMZ Medium 92)", "labels": ":ChemicalEntity", "name": "TRYPTICASE SOY YEAST EXTRACT MEDIUM (DSMZ Medium 92)", "pagerank_centrality": "0.336686554872012", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.2588726513569937", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The process of binding or confining manganese ions such that they are separated from other components of a biological system.", "font": {"size": 12}, "id": "n76", "iri": "http://purl.obolibrary.org/obo/GO_1990659", "label": "Sequestering of manganese ion", "labels": ":BiologicalProcess", "name": "Sequestering of manganese ion", "pagerank_centrality": "0.336686554872012", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "manganese ion retention|manganese ion sequestering|manganese ion sequestration|manganese ion storage|retention of manganese ion|sequestration of manganese ion|storage of manganese ion", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.2588726513569937", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The chemical reactions and pathways resulting in the breakdown of a hydrocarbon, a compound consisting of carbon and hydrogen only.", "font": {"size": 12}, "id": "n74", "iri": "http://purl.obolibrary.org/obo/GO_0120253", "label": "Hydrocarbon catabolic process", "labels": ":BiologicalProcess", "name": "Hydrocarbon catabolic process", "pagerank_centrality": "0.336686554872012", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "hydrocarbon breakdown|hydrocarbon catabolism|hydrocarbon degradation", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.2326454033771107", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n213", "iri": "", "label": "ENHYDROBACTER MEDIUM (DSMZ Medium 1432)", "labels": ":ChemicalEntity", "name": "ENHYDROBACTER MEDIUM (DSMZ Medium 1432)", "pagerank_centrality": "0.33245501045486797", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}]);
                  edges = new vis.DataSet([{"color": "#a5abb6ff", "font": {"size": 6}, "from": "n0", "id": "e188", "label": "SUBCLASS_OF", "to": "n268", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n1", "id": "e189", "label": "SUBCLASS_OF", "to": "n269", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e81", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e90", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Negative", "id": "e114", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027]", "to": "n290", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n2", "id": "e190", "label": "SUBCLASS_OF", "to": "n270", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e215", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e223", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n292", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n3", "hasattribute": "Negative", "id": "e85", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n3", "hasattribute": "Positive", "id": "e95", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n3", "hasattribute": "Positive", "id": "e118", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n290", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n3", "id": "e191", "label": "SUBCLASS_OF", "to": "n271", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n3", "hasattribute": "Positive", "id": "e220", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n4", "hasattribute": "Negative", "id": "e84", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n4", "hasattribute": "Positive", "id": "e94", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n4", "hasattribute": "Positive", "id": "e117", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n290", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n4", "id": "e192", "label": "SUBCLASS_OF", "to": "n271", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n4", "hasattribute": "Positive", "id": "e219", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n4", "hasattribute": "Negative", "id": "e227", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n292", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n5", "id": "e193", "label": "SUBCLASS_OF", "to": "n271", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n6", "hasattribute": "Negative", "id": "e86", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n6", "hasattribute": "Positive", "id": "e96", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n6", "id": "e194", "label": "SUBCLASS_OF", "to": "n272", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n6", "hasattribute": "Positive", "id": "e221", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n7", "hasattribute": "Positive", "id": "e92", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n7", "id": "e195", "label": "SUBCLASS_OF", "to": "n273", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n7", "hasattribute": "Positive", "id": "e217", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n7", "hasattribute": "Positive", "id": "e225", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n292", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e79", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e88", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Negative", "id": "e110", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n293", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e112", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n290", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n8", "id": "e196", "label": "SUBCLASS_OF", "to": "n274", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Negative", "id": "e214", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e222", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n292", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Negative", "id": "e228", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n288", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n9", "id": "e197", "label": "SUBCLASS_OF", "to": "n275", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e80", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e87", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n69", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e89", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e109", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n65", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Negative", "id": "e111", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n293", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e113", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n290", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Negative", "id": "e133", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n286", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n10", "id": "e198", "label": "SUBCLASS_OF", "to": "n276", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n11", "id": "e199", "label": "SUBCLASS_OF", "to": "n277", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e82", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e91", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e115", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n290", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Negative", "id": "e119", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n289", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e120", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n294", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n12", "id": "e200", "label": "SUBCLASS_OF", "to": "n278", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Negative", "id": "e216", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e224", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n292", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Negative", "id": "e229", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n288", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n13", "hasattribute": "Positive", "id": "e83", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n13", "hasattribute": "Positive", "id": "e93", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n291", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n13", "hasattribute": "Positive", "id": "e116", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n290", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n13", "id": "e201", "label": "SUBCLASS_OF", "to": "n278", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n13", "hasattribute": "Positive", "id": "e218", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n82", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n13", "hasattribute": "Positive", "id": "e226", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n292", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n15", "id": "e121", "label": "SUBCLASS_OF", "to": "n221", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n16", "id": "e122", "label": "SUBCLASS_OF", "to": "n222", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n17", "id": "e123", "label": "SUBCLASS_OF", "to": "n223", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n18", "id": "e124", "label": "SUBCLASS_OF", "to": "n224", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n19", "id": "e125", "label": "SUBCLASS_OF", "to": "n224", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n20", "id": "e126", "label": "SUBCLASS_OF", "to": "n225", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n21", "id": "e127", "label": "SUBCLASS_OF", "to": "n226", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n22", "id": "e128", "label": "SUBCLASS_OF", "to": "n227", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n23", "id": "e129", "label": "SUBCLASS_OF", "to": "n227", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n24", "id": "e130", "label": "SUBCLASS_OF", "to": "n228", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n25", "id": "e131", "label": "SUBCLASS_OF", "to": "n228", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n26", "id": "e132", "label": "SUBCLASS_OF", "to": "n226", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n35", "id": "e105", "label": "SUBCLASS_OF", "to": "n14", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n36", "id": "e106", "label": "SUBCLASS_OF", "to": "n14", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n37", "id": "e107", "label": "SUBCLASS_OF", "to": "n14", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n38", "id": "e108", "label": "SUBCLASS_OF", "to": "n14", "width": 1}, {"associationSlot": "0.713824394411937", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n39", "id": "e0", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n40", "width": 1}, {"associationSlot": "0.70906728841653", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n39", "id": "e1", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n43", "width": 1}, {"associationSlot": "0.73027560089881", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n39", "id": "e2", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n47", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e43", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n284", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e44", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n280", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e53", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:157631", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n153", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e63", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n27", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e64", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n34", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e70", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n251", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e71", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n259", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e154", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n101", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e155", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n98", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e156", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n94", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e157", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n90", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e158", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n128", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e159", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n138", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e160", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n129", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e161", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n103", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e162", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n107", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e163", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n124", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e164", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n111", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e165", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n114", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "id": "e166", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n134", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n39", "id": "e168", "label": "SUBCLASS_OF", "to": "n0", "width": 1}, {"associationSlot": "0.710546163926085", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n40", "id": "e3", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n43", "width": 1}, {"associationSlot": "0.705978338541878", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n40", "id": "e4", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n44", "width": 1}, {"associationSlot": "0.705136169732365", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n40", "id": "e5", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n45", "width": 1}, {"associationSlot": "0.731532895520125", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n40", "id": "e6", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n47", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n40", "id": "e45", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n280", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n40", "id": "e65", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n27", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n40", "id": "e72", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n262", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n40", "id": "e73", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n265", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n40", "id": "e74", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n255", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n40", "id": "e169", "label": "SUBCLASS_OF", "to": "n0", "width": 1}, {"associationSlot": "0.708869343839784", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n41", "id": "e7", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n46", "width": 1}, {"associationSlot": "0.706618729827547", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n41", "id": "e8", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n49", "width": 1}, {"associationSlot": "0.707371659156687", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n41", "id": "e9", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n55", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e42", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n280", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e62", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n27", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e69", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n251", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e144", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n94", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e145", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n98", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e146", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n90", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e147", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n101", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e148", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n103", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e149", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n107", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e150", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n109", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e151", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n111", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e152", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n114", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n41", "id": "e153", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n121", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n41", "id": "e170", "label": "SUBCLASS_OF", "to": "n1", "width": 1}, {"associationSlot": "0.700913456826934", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n42", "id": "e10", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n53", "width": 1}, {"associationSlot": "0.708696326304135", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n42", "id": "e11", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n54", "width": 1}, {"associationSlot": "0.709506850519645", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n42", "id": "e12", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n58", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n42", "id": "e46", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n281", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n42", "id": "e47", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n279", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n42", "id": "e48", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n283", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n42", "id": "e66", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n29", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n42", "id": "e75", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n255", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n42", "id": "e167", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n134", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n42", "id": "e171", "label": "SUBCLASS_OF", "to": "n2", "width": 1}, {"associationSlot": "0.719177472814864", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n43", "id": "e13", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n45", "width": 1}, {"associationSlot": "0.725566120141975", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n43", "id": "e14", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n47", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n43", "id": "e139", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n72", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n43", "id": "e172", "label": "SUBCLASS_OF", "to": "n3", "width": 1}, {"associationSlot": "0.728863085938593", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n44", "id": "e15", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n45", "width": 1}, {"associationSlot": "0.714696679975495", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n44", "id": "e16", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n47", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n44", "id": "e173", "label": "SUBCLASS_OF", "to": "n4", "width": 1}, {"associationSlot": "0.723981148576923", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n45", "id": "e17", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n47", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n45", "id": "e58", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:158092", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n157", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n45", "id": "e174", "label": "SUBCLASS_OF", "to": "n5", "width": 1}, {"associationSlot": "0.71340684361272", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n46", "id": "e18", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n49", "width": 1}, {"associationSlot": "0.711514340723044", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n46", "id": "e19", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n54", "width": 1}, {"associationSlot": "0.727555182517596", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n46", "id": "e20", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n55", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n46", "id": "e60", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:159477", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n154", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n46", "id": "e78", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n255", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n46", "id": "e175", "label": "SUBCLASS_OF", "to": "n6", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n47", "id": "e134", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n82", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n47", "id": "e176", "label": "SUBCLASS_OF", "to": "n7", "width": 1}, {"associationSlot": "0.714823060212988", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n48", "id": "e21", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n54", "width": 1}, {"associationSlot": "0.711717557280063", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n48", "id": "e22", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n55", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n48", "id": "e177", "label": "SUBCLASS_OF", "to": "n8", "width": 1}, {"associationSlot": "0.70655942262744", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n49", "id": "e23", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n55", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "id": "e51", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n281", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "id": "e52", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n279", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n49", "id": "e178", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"associationSlot": "0.716895209779724", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n50", "id": "e24", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n51", "width": 1}, {"associationSlot": "0.711466759665129", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n50", "id": "e25", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n52", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n50", "id": "e179", "label": "SUBCLASS_OF", "to": "n10", "width": 1}, {"associationSlot": "0.712904363245591", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n51", "id": "e26", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n52", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n51", "id": "e180", "label": "SUBCLASS_OF", "to": "n10", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n52", "id": "e181", "label": "SUBCLASS_OF", "to": "n11", "width": 1}, {"associationSlot": "0.728553104572423", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n53", "id": "e27", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n54", "width": 1}, {"associationSlot": "0.718778450245937", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n53", "id": "e28", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n55", "width": 1}, {"associationSlot": "0.715808206831859", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n53", "id": "e29", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n56", "width": 1}, {"associationSlot": "0.718403403666568", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n53", "id": "e30", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n57", "width": 1}, {"associationSlot": "0.734581106512693", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n53", "id": "e31", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n58", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e39", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n283", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e40", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n281", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e61", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n29", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e68", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n251", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e140", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n135", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e141", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n88", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e142", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n84", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n53", "id": "e143", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n117", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n53", "id": "e182", "label": "SUBCLASS_OF", "to": "n12", "width": 1}, {"associationSlot": "0.739394684528724", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n54", "id": "e32", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n55", "width": 1}, {"associationSlot": "0.716685278080891", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n54", "id": "e33", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n56", "width": 1}, {"associationSlot": "0.712130016373075", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n54", "id": "e34", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n57", "width": 1}, {"associationSlot": "0.724225283759188", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n54", "id": "e35", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n58", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n54", "id": "e50", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n281", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n54", "id": "e59", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:8089", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n150", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n54", "id": "e67", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n27", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n54", "id": "e77", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n235", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n54", "id": "e183", "label": "SUBCLASS_OF", "to": "n12", "width": 1}, {"associationSlot": "0.71673847963869", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n55", "id": "e36", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n56", "width": 1}, {"associationSlot": "0.718097898117262", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n55", "id": "e37", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n58", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e49", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n281", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e55", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:8062", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n145", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e56", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:8063", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n150", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e57", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:158828", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n144", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e76", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n262", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e136", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n76", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e137", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n81", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n55", "id": "e138", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n74", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n55", "id": "e184", "label": "SUBCLASS_OF", "to": "n12", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n56", "id": "e185", "label": "SUBCLASS_OF", "to": "n12", "width": 1}, {"associationSlot": "0.71300454760725", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n57", "id": "e38", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n58", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n57", "id": "e41", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n281", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n57", "id": "e186", "label": "SUBCLASS_OF", "to": "n12", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n58", "id": "e54", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:130456", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n213", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n58", "id": "e135", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n65", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n58", "id": "e187", "label": "SUBCLASS_OF", "to": "n13", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n221", "id": "e97", "label": "SUBCLASS_OF", "to": "n35", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n222", "id": "e98", "label": "SUBCLASS_OF", "to": "n36", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n223", "id": "e99", "label": "SUBCLASS_OF", "to": "n37", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n224", "id": "e100", "label": "SUBCLASS_OF", "to": "n37", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n225", "id": "e101", "label": "SUBCLASS_OF", "to": "n37", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n226", "id": "e102", "label": "SUBCLASS_OF", "to": "n38", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n227", "id": "e103", "label": "SUBCLASS_OF", "to": "n38", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n228", "id": "e104", "label": "SUBCLASS_OF", "to": "n38", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n268", "id": "e202", "label": "SUBCLASS_OF", "to": "n15", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n269", "id": "e203", "label": "SUBCLASS_OF", "to": "n16", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n270", "id": "e204", "label": "SUBCLASS_OF", "to": "n17", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n271", "id": "e205", "label": "SUBCLASS_OF", "to": "n18", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n272", "id": "e206", "label": "SUBCLASS_OF", "to": "n19", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n273", "id": "e207", "label": "SUBCLASS_OF", "to": "n20", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n274", "id": "e208", "label": "SUBCLASS_OF", "to": "n21", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n275", "id": "e209", "label": "SUBCLASS_OF", "to": "n22", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n276", "id": "e210", "label": "SUBCLASS_OF", "to": "n23", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n277", "id": "e211", "label": "SUBCLASS_OF", "to": "n24", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n278", "id": "e212", "label": "SUBCLASS_OF", "to": "n25", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n278", "id": "e213", "label": "SUBCLASS_OF", "to": "n26", "width": 1}]);

                  nodeColors = {};
                  allNodes = nodes.get({ returnType: "Object" });
                  for (nodeId in allNodes) {
                    nodeColors[nodeId] = allNodes[nodeId].color;
                  }
                  allEdges = edges.get({ returnType: "Object" });
                  // adding nodes and edges to the graph
                  data = {nodes: nodes, edges: edges};

                  var options = {
    "configure": {
        "enabled": false
    },
    "edges": {
        "color": {
            "inherit": true
        },
        "smooth": {
            "enabled": true,
            "type": "dynamic"
        }
    },
    "interaction": {
        "dragNodes": true,
        "hideEdgesOnDrag": false,
        "hideNodesOnDrag": false
    },
    "physics": {
        "enabled": true,
        "forceAtlas2Based": {
            "avoidOverlap": 0,
            "centralGravity": 0.005,
            "damping": 0.4,
            "gravitationalConstant": -50,
            "springConstant": 0.08,
            "springLength": 100
        },
        "solver": "forceAtlas2Based",
        "stabilization": {
            "enabled": true,
            "fit": true,
            "iterations": 1000,
            "onlyDynamicEdges": false,
            "updateInterval": 50
        }
    }
};

                  


                  

                  network = new vis.Network(container, data, options);

                  

                  

                  


                  
                      network.on("stabilizationProgress", function(params) {
                          document.getElementById('loadingBar').removeAttribute("style");
                          var maxWidth = 496;
                          var minWidth = 20;
                          var widthFactor = params.iterations/params.total;
                          var width = Math.max(minWidth,maxWidth * widthFactor);
                          document.getElementById('bar').style.width = width + 'px';
                          document.getElementById('text').innerHTML = Math.round(widthFactor*100) + '%';
                      });
                      network.once("stabilizationIterationsDone", function() {
                          document.getElementById('text').innerHTML = '100%';
                          document.getElementById('bar').style.width = '496px';
                          document.getElementById('loadingBar').style.opacity = 0;
                          // really clean the dom element
                          setTimeout(function () {document.getElementById('loadingBar').style.display = 'none';}, 500);
                      });
                  

                  return network;

              }
              drawGraph();
        </script>
    </body>
</html>