<html>
    <head>
        <meta charset="utf-8">
        
            <script src="lib/bindings/utils.js"></script>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/dist/vis-network.min.css" integrity="sha512-WgxfT5LWjfszlPHXRmBWHkV2eceiWTOBvrKCNbdgDYTHrT2AeLCGbF4sZlZw3UMN3WtL0tGUoIAKsu8mllg/XA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
            <script src="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/vis-network.min.js" integrity="sha512-LnvoEWDFrqGHlHmDD2101OrLcbsfkrzoSpvtSQtxK3RMnRV0eOkhhBN2dXHKRrUU8p2DGRTk35n4O8nWSVe1mQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
            
        
<center>
<h1></h1>
</center>

<!-- <link rel="stylesheet" href="../node_modules/vis/dist/vis.min.css" type="text/css" />
<script type="text/javascript" src="../node_modules/vis/dist/vis.js"> </script>-->
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
          crossorigin="anonymous"
        />
        <script
          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js"
          integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf"
          crossorigin="anonymous"
        ></script>


        <center>
          <h1></h1>
        </center>
        <style type="text/css">

             #mynetwork {
                 width: 100%;
                 height: 600px;
                 background-color: white;
                 border: 1px solid lightgray;
                 position: relative;
                 float: left;
             }

             

             

             
        </style>
    </head>


    <body>
        <div class="card" style="width: 100%">
            
            
            <div id="mynetwork" class="card-body"></div>
        </div>

        
        

        <script type="text/javascript">

              // initialize global variables.
              var edges;
              var nodes;
              var allNodes;
              var allEdges;
              var nodeColors;
              var originalNodes;
              var network;
              var container;
              var options, data;
              var filter = {
                  item : '',
                  property : '',
                  value : []
              };

              

              

              // This method is responsible for drawing the graph, returns the drawn network
              function drawGraph() {
                  var container = document.getElementById('mynetwork');

                  

                  // parsing and collecting nodes and edges from the python
                  nodes = new vis.DataSet([{"betweenness_centrality": "213.08333333333337", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1889763779527559", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n0", "label": "Roseibium", "labels": ":Genus", "name": "Roseibium", "pagerank_centrality": "0.8297147817532112", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 17, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11155991\u0027, \u0027PMID:17473266\u0027]"}, {"betweenness_centrality": "158.41666666666669", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1651376146788991", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n250", "label": "Stappiaceae", "labels": ":Family", "name": "Stappiaceae", "pagerank_centrality": "0.8706375026380444", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 15, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "415.2499999999999", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.19618528610354224", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n1", "label": "Sorangium", "labels": ":Genus", "name": "Sorangium", "pagerank_centrality": "2.2525978827162136", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 24, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:17473228\u0027, \u0027PMID:18984709\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.1643835616438356", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The process in which a multicellular organism, a unicellular organism or a group of unicellular organisms grow in a threadlike, filamentous shape. Excessive growth of organisms with a filamentous morphology has long been associated with the operational problem of bulking. In this condition, interfloc bridging by the filaments impedes adequate settling of the biomass, resulting in loss of solids with the treated effluent", "font": {"size": 12}, "id": "n272", "iri": "http://purl.obolibrary.org/obo/GO_0030447 ", "label": "Filamentous growth", "labels": ":BiologicalProcess", "name": "Filamentous growth", "pagerank_centrality": "0.46655823397861546", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.1643835616438356", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "A substance within a foam formulation that is used to maintain or improve its stability. Organisms with a hydrophobic surface have been associated with the stabilization of foams in both activated sludge and anaerobic digester systems. Excessive build-up of surface foams can result in carryover of solids from clarifiers in wastewater treatment systems and several operational problems in anaerobic digesters including biofouling of gas collection pipes and pumps.", "font": {"size": 12}, "id": "n273", "iri": "http://purl.obolibrary.org/obo/NCIT_C176634 ", "label": "Foam Stabilization", "labels": ":BiologicalProcess", "name": "Foam Stabilization", "pagerank_centrality": "0.46655823397861546", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "foam stabilizing agent", "xref": ""}, {"betweenness_centrality": "180.33333333333337", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.16941176470588235", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n251", "label": "Polyangiaceae", "labels": ":Family", "name": "Polyangiaceae", "pagerank_centrality": "0.8320826804355006", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 16, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "48.08333333333333", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.17266187050359713", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "4.0", "description": "The chemical reactions and pathways involving carbohydrates, any of a group of organic compounds based of the general formula Cx(H2O)y. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n270", "iri": "http://purl.obolibrary.org/obo/GO_0005975 ", "label": "Carbohydrate metabolic process", "labels": ":BiologicalProcess", "name": "Carbohydrate metabolic process", "pagerank_centrality": "0.7889012774120125", "providedBy": "MiDAS", "shape": "dot", "size": 11, "synonym": "carbohydrate metabolism", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.18798955613577023", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "4.0", "description": "The anaerobic enzymatic conversion of organic compounds, especially carbohydrates, coupling the oxidation and reduction of NAD/H and the generation of adenosine triphosphate (ATP). During fermentation, organic compounds act as both the electron acceptor and donor. Organic molecules, such as sugars and amino acids, are utilized as the energy and carbon source. In the absence of, or the inability to utilize, external electron acceptors, organic molecules act as the electron acceptor to balance the redox of the cell. Common by-products include volatile fatty acids, alcohols, hydrogen and CO2.", "font": {"size": 12}, "id": "n64", "iri": "http://purl.obolibrary.org/obo/GO_0006113", "label": "Fermentation", "labels": ":BiologicalProcess", "name": "Fermentation", "pagerank_centrality": "0.770658717576448", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": "MetaCyc:FERMENTATION-PWY|Wikipedia:Fermentation_(biochemistry)"}, {"betweenness_centrality": "402.25", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.17955112219451372", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "14.0", "font": {"size": 12}, "id": "n2", "label": "Pseudomonas", "labels": ":Genus", "name": "Pseudomonas", "pagerank_centrality": "2.6777569956362544", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 23, "synonym": "[\u0027\"\\\"Chlorobacterium\\\" Guillebeau 1890, nom. rejic. Opin. 6 (not \\\"Chlorobacterium\\\" Lauterborn 1916)\" RELATED synonym []\u0027, \u0027\"Liquidomonas\" RELATED synonym []\u0027, \u0027\"Loefflerella\" RELATED synonym []\u0027, \u0027\"RNA similarity group I\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10758879\u0027, \u0027PMID:10939664\u0027, \u0027PMID:15950132\u0027, \u0027PMID:18048745\u0027, \u0027PMID:23918787\u0027, \u0027PMID:7727274\u0027, \u0027PMID:9103607\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.15254237288135594", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "A trophic process during which a living entity acquires food and energy by consuming another organism or organism part, living or dead.", "font": {"size": 12}, "id": "n63", "iri": "http://purl.obolibrary.org/obo/ECOCORE_00000011", "label": "Heterotrophy", "labels": ":BiologicalProcess", "name": "Heterotrophy", "pagerank_centrality": "0.47234304343339706", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.15254237288135594", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The chemical reactions and pathways involving a short-chain fatty acid. A short-chain fatty acid has an aliphatic tail containing fewer than 6 carbons. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n269", "iri": "http://purl.obolibrary.org/obo/GO_0046459 ", "label": "Short-chain fatty acid metabolic process", "labels": ":BiologicalProcess", "name": "Short-chain fatty acid metabolic process", "pagerank_centrality": "0.47234304343339706", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "short-chain fatty acid metabolism", "xref": ""}, {"betweenness_centrality": "146.08333333333331", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.15824175824175823", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n252", "label": "Pseudomonadaceae", "labels": ":Family", "name": "Pseudomonadaceae", "pagerank_centrality": "0.8344346434112763", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 15, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.15254237288135594", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The chemical reactions and pathways involving amino acids, carboxylic acids containing one or more amino groups. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n271", "iri": "http://purl.obolibrary.org/obo/GO_0006520 ", "label": "Amino acid metabolic process", "labels": ":BiologicalProcess", "name": "Amino acid metabolic process", "pagerank_centrality": "0.47234304343339706", "providedBy": "MiDAS", "shape": "dot", "size": 10, "synonym": "cellular amino acid metabolic process|cellular amino acid metabolism|amino acid and derivative metabolism|cellular amino acid and derivative metabolic process", "xref": ""}, {"betweenness_centrality": "113.1666666666667", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.192", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "4.0", "description": "The reduction of nitrate to dinitrogen by four reactions; each intermediate is transformed to the next lower oxidation state; also part of cellular bioenergetics; the nitrogen compounds can serve as terminal acceptors for electron transport phosphorylation in place of oxygen. The denitrifiers are usually facultative-anaerobic heterotrophic organisms utilizing nitrate and/or nitrite as electron acceptors in the absence of oxygen. Denitrification is the final step of nitrogen removal from wastewater and generally occurs under anaerobic conditions. Nitrate is available from the oxidation of ammonia by the aerobic activities of the nitrifiers (AOB and NOB). Denitrification involves the sequential reduction of nitrate to inert dinitrogen gas, via several intermediates (nitrite, nitric oxide (gas) and nitrous oxide (gas)). Not all organisms involved in denitrification possess all the genes for reduction of nitrate all the way to dinitrogen gas. As such, for MiDAS profiles, the nitrite reducing organisms are selected to represent the denitrifiers; given denitrification is defined as \u201cthe dissimilatory transformation of nitrate or nitrite to a gas species concomitant with energy conservation\u201d and not all organisms capable of nitrate reduction can reduce nitrite.", "font": {"size": 12}, "id": "n47", "iri": "http://purl.obolibrary.org/obo/GO_0019333", "label": "Denitrification pathway", "labels": ":BiologicalProcess", "name": "Denitrification pathway", "pagerank_centrality": "0.784618085581976", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 13, "synonym": "", "xref": ""}, {"betweenness_centrality": "1127.0", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2028169014084507", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n3", "label": "Adineta", "labels": ":Genus", "name": "Adineta", "pagerank_centrality": "0.8265518651544208", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 49, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "1100.0", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.18848167539267016", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n253", "label": "Adinetidae", "labels": ":Family", "name": "Adinetidae", "pagerank_centrality": "0.8582862015654728", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 48, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "259.1666666666668", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.17307692307692307", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n4", "label": "Mesomycoplasma", "labels": ":Genus", "name": "Mesomycoplasma", "pagerank_centrality": "0.7406355748967146", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 18, "synonym": "Unknown", "xref": "[\u0027GC_ID:4\u0027, \u0027PMID:31385780\u0027, \u0027PMID:31971499\u0027, \u0027PMID:35947640\u0027]"}, {"betweenness_centrality": "207.83333333333346", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.15384615384615385", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n254", "label": "Metamycoplasmataceae", "labels": ":Family", "name": "Metamycoplasmataceae", "pagerank_centrality": "0.9664309404547871", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 17, "synonym": "Unknown", "xref": "[\u0027GC_ID:4\u0027, \u0027PMID:31385780\u0027, \u0027PMID:31971499\u0027, \u0027PMID:35947640\u0027]"}, {"betweenness_centrality": "138.48571428571412", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1054172767203514", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n5", "label": "Jakoba", "labels": ":Genus", "name": "Jakoba", "pagerank_centrality": "0.6207741028396364", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 14, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "196.19999999999982", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.11501597444089456", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n255", "label": "Jakobidae", "labels": ":Family", "name": "Jakobidae", "pagerank_centrality": "0.9380853935282562", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 16, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "39.04285714285717", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.09716599190283401", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n6", "label": "Vahlkampfia", "labels": ":Genus", "name": "Vahlkampfia", "pagerank_centrality": "0.681525663848372", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "93.90000000000013", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.10495626822157435", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n256", "label": "Vahlkampfiidae", "labels": ":Family", "name": "Vahlkampfiidae", "pagerank_centrality": "0.898241076619345", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 13, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "39.04285714285716", "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.09716599190283401", "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n7", "label": "Plasmopara", "labels": ":Genus", "name": "Plasmopara", "pagerank_centrality": "0.681525663848372", "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "93.90000000000015", "border-color": "#31c17aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.10495626822157435", "color": {"background": "#2793146b", "border": "#31c17aff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n257", "label": "Peronosporaceae", "labels": ":Family", "name": "Peronosporaceae", "pagerank_centrality": "0.898241076619345", "propertyValue": "[\u0027has_rank NCBITaxon:family\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"downy mildews\" RELATED blast_name []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "109.08333333333331", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.14845360824742268", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n10", "label": "Hyphomicrobiales", "labels": ":Order", "name": "Hyphomicrobiales", "pagerank_centrality": "0.8764482903108171", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"alpha-2 proteobacteria\" RELATED in_part []\u0027, \u0027\"rhizobacteria\" RELATED synonym []\u0027, \u0027\"Rhizobiaceae group\" RELATED synonym []\u0027, \u0027\"Rhizobiales\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:1854635\u0027]"}, {"betweenness_centrality": "68.41666666666666", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.13688212927756654", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n203", "label": "Alphaproteobacteria", "labels": ":Class", "name": "Alphaproteobacteria", "pagerank_centrality": "0.8502919902747812", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"a-proteobacteria\" RELATED blast_name []\u0027, \u0027\"alpha proteobacteria\" RELATED synonym []\u0027, \u0027\"alpha subdivision\" RELATED synonym []\u0027, \u0027\"alpha subgroup\" RELATED synonym []\u0027, \u0027\"Alphabacteria\" RELATED synonym []\u0027, \u0027\"Proteobacteria alpha subdivision\" RELATED synonym []\u0027, \u0027\"Purple bacteria, alpha subdivision\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11541974\u0027, \u0027PMID:11837318\u0027, \u0027PMID:16166687\u0027, \u0027PMID:16403855\u0027, \u0027PMID:19060069\u0027]"}, {"betweenness_centrality": "128.0", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1509433962264151", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n11", "label": "Polyangiales", "labels": ":Order", "name": "Polyangiales", "pagerank_centrality": "0.8660779193490685", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 14, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "80.3333333333333", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.13714285714285715", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n204", "label": "Deltaproteobacteria", "labels": ":Class", "name": "Deltaproteobacteria", "pagerank_centrality": "0.8631110957675676", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"\\\"Deltabacteria\\\" Cavalier-Smith 1992\" RELATED synonym []\u0027, \u0027\"d-proteobacteria\" RELATED blast_name []\u0027, \u0027\"delta proteobacteria\" RELATED synonym []\u0027, \u0027\"delta subdivision\" RELATED synonym []\u0027, \u0027\"delta subgroup\" RELATED synonym []\u0027, \u0027\"Deltabacteria Cavalier-Smith 2002\" RELATED synonym []\u0027, \u0027\"Proteobacteria delta subdivision\" RELATED synonym []\u0027, \u0027\"Purple bacteria, delta subdivision\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11837318\u0027, \u0027PMID:16403855\u0027, \u0027PMID:27453056\u0027, \u0027PMID:28771119\u0027, \u0027PMID:33151140\u0027]"}, {"betweenness_centrality": "97.08333333333333", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.14285714285714285", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n12", "label": "Pseudomonadales", "labels": ":Order", "name": "Pseudomonadales", "pagerank_centrality": "0.8584130137789036", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 13, "synonym": "[\u0027\"gamma-3 proteobacteria\" RELATED in_part []\u0027, \u0027\"Pseudomonaceae/Moraxellaceae group\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "52.083333333333336", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.13114754098360656", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n205", "label": "Gammaproteobacteria", "labels": ":Class", "name": "Gammaproteobacteria", "pagerank_centrality": "0.8421962668694584", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 11, "synonym": "[\u0027\"g-proteobacteria\" RELATED blast_name []\u0027, \u0027\"gamma proteobacteria\" RELATED synonym []\u0027, \u0027\"gamma subdivision\" RELATED synonym []\u0027, \u0027\"gamma subgroup\" RELATED synonym []\u0027, \u0027\"Proteobacteria gamma subdivision\" RELATED synonym []\u0027, \u0027\"Pseudomonadia\" RELATED synonym []\u0027, \u0027\"Purple bacteria, gamma subdivision\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:23334881\u0027]"}, {"betweenness_centrality": "1071.0", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.17518248175182483", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n13", "label": "Adinetida", "labels": ":Order", "name": "Adinetida", "pagerank_centrality": "0.8509359087276794", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 47, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "1040.0", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.16289592760180996", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n206", "label": "Eurotatoria", "labels": ":Class", "name": "Eurotatoria", "pagerank_centrality": "0.8000078891479239", "propertyValue": "[\u0027has_rank NCBITaxon:class\u0027]", "shape": "dot", "size": 46, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "50.58333333333327", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.13926499032882012", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n14", "label": "Mycoplasmoidales", "labels": ":Order", "name": "Mycoplasmoidales", "pagerank_centrality": "1.1963105967579375", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:4\u0027, \u0027PMID:31385780\u0027, \u0027PMID:31971499\u0027, \u0027PMID:35947640\u0027]"}, {"betweenness_centrality": "71.00000000000003", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.12880143112701253", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n29", "label": "Mycoplasmatota", "labels": ":Phylum", "name": "Mycoplasmatota", "pagerank_centrality": "0.9630451092444952", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"\\\"Mollicutaeota\\\" Oren et al. 2015\" RELATED synonym []\u0027, \u0027\"\\\"Tenericutes\\\" Murray 1984\" RELATED synonym []\u0027, \u0027\"Mollicutota\" RELATED synonym []\u0027, \u0027\"Tenericutes\" EXACT equivalent_name []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "50.58333333333327", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.13926499032882012", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n207", "label": "Mycoplasmoidales", "labels": ":Class", "name": "Mycoplasmoidales", "pagerank_centrality": "1.1963105967579375", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:4\u0027, \u0027PMID:31385780\u0027, \u0027PMID:31971499\u0027, \u0027PMID:35947640\u0027]"}, {"betweenness_centrality": "81.6380952380953", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1265377855887522", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n15", "label": "Jakobida", "labels": ":Order", "name": "Jakobida", "pagerank_centrality": "1.248769099346407", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "81.6380952380953", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1265377855887522", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n30", "label": "Jakobida", "labels": ":Phylum", "name": "Jakobida", "pagerank_centrality": "1.248769099346407", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "81.6380952380953", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1265377855887522", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n208", "label": "Jakobida", "labels": ":Class", "name": "Jakobida", "pagerank_centrality": "1.248769099346407", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "1030.9142857142856", "border-color": "#cf8f44ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1411764705882353", "color": {"background": "#e9ccaaff", "border": "#cf8f44ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n9", "label": "Eukaryota", "labels": ":Kingdom", "name": "Eukaryota", "pagerank_centrality": "1.4221726772967183", "propertyValue": "[\u0027has_rank NCBITaxon:superkingdom\u0027]", "shape": "dot", "size": 45, "synonym": "[\u0027\"Eucarya\" RELATED synonym []\u0027, \u0027\"Eucaryotae\" RELATED synonym []\u0027, \u0027\"eucaryotes\" EXACT genbank_common_name []\u0027, \u0027\"Eukarya\" RELATED synonym []\u0027, \u0027\"Eukaryotae\" RELATED synonym []\u0027, \u0027\"eukaryotes\" EXACT OMO:0003003 []\u0027, \u0027\"eukaryotes\" RELATED blast_name []\u0027]", "xref": "[\u0027GC_ID:1\u0027, \u0027PMID:23020233\u0027, \u0027PMID:30257078\u0027]"}, {"betweenness_centrality": "73.12857142857145", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.11446740858505565", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n16", "label": "Schizopyrenida", "labels": ":Order", "name": "Schizopyrenida", "pagerank_centrality": "1.0909602265276337", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "269.3571428571425", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.12631578947368421", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n31", "label": "Heterolobosea", "labels": ":Phylum", "name": "Heterolobosea", "pagerank_centrality": "0.8101238042809776", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 19, "synonym": "[\u0027\"Lobosa\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "73.12857142857145", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.11446740858505565", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n209", "label": "Schizopyrenida", "labels": ":Class", "name": "Schizopyrenida", "pagerank_centrality": "1.0909602265276337", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "73.12857142857145", "border-color": "#9b6bd9ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.11446740858505565", "color": {"background": "#c5aae9ff", "border": "#9b6bd9ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n17", "label": "Peronosporales", "labels": ":Order", "name": "Peronosporales", "pagerank_centrality": "1.0909602265276337", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "269.35714285714255", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.12631578947368421", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n32", "label": "Oomycota", "labels": ":Phylum", "name": "Oomycota", "pagerank_centrality": "0.8101238042809776", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 19, "synonym": "[\u0027\"Oomycetes\" RELATED synonym []\u0027, \u0027\"oomycetes\" RELATED blast_name []\u0027, \u0027\"Peronosporomycetes\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "73.12857142857145", "border-color": "#888408ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.11446740858505565", "color": {"background": "#d6d211ff", "border": "#888408ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n210", "label": "Peronosporales", "labels": ":Class", "name": "Peronosporales", "pagerank_centrality": "1.0909602265276337", "propertyValue": "[\u0027has_rank NCBITaxon:order\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "82.0833333333333", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.12972972972972974", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n26", "label": "Pseudomonadota", "labels": ":Phylum", "name": "Pseudomonadota", "pagerank_centrality": "1.1738050321477085", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"\\\"Alphaproteobacteriota\\\" Whitman et al. 2018\" RELATED synonym []\u0027, \u0027\"Alphaproteobacteriota\" RELATED synonym []\u0027, \u0027\"Proteobacteria\" RELATED synonym []\u0027, \u0027\"proteobacteria\" RELATED blast_name []\u0027, \u0027\"Proteobacteriota\" RELATED synonym []\u0027, \u0027\"purple photosynthetic bacteria and relatives\" EXACT OMO:0003003 []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11321122\u0027, \u0027PMID:11837318\u0027]"}, {"betweenness_centrality": "79.41666666666669", "border-color": "#cf8f44ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.12286689419795221", "color": {"background": "#e9ccaaff", "border": "#cf8f44ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n8", "label": "Bacteria", "labels": ":Kingdom", "name": "Bacteria", "pagerank_centrality": "1.0983023543893555", "propertyValue": "[\u0027has_rank NCBITaxon:superkingdom\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"bacteria\" RELATED blast_name []\u0027, \u0027\"eubacteria\" EXACT genbank_common_name []\u0027, \u0027\"Monera\" RELATED in_part []\u0027, \u0027\"Procaryotae\" RELATED in_part []\u0027, \u0027\"Prokaryota\" RELATED in_part []\u0027, \u0027\"Prokaryotae\" RELATED in_part []\u0027, \u0027\"prokaryote\" RELATED in_part []\u0027, \u0027\"prokaryotes\" RELATED in_part []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10425795\u0027, \u0027PMID:10425796\u0027, \u0027PMID:10425797\u0027, \u0027PMID:10490293\u0027, \u0027PMID:10843050\u0027, \u0027PMID:10939651\u0027, \u0027PMID:10939673\u0027, \u0027PMID:10939677\u0027, \u0027PMID:11211268\u0027, \u0027PMID:11321083\u0027, \u0027PMID:11321113\u0027, \u0027PMID:11411719\u0027, \u0027PMID:11540071\u0027, \u0027PMID:11542017\u0027, \u0027PMID:11542087\u0027, \u0027PMID:11760965\u0027, \u0027PMID:12054223\u0027, \u0027PMID:2112744\u0027, \u0027PMID:270744\u0027, \u0027PMID:32628106\u0027, \u0027PMID:36748408\u0027, \u0027PMID:7520741\u0027, \u0027PMID:8123559\u0027, \u0027PMID:8186100\u0027, \u0027PMID:8590690\u0027, \u0027PMID:9103655\u0027, \u0027PMID:9336922\u0027]"}, {"betweenness_centrality": "40.50000000000001", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.1265377855887522", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n27", "label": "Myxococcota", "labels": ":Phylum", "name": "Myxococcota", "pagerank_centrality": "0.8236488083153901", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": "1007.0", "border-color": "#47b9d1ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.15157894736842106", "color": {"background": "#b5e3ecff", "border": "#47b9d1ff"}, "degree_centrality": "4.0", "font": {"size": 12}, "id": "n28", "label": "Rotifera", "labels": ":Phylum", "name": "Rotifera", "pagerank_centrality": "0.6880048250611409", "propertyValue": "[\u0027has_rank NCBITaxon:phylum\u0027]", "shape": "dot", "size": 44, "synonym": "[\u0027\"rotifers\" EXACT genbank_common_name []\u0027, \u0027\"rotifers\" RELATED blast_name []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "1059.2500000000002", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2236024844720497", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "26.0", "font": {"size": 12}, "id": "n33", "label": "Roseibium marinum", "labels": ":Species", "name": "Roseibium marinum", "pagerank_centrality": "4.816392463383248", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 46, "synonym": "[\u0027\"Labrenzia marina\" RELATED synonym []\u0027, \u0027\"Stappia marina\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16403869\u0027, \u0027PMID:17473266\u0027]"}, {"betweenness_centrality": "978.4999999999997", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.22429906542056074", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "22.0", "font": {"size": 12}, "id": "n34", "label": "Sorangium cellulosum", "labels": ":Species", "name": "Sorangium cellulosum", "pagerank_centrality": "3.967244865391325", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 43, "synonym": "[\u0027\"Polyangium cellulosum\" RELATED synonym []\u0027, \u0027\"Polyangium compositum\" RELATED synonym []\u0027, \u0027\"Sorangium compositum\" RELATED synonym []\u0027, \u0027\"Sorangium nigrescens\" RELATED synonym []\u0027, \u0027\"Sorangium nigrum\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:17473228\u0027]"}, {"betweenness_centrality": "306.9166666666666", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.20055710306406685", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "8.0", "font": {"size": 12}, "id": "n35", "label": "Pseudomonas moraviensis", "labels": ":Species", "name": "Pseudomonas moraviensis", "pagerank_centrality": "1.4788106181405392", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 20, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:17082407\u0027]"}, {"betweenness_centrality": "1152.0", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.21818181818181817", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n36", "label": "Adineta vaga", "labels": ":Species", "name": "Adineta vaga", "pagerank_centrality": "1.115208329980124", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 50, "synonym": "[\u0027\"Callidina vaga\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "502.4999999999998", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.2", "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "12.0", "font": {"size": 12}, "id": "n37", "label": "Mesomycoplasma hyorhinis", "labels": ":Species", "name": "Mesomycoplasma hyorhinis", "pagerank_centrality": "2.2670681074521255", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 27, "synonym": "[\u0027\"Asterococcus hyorhinis\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:4\u0027, \u0027PMID:1374621\u0027, \u0027PMID:29556819\u0027, \u0027PMID:31385780\u0027, \u0027PMID:31971499\u0027, \u0027PMID:33888501\u0027]"}, {"betweenness_centrality": "0.0", "border-color": "#256491c9", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ActivityAndBehavior", "closeness_centrality": "0.183206106870229", "color": {"background": "#77b3ddc9", "border": "#256491c9"}, "degree_centrality": "2.0", "description": "an aerobe that requires oxygen to grow", "font": {"size": 12}, "id": "n262", "iri": "", "label": "obligate aerobe", "labels": ":ActivityAndBehavior", "name": "obligate aerobe", "pagerank_centrality": "0.462275042148579", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.183206106870229", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n130", "iri": "", "label": "BACTO MARINE BROTH (DIFCO 2216) (DSMZ Medium 514)", "labels": ":ChemicalEntity", "name": "BACTO MARINE BROTH (DIFCO 2216) (DSMZ Medium 514)", "pagerank_centrality": "0.462275042148579", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#e75d24ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "PhenotypicFeature", "closeness_centrality": "0.183206106870229", "color": {"background": "#f3ab8eff", "border": "#e75d24ff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n18", "iri": "", "label": "bacillus", "labels": ":PhenotypicFeature", "name": "bacillus", "pagerank_centrality": "0.462275042148579", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.183206106870229", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "2.0", "description": "Water which has physicochemical properties that have been determined by the processes occuring in a sea or ocean.", "font": {"size": 12}, "id": "n225", "iri": "http://purl.obolibrary.org/obo/ENVO_00002149", "label": "sea water", "labels": ":EnvironmentalFeature", "name": "sea water", "pagerank_centrality": "0.462275042148579", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "ocean water|seawater", "xref": "EcoLexicon:sea_water|LTER:701|SWEETRealm:SeaWater|https://en.wikipedia.org/wiki/Sea_water"}, {"betweenness_centrality": "0.0", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.183206106870229", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "2.0", "description": "Sediment which has been transported through the marine water column, settling on the seafloor.", "font": {"size": 12}, "id": "n235", "iri": "http://purl.obolibrary.org/obo/ENVO_03000033", "label": "marine sediment", "labels": ":EnvironmentalFeature", "name": "marine sediment", "pagerank_centrality": "0.462275042148579", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "marine sediments", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.183206106870229", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "A trophic process during which a living entity generates food from abiotic sources or through consumption of living or dead material, and captures energy from preformed molecules. Autotrophs are able to fix CO2 as a carbon source for growth. Important examples in wastewater treatment include nitrite and ammonia oxidizers (NOB and AOB), anaerobic ammonia oxidizers (anammox). Mixotrophs can obtain carbon by fixing CO2 and or from organic sources (heterotrophy). The criterium for this MiDAS field is the potential for CO2 fixation i.e. organisms possessing the Calvin-cycle.", "font": {"size": 12}, "id": "n51", "iri": "http://purl.obolibrary.org/obo/ECOCORE_00000163", "label": "Chemotrophy", "labels": ":BiologicalProcess", "name": "Chemotrophy", "pagerank_centrality": "0.462275042148579", "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.183206106870229", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "An aldohexose used as a source of energy and metabolic intermediate.", "font": {"size": 12}, "id": "n80", "iri": "http://purl.obolibrary.org/obo/CHEBI_17234", "label": "glucose", "labels": ":ChemicalEntity", "name": "glucose", "pagerank_centrality": "0.462275042148579", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "DL-glucose|Glc|Glucose|Glukose|gluco-hexose|glucose", "xref": "CAS:50-99-7|KEGG:C00293|Wikipedia:Glucose"}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.1836734693877551", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n129", "iri": "", "label": "VY/2 AGAR (DSMZ Medium 9)", "labels": ":ChemicalEntity", "name": "VY/2 AGAR (DSMZ Medium 9)", "pagerank_centrality": "0.45410048359783256", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.1836734693877551", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n189", "iri": "", "label": "ST21 AGAR FOR CELLULOSE DEGRADERS (DSMZ Medium 938)", "labels": ":ChemicalEntity", "name": "ST21 AGAR FOR CELLULOSE DEGRADERS (DSMZ Medium 938)", "pagerank_centrality": "0.45410048359783256", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.1836734693877551", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "2.0", "description": "An environmental material which is primarily composed of minerals, varying proportions of sand, silt, and clay, organic material such as humus, interstitial gases, liquids, and a broad range of resident micro- and macroorganisms.", "font": {"size": 12}, "id": "n215", "iri": "http://purl.obolibrary.org/obo/ENVO_00001998", "label": "soil", "labels": ":EnvironmentalFeature", "name": "soil", "pagerank_centrality": "0.45410048359783256", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "regolith", "xref": "LTER:535|SPIRE:Soil|SWEETRealm:Soil|https://en.wikipedia.org/wiki/Soil"}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.1836734693877551", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The chemical reactions and pathways resulting in the breakdown of xylan, a polymer containing a beta-1,4-linked D-xylose backbone.", "font": {"size": 12}, "id": "n41", "iri": "http://purl.obolibrary.org/obo/GO_0045493", "label": "Xylan catabolic process", "labels": ":BiologicalProcess", "name": "Xylan catabolic process", "pagerank_centrality": "0.45410048359783256", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "xylan breakdown|xylan catabolism|xylan degradation", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": "0.1836734693877551", "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": "2.0", "description": "The chemical reactions and pathways resulting in the breakdown of cellulose, a linear beta1-4 glucan of molecular mass 50-400 kDa with the pyranose units in the -4C1 conformation.", "font": {"size": 12}, "id": "n44", "iri": "http://purl.obolibrary.org/obo/GO_0030245", "label": "Cellulose catabolic process", "labels": ":BiologicalProcess", "name": "Cellulose catabolic process", "pagerank_centrality": "0.45410048359783256", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "cellulose breakdown|cellulose catabolism|cellulose degradation", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.16744186046511628", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n132", "iri": "", "label": "CASO AGAR (MERCK 105458) (DSMZ Medium 220)", "labels": ":ChemicalEntity", "name": "CASO AGAR (MERCK 105458) (DSMZ Medium 220)", "pagerank_centrality": "0.462104870471435", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": "0.16705336426914152", "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n186", "iri": "", "label": "FRIIS MEDIUM (DSMZ Medium 1078)", "labels": ":ChemicalEntity", "name": "FRIIS MEDIUM (DSMZ Medium 1078)", "pagerank_centrality": "0.4689835944880567", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#e75d24ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "PhenotypicFeature", "closeness_centrality": "0.16705336426914152", "color": {"background": "#f3ab8eff", "border": "#e75d24ff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n19", "iri": "", "label": "coccus", "labels": ":PhenotypicFeature", "name": "coccus", "pagerank_centrality": "0.4689835944880567", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "0.0", "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": "0.16705336426914152", "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": "2.0", "description": "", "font": {"size": 12}, "id": "n244", "iri": "", "label": "host_animal_endotherm", "labels": ":EnvironmentalFeature", "name": "host_animal_endotherm", "pagerank_centrality": "0.4689835944880567", "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "", "xref": ""}, {"betweenness_centrality": "85.28571428571414", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.09809264305177112", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n38", "label": "Jakoba libera", "labels": ":Species", "name": "Jakoba libera", "pagerank_centrality": "0.9696150795972398", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 12, "synonym": "[\u0027\"Cryptobia libera\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "22.042857142857144", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.09125475285171103", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n39", "label": "Vahlkampfia avara", "labels": ":Species", "name": "Vahlkampfia avara", "pagerank_centrality": "0.989703744109678", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 10, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": "22.042857142857144", "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": "0.09125475285171103", "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": "6.0", "font": {"size": 12}, "id": "n40", "label": "Plasmopara viticola", "labels": ":Species", "name": "Plasmopara viticola", "pagerank_centrality": "0.989703744109678", "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 10, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}]);
                  edges = new vis.DataSet([{"color": "#a5abb6ff", "font": {"size": 6}, "from": "n0", "id": "e68", "label": "SUBCLASS_OF", "to": "n250", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n1", "hasattribute": "Negative", "id": "e51", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n272", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n1", "hasattribute": "Positive", "id": "e53", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n273", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n1", "id": "e69", "label": "SUBCLASS_OF", "to": "n251", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n1", "hasattribute": "Positive", "id": "e106", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n270", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n1", "hasattribute": "Positive", "id": "e107", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n64", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e22", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n63", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e52", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n269", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n2", "id": "e70", "label": "SUBCLASS_OF", "to": "n252", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e105", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n270", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e108", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n271", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n2", "hasattribute": "Positive", "id": "e109", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n47", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n3", "id": "e71", "label": "SUBCLASS_OF", "to": "n253", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n4", "id": "e72", "label": "SUBCLASS_OF", "to": "n254", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n5", "id": "e73", "label": "SUBCLASS_OF", "to": "n255", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n6", "id": "e74", "label": "SUBCLASS_OF", "to": "n256", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n7", "id": "e75", "label": "SUBCLASS_OF", "to": "n257", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n10", "id": "e89", "label": "SUBCLASS_OF", "to": "n203", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n11", "id": "e90", "label": "SUBCLASS_OF", "to": "n204", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n12", "id": "e91", "label": "SUBCLASS_OF", "to": "n205", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n13", "id": "e92", "label": "SUBCLASS_OF", "to": "n206", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n14", "id": "e27", "label": "SUBCLASS_OF", "to": "n29", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n14", "id": "e94", "label": "SUBCLASS_OF", "to": "n207", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n15", "id": "e30", "label": "SUBCLASS_OF", "to": "n30", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n15", "id": "e31", "label": "SUBCLASS_OF", "to": "n208", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n15", "id": "e46", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n16", "id": "e38", "label": "SUBCLASS_OF", "to": "n31", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n16", "id": "e98", "label": "SUBCLASS_OF", "to": "n209", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n17", "id": "e40", "label": "SUBCLASS_OF", "to": "n32", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n17", "id": "e102", "label": "SUBCLASS_OF", "to": "n210", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n26", "id": "e42", "label": "SUBCLASS_OF", "to": "n8", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n27", "id": "e43", "label": "SUBCLASS_OF", "to": "n8", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n28", "id": "e44", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n29", "id": "e45", "label": "SUBCLASS_OF", "to": "n8", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n30", "id": "e34", "label": "SUBCLASS_OF", "to": "n208", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n30", "id": "e47", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n31", "id": "e49", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n32", "id": "e50", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"associationSlot": "-0.352631554074716", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n33", "id": "e0", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n34", "width": 1}, {"associationSlot": "0.440130853657087", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n33", "id": "e1", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n35", "width": 1}, {"associationSlot": "0.143274979454454", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n33", "id": "e2", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n36", "width": 1}, {"associationSlot": "-0.219739990857799", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n33", "id": "e3", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n37", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e10", "label": "CAPABLE_OF", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n262", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e12", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:13835", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n130", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e17", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n18", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e19", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n225", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e20", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n235", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e54", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n47", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e55", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n51", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "id": "e59", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n80", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n33", "id": "e60", "label": "SUBCLASS_OF", "to": "n0", "width": 1}, {"associationSlot": "0.440130853657082", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n34", "id": "e4", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n35", "width": 1}, {"associationSlot": "0.143274979454447", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n34", "id": "e5", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n36", "width": 1}, {"associationSlot": "-0.219739990857804", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n34", "id": "e6", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n37", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n34", "id": "e14", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:12219", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n129", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n34", "id": "e15", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:12431", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n189", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n34", "id": "e21", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n215", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n34", "id": "e56", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n41", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n34", "id": "e57", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n44", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n34", "id": "e58", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n64", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n34", "id": "e61", "label": "SUBCLASS_OF", "to": "n1", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n35", "id": "e13", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:13149", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n132", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n35", "id": "e62", "label": "SUBCLASS_OF", "to": "n2", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n36", "id": "e63", "label": "SUBCLASS_OF", "to": "n3", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n37", "id": "e11", "label": "HAS_NUTRIENT", "primaryKnowledgeSource": "bacdive:8608", "providedBy": "KG-Microbe", "relation": "BAO:0002924", "to": "n186", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n37", "id": "e16", "label": "HAS_PHENOTYPE", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002200", "to": "n19", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n37", "id": "e18", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n244", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n37", "id": "e64", "label": "SUBCLASS_OF", "to": "n4", "width": 1}, {"associationSlot": "0.132752226464627", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n38", "id": "e7", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n39", "width": 1}, {"associationSlot": "0.11641347503641", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n38", "id": "e8", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n40", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n38", "id": "e65", "label": "SUBCLASS_OF", "to": "n5", "width": 1}, {"associationSlot": "0.193651826068963", "biome": "ENVO_00002046", "biomeName": "activatedsludge", "color": "#B53C44FF", "font": {"size": 6}, "from": "n39", "id": "e9", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "assembly", "to": "n40", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n39", "id": "e66", "label": "SUBCLASS_OF", "to": "n6", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n40", "id": "e67", "label": "SUBCLASS_OF", "to": "n7", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n203", "id": "e23", "label": "SUBCLASS_OF", "to": "n26", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n204", "id": "e24", "label": "SUBCLASS_OF", "to": "n27", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n205", "id": "e25", "label": "SUBCLASS_OF", "to": "n26", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n206", "id": "e26", "label": "SUBCLASS_OF", "to": "n28", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n207", "id": "e28", "label": "SUBCLASS_OF", "to": "n29", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n208", "id": "e48", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n209", "id": "e39", "label": "SUBCLASS_OF", "to": "n31", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n210", "id": "e41", "label": "SUBCLASS_OF", "to": "n32", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n250", "id": "e76", "label": "SUBCLASS_OF", "to": "n10", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n251", "id": "e77", "label": "SUBCLASS_OF", "to": "n11", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n252", "id": "e78", "label": "SUBCLASS_OF", "to": "n12", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n253", "id": "e79", "label": "SUBCLASS_OF", "to": "n13", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n254", "id": "e80", "label": "SUBCLASS_OF", "to": "n14", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n254", "id": "e81", "label": "SUBCLASS_OF", "to": "n207", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n255", "id": "e82", "label": "SUBCLASS_OF", "to": "n15", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n255", "id": "e83", "label": "SUBCLASS_OF", "to": "n30", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n255", "id": "e84", "label": "SUBCLASS_OF", "to": "n208", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n256", "id": "e85", "label": "SUBCLASS_OF", "to": "n16", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n256", "id": "e86", "label": "SUBCLASS_OF", "to": "n209", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n257", "id": "e87", "label": "SUBCLASS_OF", "to": "n17", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n257", "id": "e88", "label": "SUBCLASS_OF", "to": "n210", "width": 1}]);

                  nodeColors = {};
                  allNodes = nodes.get({ returnType: "Object" });
                  for (nodeId in allNodes) {
                    nodeColors[nodeId] = allNodes[nodeId].color;
                  }
                  allEdges = edges.get({ returnType: "Object" });
                  // adding nodes and edges to the graph
                  data = {nodes: nodes, edges: edges};

                  var options = {
    "configure": {
        "enabled": false
    },
    "edges": {
        "color": {
            "inherit": true
        },
        "smooth": {
            "enabled": true,
            "type": "dynamic"
        }
    },
    "interaction": {
        "dragNodes": true,
        "hideEdgesOnDrag": false,
        "hideNodesOnDrag": false
    },
    "physics": {
        "enabled": true,
        "forceAtlas2Based": {
            "avoidOverlap": 0,
            "centralGravity": 0.005,
            "damping": 0.4,
            "gravitationalConstant": -50,
            "springConstant": 0.08,
            "springLength": 100
        },
        "solver": "forceAtlas2Based",
        "stabilization": {
            "enabled": true,
            "fit": true,
            "iterations": 1000,
            "onlyDynamicEdges": false,
            "updateInterval": 50
        }
    }
};

                  


                  

                  network = new vis.Network(container, data, options);

                  

                  

                  


                  

                  return network;

              }
              drawGraph();
        </script>
    </body>
</html>