<html>
    <head>
        <meta charset="utf-8">
        
            <script src="lib/bindings/utils.js"></script>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/dist/vis-network.min.css" integrity="sha512-WgxfT5LWjfszlPHXRmBWHkV2eceiWTOBvrKCNbdgDYTHrT2AeLCGbF4sZlZw3UMN3WtL0tGUoIAKsu8mllg/XA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
            <script src="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/vis-network.min.js" integrity="sha512-LnvoEWDFrqGHlHmDD2101OrLcbsfkrzoSpvtSQtxK3RMnRV0eOkhhBN2dXHKRrUU8p2DGRTk35n4O8nWSVe1mQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
            
        
<center>
<h1></h1>
</center>

<!-- <link rel="stylesheet" href="../node_modules/vis/dist/vis.min.css" type="text/css" />
<script type="text/javascript" src="../node_modules/vis/dist/vis.js"> </script>-->
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
          crossorigin="anonymous"
        />
        <script
          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js"
          integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf"
          crossorigin="anonymous"
        ></script>


        <center>
          <h1></h1>
        </center>
        <style type="text/css">

             #mynetwork {
                 width: 100%;
                 height: 600px;
                 background-color: white;
                 border: 1px solid lightgray;
                 position: relative;
                 float: left;
             }

             

             

             
        </style>
    </head>


    <body>
        <div class="card" style="width: 100%">
            
            
            <div id="mynetwork" class="card-body"></div>
        </div>

        
        

        <script type="text/javascript">

              // initialize global variables.
              var edges;
              var nodes;
              var allNodes;
              var allEdges;
              var nodeColors;
              var originalNodes;
              var network;
              var container;
              var options, data;
              var filter = {
                  item : '',
                  property : '',
                  value : []
              };

              

              

              // This method is responsible for drawing the graph, returns the drawn network
              function drawGraph() {
                  var container = document.getElementById('mynetwork');

                  

                  // parsing and collecting nodes and edges from the python
                  nodes = new vis.DataSet([{"betweenness_centrality": 2312.9753068107366, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24203133441383037, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n7", "label": "Legionella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Legionella", "pagerank_centrality": 0.7987578413521903, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Sarcobium\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16166707\u0027, \u0027PMID:434652\u0027, \u0027PMID:8573522\u0027, \u0027PMID:9734026\u0027]"}, {"betweenness_centrality": 11335.333778289832, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.2689075630252101, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 82.0, "description": "A trophic process during which a living entity acquires food and energy by consuming another organism or organism part, living or dead.", "font": {"size": 12}, "id": "n556", "iri": "http://purl.obolibrary.org/obo/ECOCORE_00000011", "label": "Heterotrophy", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Heterotrophy", "pagerank_centrality": 4.810868420146474, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 17, "synonym": "", "xref": ""}, {"betweenness_centrality": 3905.6939534506137, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.2501395868230039, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 54.0, "description": "The chemical reactions and pathways involving amino acids, carboxylic acids containing one or more amino groups. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n939", "iri": "http://purl.obolibrary.org/obo/GO_0006520 ", "label": "Amino acid metabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Amino acid metabolic process", "pagerank_centrality": 3.1025410816790253, "providedBy": "MiDAS", "shape": "dot", "size": 11, "synonym": "cellular amino acid metabolic process|cellular amino acid metabolism|amino acid and derivative metabolism|cellular amino acid and derivative metabolic process", "xref": ""}, {"betweenness_centrality": 1711.103151252421, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23848815544317276, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 14.0, "font": {"size": 12}, "id": "n8", "label": "Microbacterium", "labels": ":Genus", "metabolic_cluster": 114, "name": "Microbacterium", "pagerank_centrality": 0.8511220791725651, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Aureibacterium\" EXACT equivalent_name []\u0027, \u0027\"Aureobacterium\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:22058326\u0027, \u0027PMID:25239238\u0027, \u0027PMID:27498827\u0027, \u0027PMID:8347513\u0027, \u0027PMID:9734028\u0027]"}, {"betweenness_centrality": 3862.2225893412833, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.2440087145969499, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 44.0, "description": "The chemical reactions and pathways involving a short-chain fatty acid. A short-chain fatty acid has an aliphatic tail containing fewer than 6 carbons. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n937", "iri": "http://purl.obolibrary.org/obo/GO_0046459 ", "label": "Short-chain fatty acid metabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 25, "name": "Short-chain fatty acid metabolic process", "pagerank_centrality": 2.6273139149710527, "providedBy": "MiDAS", "shape": "dot", "size": 11, "synonym": "short-chain fatty acid metabolism", "xref": ""}, {"betweenness_centrality": 19402.53922432753, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.275777162203755, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 100.0, "description": "The chemical reactions and pathways involving carbohydrates, any of a group of organic compounds based of the general formula Cx(H2O)y. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n938", "iri": "http://purl.obolibrary.org/obo/GO_0005975 ", "label": "Carbohydrate metabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Carbohydrate metabolic process", "pagerank_centrality": 6.027438888431428, "providedBy": "MiDAS", "shape": "dot", "size": 24, "synonym": "carbohydrate metabolism", "xref": ""}, {"betweenness_centrality": 3802.855095924738, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2563662374821173, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 18.0, "font": {"size": 12}, "id": "n10", "label": "Nitrosomonas", "labels": ":Genus", "metabolic_cluster": 114, "name": "Nitrosomonas", "pagerank_centrality": 1.2540224852000383, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 6187.397906429742, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.27069486404833837, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 28.0, "description": "A trophic process during which a living entity generates food from abiotic sources or through consumption of living or dead material, and captures energy from preformed molecules. Autotrophs are able to fix CO2 as a carbon source for growth. Important examples in wastewater treatment include nitrite and ammonia oxidizers (NOB and AOB), anaerobic ammonia oxidizers (anammox). Mixotrophs can obtain carbon by fixing CO2 and or from organic sources (heterotrophy). The criterium for this MiDAS field is the potential for CO2 fixation i.e. organisms possessing the Calvin-cycle.", "font": {"size": 12}, "id": "n544", "iri": "http://purl.obolibrary.org/obo/ECOCORE_00000163", "label": "Chemotrophy", "labels": ":BiologicalProcess", "metabolic_cluster": 78, "name": "Chemotrophy", "pagerank_centrality": 1.34546074033267, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 13, "synonym": "", "xref": ""}, {"betweenness_centrality": 17721.11500321186, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.29176164115923153, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 70.0, "description": "The reduction of nitrate to dinitrogen by four reactions; each intermediate is transformed to the next lower oxidation state; also part of cellular bioenergetics; the nitrogen compounds can serve as terminal acceptors for electron transport phosphorylation in place of oxygen. The denitrifiers are usually facultative-anaerobic heterotrophic organisms utilizing nitrate and/or nitrite as electron acceptors in the absence of oxygen. Denitrification is the final step of nitrogen removal from wastewater and generally occurs under anaerobic conditions. Nitrate is available from the oxidation of ammonia by the aerobic activities of the nitrifiers (AOB and NOB). Denitrification involves the sequential reduction of nitrate to inert dinitrogen gas, via several intermediates (nitrite, nitric oxide (gas) and nitrous oxide (gas)). Not all organisms involved in denitrification possess all the genes for reduction of nitrate all the way to dinitrogen gas. As such, for MiDAS profiles, the nitrite reducing organisms are selected to represent the denitrifiers; given denitrification is defined as \u201cthe dissimilatory transformation of nitrate or nitrite to a gas species concomitant with energy conservation\u201d and not all organisms capable of nitrate reduction can reduce nitrite.", "font": {"size": 12}, "id": "n540", "iri": "http://purl.obolibrary.org/obo/GO_0019333", "label": "Denitrification pathway", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Denitrification pathway", "pagerank_centrality": 2.5029258657669415, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 22, "synonym": "", "xref": ""}, {"betweenness_centrality": 855.0475493277958, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2332725852642541, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n11", "label": "Pseudoxanthomonas", "labels": ":Genus", "metabolic_cluster": 114, "name": "Pseudoxanthomonas", "pagerank_centrality": 0.7576354834234018, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10826814\u0027, \u0027PMID:15545466\u0027, \u0027PMID:18768635\u0027]"}, {"betweenness_centrality": 1036.7703715398338, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2451436388508892, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 14.0, "font": {"size": 12}, "id": "n12", "label": "Stenotrophomonas", "labels": ":Genus", "metabolic_cluster": 114, "name": "Stenotrophomonas", "pagerank_centrality": 0.8460213411116669, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10555357\u0027, \u0027PMID:28629502\u0027, \u0027PMID:8347518\u0027]"}, {"betweenness_centrality": 1827.8297833398635, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2498605688789738, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 32.0, "font": {"size": 12}, "id": "n319", "label": "Agrococcus jenensis", "labels": ":Species", "metabolic_cluster": 424, "name": "Agrococcus jenensis", "pagerank_centrality": 1.5445036243271404, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:8573501\u0027]"}, {"betweenness_centrality": 2378.3851543660953, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2520393811533052, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 38.0, "font": {"size": 12}, "id": "n321", "label": "Brevibacterium casei", "labels": ":Species", "metabolic_cluster": 424, "name": "Brevibacterium casei", "pagerank_centrality": 1.7589317381705762, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"CDC coryneform group B-1\" RELATED synonym []\u0027, \u0027\"CDC coryneform group B-3\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11594628\u0027]"}, {"betweenness_centrality": 5164.350899133885, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24534501642935377, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 52.0, "font": {"size": 12}, "id": "n322", "label": "Filimonas lacunae", "labels": ":Species", "metabolic_cluster": 424, "name": "Filimonas lacunae", "pagerank_centrality": 2.5143541192404277, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 12, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19406807\u0027, \u0027PMID:23667140\u0027, \u0027PMID:27417842\u0027]"}, {"betweenness_centrality": 2034.7371430605708, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22473037371457236, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 32.0, "font": {"size": 12}, "id": "n324", "label": "Halorhodospira halophila", "labels": ":Species", "metabolic_cluster": 424, "name": "Halorhodospira halophila", "pagerank_centrality": 1.548974023125325, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Ectothiorhodospira halophila\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:4192367\u0027, \u0027PMID:8593098\u0027]"}, {"betweenness_centrality": 1210.992689132325, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24778761061946902, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 32.0, "font": {"size": 12}, "id": "n325", "label": "Kerstersia gyiorum", "labels": ":Species", "metabolic_cluster": 424, "name": "Kerstersia gyiorum", "pagerank_centrality": 1.4978036520399616, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:14657111\u0027]"}, {"betweenness_centrality": 3015.8687115079542, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.26229508196721313, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 40.0, "font": {"size": 12}, "id": "n326", "label": "Legionella dresdenensis", "labels": ":Species", "metabolic_cluster": 424, "name": "Legionella dresdenensis", "pagerank_centrality": 1.8657749786650628, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 10, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:20008105\u0027]"}, {"betweenness_centrality": 2002.697410184882, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2606924643584521, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 42.0, "font": {"size": 12}, "id": "n327", "label": "Microbacterium lacticum", "labels": ":Species", "metabolic_cluster": 424, "name": "Microbacterium lacticum", "pagerank_centrality": 1.8398606794169858, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 4270.634821245146, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2637621430674124, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 50.0, "font": {"size": 12}, "id": "n329", "label": "Roseateles chitosanitabidus", "labels": ":Species", "metabolic_cluster": 424, "name": "Roseateles chitosanitabidus", "pagerank_centrality": 2.233000757175932, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 11, "synonym": "[\u0027\"Mitsuaria chitosanitabida Amakata et al. 2005\" RELATED synonym []\u0027, \u0027\"Roseateles chitosanitabidus (Amakata et al. 2005) Liu et al. 2023\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 1303.166024979197, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2279318239633681, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 32.0, "font": {"size": 12}, "id": "n330", "label": "Nitrosomonas nitrosa", "labels": ":Species", "metabolic_cluster": 424, "name": "Nitrosomonas nitrosa", "pagerank_centrality": 1.5770677788283982, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 1810.6610705421926, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2546177891446434, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 36.0, "font": {"size": 12}, "id": "n331", "label": "Pseudoxanthomonas taiwanensis", "labels": ":Species", "metabolic_cluster": 424, "name": "Pseudoxanthomonas taiwanensis", "pagerank_centrality": 1.7180720314852833, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12508883\u0027]"}, {"betweenness_centrality": 436.0039972658743, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22968469623173546, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 30.0, "font": {"size": 12}, "id": "n333", "label": "Stenotrophomonas geniculata", "labels": ":Species", "metabolic_cluster": 424, "name": "Stenotrophomonas geniculata", "pagerank_centrality": 1.370016151643956, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Pseudomonas geniculata (Wright 1895) Chester 1901\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10939664\u0027]"}, {"betweenness_centrality": 58.17140515432553, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.20968874327170606, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 4.0, "description": "Manure which has been derived from the aerobically decomposed remnants of organic materials.", "font": {"size": 12}, "id": "n762", "iri": "http://purl.obolibrary.org/obo/ENVO_00002170", "label": "compost", "labels": ":EnvironmentalFeature", "name": "compost", "pagerank_centrality": 0.34901085032420476, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "", "xref": "https://en.wikipedia.org/wiki/Compost"}, {"betweenness_centrality": 167.10920555511316, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20735940754454987, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n0", "label": "Agrococcus", "labels": ":Genus", "metabolic_cluster": 424, "name": "Agrococcus", "pagerank_centrality": 0.3850944899683395, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:8573501\u0027]"}, {"betweenness_centrality": 2247.0385949487877, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22950819672131148, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 26.0, "font": {"size": 12}, "id": "n323", "label": "Fulvimonas soli", "labels": ":Species", "metabolic_cluster": 424, "name": "Fulvimonas soli", "pagerank_centrality": 1.4128457091861975, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12148641\u0027]"}, {"betweenness_centrality": 1651.6777863656682, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.264619019492026, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 32.0, "font": {"size": 12}, "id": "n328", "label": "Microbacterium liquefaciens", "labels": ":Species", "metabolic_cluster": 424, "name": "Microbacterium liquefaciens", "pagerank_centrality": 1.4292646270302274, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Aureibacterium liquefaciens\" EXACT equivalent_name []\u0027, \u0027\"Aureobacterium liquefaciens (ex Orla-Jensen 1919) Collins et al. 1983\" RELATED synonym []\u0027, \u0027\"Flavobacterium marinotypicum\" RELATED synonym []\u0027, \u0027\"Microbacterium maritypicum\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:36598287\u0027, \u0027PMID:9734028\u0027]"}, {"betweenness_centrality": 737.8464145291281, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24696802646085997, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 30.0, "font": {"size": 12}, "id": "n332", "label": "Roseateles depolymerans", "labels": ":Species", "metabolic_cluster": 424, "name": "Roseateles depolymerans", "pagerank_centrality": 1.4108589501377085, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 4218.20603797534, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.26145316603443247, "cluster": "0", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 46.0, "font": {"size": 12}, "id": "n334", "label": "Thalassobacter stenotrophicus", "labels": ":Species", "metabolic_cluster": 424, "name": "Thalassobacter stenotrophicus", "pagerank_centrality": 2.153854657445241, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 11, "synonym": "[\u0027\"Jannaschia cystaugens\" RELATED synonym []\u0027, \u0027\"Thalassobacter oligotrophus\" EXACT equivalent_name []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:15388729\u0027, \u0027PMID:15653862\u0027, \u0027PMID:16166695\u0027]"}, {"betweenness_centrality": 751.2048047269486, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20503432494279175, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n2", "label": "Brevibacterium", "labels": ":Genus", "metabolic_cluster": 424, "name": "Brevibacterium", "pagerank_centrality": 0.45565266743445887, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:7520745\u0027]"}, {"betweenness_centrality": 193.28635792808444, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.2309278350515464, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 10.0, "description": "Water which has a low concentration of dissolved solutes, particularly that of sodium chloride.", "font": {"size": 12}, "id": "n755", "iri": "http://purl.obolibrary.org/obo/ENVO_00002011", "label": "fresh water", "labels": ":EnvironmentalFeature", "name": "fresh water", "pagerank_centrality": 0.5237821892608215, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "freshwater|sweet water", "xref": "EcoLexicon:fresh_water|LTER:216|SWEETRealm:FreshWater|https://en.wikipedia.org/wiki/Fresh_water"}, {"betweenness_centrality": 636.9713387502323, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.25440090857467346, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 14.0, "description": "An aldohexose that is the C-4 epimer of glucose.", "font": {"size": 12}, "id": "n605", "iri": "http://purl.obolibrary.org/obo/CHEBI_28260", "label": "galactose", "labels": ":ChemicalEntity", "metabolic_cluster": 25, "name": "galactose", "pagerank_centrality": 0.5908334419430574, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "Gal|Galactose|Galaktose|galacto-hexose|galactose", "xref": "CAS:26566-61-0|KEGG:C01582|Wikipedia:Galactose"}, {"betweenness_centrality": 896.7995741117296, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.23357664233576642, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 16.0, "description": "A triol with a structure of propane substituted at positions 1, 2 and 3 by hydroxy groups.", "font": {"size": 12}, "id": "n580", "iri": "http://purl.obolibrary.org/obo/CHEBI_17754", "label": "glycerol", "labels": ":ChemicalEntity", "metabolic_cluster": 424, "name": "glycerol", "pagerank_centrality": 0.8423954215273004, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "1,2,3-Propanetriol|1,2,3-Trihydroxypropane|GLYCEROL|Glycerin|Glyceritol|Glycerol|Glyzerin|Gro|Oelsuess|Propanetriol|Trihydroxypropane|glycerine|glycerol|glycerolum|glycyl alcohol|propane-1,2,3-triol", "xref": "Beilstein:635685|CAS:56-81-5|Chemspider:733|DrugBank:DB04077|Drug_Central:1316|ECMDB:ECMDB00131|FooDB:FDB000756|Gmelin:26279|HMDB:HMDB0000131|KEGG:C00116|KEGG:D00028|KNApSAcK:C00001163|LINCS:LSM-37180|MetaCyc:GLYCEROL|PDB:2AJS|PDB:2D03|PDBeChem:GOL|PMID:11302662|PMID:11958517|PMID:11994365|PMID:12672239|PMID:12687625|PMID:12689633|PMID:12713573|PMID:14559393|PMID:14563847|PMID:15026783|PMID:15342117|PMID:15786693|PMID:15983192|PMID:16244855|PMID:16258193|PMID:16319039|PMID:16349488|PMID:16475911|PMID:16651733|PMID:16664750|PMID:16901854|PMID:17336832|PMID:17439666|PMID:17979222|PMID:19184438|PMID:19231894|PMID:19460032|PMID:19548674|PMID:19795216|PMID:19956799|PMID:22705534|PMID:23562176|PMID:23747440|PMID:24643482|PMID:24835191|PMID:25108762|PMID:558160|PMID:6299616|PMID:7031247|PMID:7392035|PPDB:1317|Reaxys:635685|UM-BBD_compID:c0066|Wikipedia:Glycerol|YMDB:YMDB00283"}, {"betweenness_centrality": 1559.5658342799566, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.2637621430674124, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 20.0, "description": "An aldohexose that is the C-2 epimer of glucose.", "font": {"size": 12}, "id": "n616", "iri": "http://purl.obolibrary.org/obo/CHEBI_37684", "label": "mannose", "labels": ":ChemicalEntity", "metabolic_cluster": 25, "name": "mannose", "pagerank_centrality": 0.8397680545832089, "providedBy": "KG-Microbe", "shape": "dot", "size": 9, "synonym": "Man|manno-hexose|mannose", "xref": "PMID:16180318|PMID:24407290"}, {"betweenness_centrality": 2708.8042947230515, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.27094042939219837, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 30.0, "description": "An aldohexose used as a source of energy and metabolic intermediate.", "font": {"size": 12}, "id": "n573", "iri": "http://purl.obolibrary.org/obo/CHEBI_17234", "label": "glucose", "labels": ":ChemicalEntity", "metabolic_cluster": 424, "name": "glucose", "pagerank_centrality": 1.241555559863512, "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "DL-glucose|Glc|Glucose|Glukose|gluco-hexose|glucose", "xref": "CAS:50-99-7|KEGG:C00293|Wikipedia:Glucose"}, {"betweenness_centrality": 43.416556743188245, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.21197066477407145, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 4.0, "description": "Glucans produced by the hydrolysis of starch or glycogen. They are mixtures of polymers of D-glucose units linked by alpha(1-\u003e4) or alpha(1-\u003e6) glycosidic bonds.", "font": {"size": 12}, "id": "n606", "iri": "http://purl.obolibrary.org/obo/CHEBI_28675", "label": "dextrin", "labels": ":ChemicalEntity", "metabolic_cluster": 464, "name": "dextrin", "pagerank_centrality": 0.3187515144947211, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "dextrine|dextrines|dextrins", "xref": "CAS:9004-53-9|FooDB:FDB012148|HMDB:HMDB0006857|KEGG:C00721|Wikipedia:Dextrin"}, {"betweenness_centrality": 2461.044613544097, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.26842420611144396, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 30.0, "description": "A glycosylglucose consisting of two D-glucopyranose units connected by an alpha-(1-\u003e4)-linkage.", "font": {"size": 12}, "id": "n576", "iri": "http://purl.obolibrary.org/obo/CHEBI_17306", "label": "maltose", "labels": ":ChemicalEntity", "metabolic_cluster": 78, "name": "maltose", "pagerank_centrality": 1.1612706702657964, "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "1-alpha-D-Glucopyranosyl-4-alpha-D-glucopyranose|4-(alpha-D-glucopyranosido)-alpha-glucopyranose|4-(alpha-D-glucosido)-D-glucose|4-O-alpha-D-glucopyranosyl-D-glucopyranose|4-O-alpha-D-glucopyranosyl-D-glucose|Cextromaltose|D-(+)-maltose|D-maltose|Malt sugar|Maltose|Malzzucker|alpha-D-Glcp-(1-\u003e4)-D-Glcp|alpha-D-Glucopyranosyl-(1-\u003e4)-D-glucopyranose|alpha-D-glucopyranosyl-(1-\u003e4)-D-glucopyranose|alpha-D-glucopyranosyl-(1-\u003e4)-D-glucose|alpha-malt sugar|maltobiose", "xref": "Beilstein:1292747|CAS:69-79-4|DrugBank:DB03323|KEGG:C00208|KEGG:D00044|KEGG:G00275|KNApSAcK:C00001140|PMID:16332759|PMID:17723085|PMID:22094343|PMID:22185612|PMID:22246222|PMID:22252265|PMID:22411612|PMID:22424089|PMID:22451670|PMID:22469630|PMID:22529943|PMID:22573161|PMID:22669197|Reaxys:1292747|Wikipedia:Maltose"}, {"betweenness_centrality": 0.0, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.19705300197932704, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 2.0, "description": "A carbonyl group with two C-bound amine groups. The commercially available fertilizer has an analysis of 46-0-0 (N-P2O5-K2O).", "font": {"size": 12}, "id": "n562", "iri": "http://purl.obolibrary.org/obo/CHEBI_16199", "label": "urea", "labels": ":ChemicalEntity", "metabolic_cluster": 424, "name": "urea", "pagerank_centrality": 0.23153808674445964, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "1728|Carbamide|E927b|H2NC(O)NH2|Harnstoff|Karbamid|UREA|Urea|carbamide|carbonyldiamide|ur|urea|uree", "xref": "Beilstein:635724|CAS:57-13-6|DrugBank:DB03904|Drug_Central:4264|ECMDB:ECMDB04172|Gmelin:1378|HMDB:HMDB0000294|KEGG:C00086|KEGG:D00023|KNApSAcK:C00007314|MetaCyc:UREA|PDBeChem:URE|PMID:18037357|PMID:22770225|PPDB:1728|Reaxys:635724|UM-BBD_compID:c0165|Wikipedia:Urea|YMDB:YMDB00003"}, {"betweenness_centrality": 3359.7382069695536, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.26810293237582283, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 22.0, "description": "", "font": {"size": 12}, "id": "n589", "iri": "http://purl.obolibrary.org/obo/CHEBI_22599", "label": "arabinose", "labels": ":ChemicalEntity", "metabolic_cluster": 114, "name": "arabinose", "pagerank_centrality": 1.0236235921807553, "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "Ara|arabino-pentose|arabinose", "xref": "CAS:147-81-9|HMDB:HMDB0029942|Wikipedia:Arabinose"}, {"betweenness_centrality": 1980.6147056817917, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.1997325011145787, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n3", "label": "Filimonas", "labels": ":Genus", "metabolic_cluster": 424, "name": "Filimonas", "pagerank_centrality": 0.5236304941395309, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19406807\u0027, \u0027PMID:27412118\u0027]"}, {"betweenness_centrality": 2915.41891751151, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.2575452716297787, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 34.0, "description": "An environmental material which is primarily composed of minerals, varying proportions of sand, silt, and clay, organic material such as humus, interstitial gases, liquids, and a broad range of resident micro- and macroorganisms.", "font": {"size": 12}, "id": "n751", "iri": "http://purl.obolibrary.org/obo/ENVO_00001998", "label": "soil", "labels": ":EnvironmentalFeature", "name": "soil", "pagerank_centrality": 1.3910496476433358, "providedBy": "KG-Microbe", "shape": "dot", "size": 10, "synonym": "regolith", "xref": "LTER:535|SPIRE:Soil|SWEETRealm:Soil|https://en.wikipedia.org/wiki/Soil"}, {"betweenness_centrality": 522.5515825978927, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19173978172480205, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n4", "label": "Fulvimonas", "labels": ":Genus", "metabolic_cluster": 424, "name": "Fulvimonas", "pagerank_centrality": 0.47402172172190915, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12148641\u0027, \u0027PMID:24860108\u0027]"}, {"betweenness_centrality": 452.66704409756414, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.22770012706480305, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 10.0, "description": "Phototrophs are the organisms that carry out photon capture to acquire energy. They use the energy from light to carry out various cellular metabolic processes. Most of the well-recognized phototrophs are autotrophs, also known as photoautotrophs, and can fix carbon.", "font": {"size": 12}, "id": "n553", "iri": "http://purl.obolibrary.org/obo/MICRO_0000498", "label": "Phototrophy", "labels": ":BiologicalProcess", "metabolic_cluster": 424, "name": "Phototrophy", "pagerank_centrality": 0.5287172022070357, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "", "xref": ""}, {"betweenness_centrality": 1230.8306643446545, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.1903547907371999, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n5", "label": "Halorhodospira", "labels": ":Genus", "metabolic_cluster": 424, "name": "Halorhodospira", "pagerank_centrality": 0.4640871037871941, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Halorhodospira Imhoff and Suling 1997\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12656167\u0027]"}, {"betweenness_centrality": 182.17525925597508, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20322068496257656, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n6", "label": "Kerstersia", "labels": ":Genus", "metabolic_cluster": 424, "name": "Kerstersia", "pagerank_centrality": 0.40254482182983603, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:14657111\u0027]"}, {"betweenness_centrality": 14.140280782969873, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.2248995983935743, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 6.0, "description": "An emulsion of fat globules within a fluid that is secreted by the mammary gland during lactation.", "font": {"size": 12}, "id": "n778", "iri": "http://purl.obolibrary.org/obo/UBERON_0001913", "label": "milk", "labels": ":EnvironmentalFeature", "name": "milk", "pagerank_centrality": 0.3794637634328347, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "mammary gland milk", "xref": ""}, {"betweenness_centrality": 252.8799001693306, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.23785505707459517, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 10.0, "description": "A physiological role played by any substance of either plant, animal or artificial origin which contains essential body nutrients that can be ingested by an organism to provide energy, promote growth, and maintain the processes of life.", "font": {"size": 12}, "id": "n747", "iri": "http://purl.obolibrary.org/obo/CHEBI_33290", "label": "food", "labels": ":EnvironmentalFeature", "name": "food", "pagerank_centrality": 0.5673561699438558, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "food material|food materials|food role|foods|foodstuff|foodstuffs", "xref": ""}, {"betweenness_centrality": 0.0, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.2087604846225536, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 2.0, "description": "", "font": {"size": 12}, "id": "n618", "iri": "http://purl.obolibrary.org/obo/CHEBI_5417", "label": "glucosamine", "labels": ":ChemicalEntity", "metabolic_cluster": 468, "name": "glucosamine", "pagerank_centrality": 0.2252675118651931, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "2-Amino-2-deoxy-glucose|2-amino-2-deoxyglucose|GlcN|Glucosamin|Glucosamine|Glukosamin|glucosamine", "xref": "DrugBank:DB01296|KEGG:C01811"}, {"betweenness_centrality": 755.9487000420106, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.21227197346600332, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 6.0, "font": {"size": 12}, "id": "n9", "label": "Roseateles", "labels": ":Genus", "metabolic_cluster": 424, "name": "Roseateles", "pagerank_centrality": 0.5065250926510557, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Kinneretia\" RELATED synonym []\u0027, \u0027\"Matsuenia\" EXACT equivalent_name []\u0027, \u0027\"Mitsuaria\" RELATED synonym []\u0027, \u0027\"Mitsuaria Amakata et al. 2005\" RELATED synonym []\u0027, \u0027\"Paucibacter\" RELATED synonym []\u0027, \u0027\"Pelomonas\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 101.07941212053674, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.2312258064516129, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 8.0, "description": "A spring through which groundwater, heated by geothermal energy, flows.", "font": {"size": 12}, "id": "n748", "iri": "http://purl.obolibrary.org/obo/ENVO_00000051", "label": "hot spring", "labels": ":EnvironmentalFeature", "name": "hot spring", "pagerank_centrality": 0.4283118164244562, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "hotspring|spring|thermal spring", "xref": "EcoLexicon:thermal_spring|FTT:732|FTT:815|Geonames:H.SPNT|https://en.wikipedia.org/wiki/Hot_spring"}, {"betweenness_centrality": 104.47853617402569, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.23116615067079463, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 8.0, "description": "An environmental material primarily composed of dihydrogen oxide in its liquid form.", "font": {"size": 12}, "id": "n753", "iri": "http://purl.obolibrary.org/obo/ENVO_00002006", "label": "liquid water", "labels": ":EnvironmentalFeature", "name": "liquid water", "pagerank_centrality": 0.42988291793824623, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "water", "xref": "EcoLexicon:water|LTER:617|https://en.wikipedia.org/wiki/Water"}, {"betweenness_centrality": 602.4118894700409, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.2463568875446797, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 12.0, "description": "Water which has physicochemical properties that have been determined by the processes occuring in a sea or ocean.", "font": {"size": 12}, "id": "n761", "iri": "http://purl.obolibrary.org/obo/ENVO_00002149", "label": "sea water", "labels": ":EnvironmentalFeature", "name": "sea water", "pagerank_centrality": 0.5659280719577686, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "ocean water|seawater", "xref": "EcoLexicon:sea_water|LTER:701|SWEETRealm:SeaWater|https://en.wikipedia.org/wiki/Sea_water"}, {"betweenness_centrality": 414.96728680568344, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.23678646934460887, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 12.0, "description": "A dicarboxylic acid anion obtained by deprotonation of at least one of the carboxy groups of succinic acid.", "font": {"size": 12}, "id": "n597", "iri": "http://purl.obolibrary.org/obo/CHEBI_26806", "label": "succinate", "labels": ":ChemicalEntity", "metabolic_cluster": 57, "name": "succinate", "pagerank_centrality": 0.6638858301866883, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "succinate anion|succinates|succinic acid anion", "xref": ""}, {"betweenness_centrality": 1012.6570563140104, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.26008708272859216, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 16.0, "description": "A hexitol produced by a variety of organisms including bacteria, fungi, lichens and plants.", "font": {"size": 12}, "id": "n609", "iri": "http://purl.obolibrary.org/obo/CHEBI_29864", "label": "mannitol", "labels": ":ChemicalEntity", "metabolic_cluster": 78, "name": "mannitol", "pagerank_centrality": 0.7664371937626625, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "mannitol", "xref": "PMID:24269997|PMID:24323504|PMID:24374122|PMID:24861101|Wikipedia:Mannitol"}, {"betweenness_centrality": 1032.3158040563278, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.25963488843813387, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 16.0, "description": "A 2-oxo monocarboxylic acid anion that is the conjugate base of pyruvic acid, arising from deprotonation of the carboxy group.", "font": {"size": 12}, "id": "n560", "iri": "http://purl.obolibrary.org/obo/CHEBI_15361", "label": "pyruvate", "labels": ":ChemicalEntity", "metabolic_cluster": 424, "name": "pyruvate", "pagerank_centrality": 0.8122519763571289, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "2-oxopropanoate|2-oxopropanoic acid, ion(1-)|pyruvate", "xref": "Beilstein:3587721|CAS:57-60-3|Gmelin:2502|KEGG:C00022|PMID:17190852|PMID:21603897|PMID:21823181|PMID:21854850|PMID:22006570|PMID:22016370|PMID:22215378|PMID:22311625|PMID:22451307|PMID:22458763|Reaxys:3587721|UM-BBD_compID:c0159"}, {"betweenness_centrality": 10.325897469532203, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.2157476522995425, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 4.0, "description": "", "font": {"size": 12}, "id": "n591", "iri": "http://purl.obolibrary.org/obo/CHEBI_24266", "label": "gluconic acid", "labels": ":ChemicalEntity", "metabolic_cluster": 444, "name": "gluconic acid", "pagerank_centrality": 0.2993032426569879, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "", "xref": "DrugBank:DB04304"}, {"betweenness_centrality": 1100.246894938073, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.2398929049531459, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 12.0, "description": "A monocarboxylic acid anion resulting from the removal of a proton from the carboxy group of acetic acid.", "font": {"size": 12}, "id": "n610", "iri": "http://purl.obolibrary.org/obo/CHEBI_30089", "label": "acetate", "labels": ":ChemicalEntity", "metabolic_cluster": 78, "name": "acetate", "pagerank_centrality": 0.7450895488639689, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "ACETATE ION|Azetat|CH3-COO(-)|Ethanoat|MeCO2 anion|acetate|acetic acid, ion(1-)|ethanoate", "xref": "Beilstein:1901470|CAS:71-50-1|DrugBank:DB03166|Gmelin:1379|KEGG:C00033|MetaCyc:ACET|PDBeChem:ACT|PMID:17190852|PMID:22211106|PMID:22371380|Reaxys:1901470|UM-BBD_compID:c0050|Wikipedia:Acetate"}, {"betweenness_centrality": 794.9378004563956, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.21032863849765257, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n13", "label": "Thalassobacter", "labels": ":Genus", "metabolic_cluster": 424, "name": "Thalassobacter", "pagerank_centrality": 0.4748981260273548, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:15653862\u0027, \u0027PMID:16166695\u0027]"}]);
                  edges = new vis.DataSet([{"color": "#11449cff", "font": {"size": 6}, "from": "n7", "hasattribute": "Positive", "id": "e2335", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n7", "hasattribute": "Positive", "id": "e2710", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e2342", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e2497", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n937", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e2524", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n8", "hasattribute": "Positive", "id": "e2715", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e2333", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e2371", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n544", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e2482", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n540", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e2492", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n937", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n10", "hasattribute": "Positive", "id": "e2515", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n11", "hasattribute": "Positive", "id": "e2339", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n11", "hasattribute": "Positive", "id": "e2496", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n937", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n11", "hasattribute": "Positive", "id": "e2521", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n11", "hasattribute": "Positive", "id": "e2713", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e2357", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e2509", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n937", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e2547", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n12", "hasattribute": "Positive", "id": "e2728", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"associationSlot": 0.295789239067757, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e0", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n321", "width": 1}, {"associationSlot": 0.295789239067666, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e1", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n322", "width": 1}, {"associationSlot": 0.295789239067729, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e2", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n324", "width": 1}, {"associationSlot": 0.295789239067706, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e3", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n325", "width": 1}, {"associationSlot": 0.295789239067605, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e4", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n326", "width": 1}, {"associationSlot": 0.29578923906769, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e5", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n327", "width": 1}, {"associationSlot": 0.277071009394581, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e6", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.295789239067694, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e7", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.295789239067679, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e8", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067696, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n319", "id": "e9", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n319", "id": "e2288", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n762", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n319", "id": "e2812", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n544", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n319", "id": "e3078", "label": "SUBCLASS_OF", "to": "n0", "width": 1}, {"associationSlot": 0.457741853243237, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e20", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n322", "width": 1}, {"associationSlot": 0.295789239067734, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e21", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n323", "width": 1}, {"associationSlot": 0.457741853243908, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e22", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n324", "width": 1}, {"associationSlot": 0.457741853243805, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e23", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n325", "width": 1}, {"associationSlot": 0.457741853243436, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e24", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n326", "width": 1}, {"associationSlot": 0.457741853243996, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e25", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n327", "width": 1}, {"associationSlot": 0.295789239067644, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e26", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n328", "width": 1}, {"associationSlot": 0.292209917458086, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e27", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.457741853243881, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e28", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.45774185324386, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e29", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067731, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e30", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243846, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e31", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067709, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n321", "id": "e32", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n321", "id": "e3082", "label": "SUBCLASS_OF", "to": "n2", "width": 1}, {"associationSlot": 0.295789239067726, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e33", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n323", "width": 1}, {"associationSlot": 0.457741853243612, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e34", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n324", "width": 1}, {"associationSlot": 0.457741853243761, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e35", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n325", "width": 1}, {"associationSlot": 0.457741853243719, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e36", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n326", "width": 1}, {"associationSlot": 0.457741853244009, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e37", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n327", "width": 1}, {"associationSlot": 0.295789239067646, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e38", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n328", "width": 1}, {"associationSlot": 0.292209917458036, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e39", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.457741853243835, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e40", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.457741853243825, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e41", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067669, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e42", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243846, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e43", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067659, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n322", "id": "e44", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e2291", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n755", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e2999", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n605", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e3000", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n580", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e3001", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n616", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e3002", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n573", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e3003", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n606", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e3004", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n576", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e3005", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n562", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n322", "id": "e3006", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n589", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n322", "id": "e3083", "label": "SUBCLASS_OF", "to": "n3", "width": 1}, {"associationSlot": 0.295789239067833, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e45", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n324", "width": 1}, {"associationSlot": 0.295789239067566, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e46", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n325", "width": 1}, {"associationSlot": 0.295789239067439, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e47", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n326", "width": 1}, {"associationSlot": 0.295789239067714, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e48", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n327", "width": 1}, {"associationSlot": 0.277071009394576, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e49", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.295789239067676, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e50", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.295789239067594, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e51", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067626, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n323", "id": "e52", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n323", "id": "e2200", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n751", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n323", "id": "e3084", "label": "SUBCLASS_OF", "to": "n4", "width": 1}, {"associationSlot": 0.457741853243849, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e53", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n325", "width": 1}, {"associationSlot": 0.457741853243645, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e54", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n326", "width": 1}, {"associationSlot": 0.457741853243774, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e55", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n327", "width": 1}, {"associationSlot": 0.295789239067703, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e56", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n328", "width": 1}, {"associationSlot": 0.29220991745798, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e57", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.457741853243874, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e58", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.457741853243833, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e59", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067698, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e60", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243842, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e61", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067696, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n324", "id": "e62", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n324", "id": "e2739", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n553", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n324", "id": "e3085", "label": "SUBCLASS_OF", "to": "n5", "width": 1}, {"associationSlot": 0.45774185324395, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e63", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n326", "width": 1}, {"associationSlot": 0.457741853243732, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e64", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n327", "width": 1}, {"associationSlot": 0.295789239067634, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e65", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n328", "width": 1}, {"associationSlot": 0.292209917457966, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e66", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.457741853243822, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e67", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.457741853243757, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e68", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067628, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e69", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243783, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e70", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067642, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n325", "id": "e71", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n325", "id": "e3086", "label": "SUBCLASS_OF", "to": "n6", "width": 1}, {"associationSlot": 0.457741853243576, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e72", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n327", "width": 1}, {"associationSlot": 0.295789239067607, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e73", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n328", "width": 1}, {"associationSlot": 0.292209917457901, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e74", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.457741853243744, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e75", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.457741853243733, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e76", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067626, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e77", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243747, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e78", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067604, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n326", "id": "e79", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n326", "id": "e2287", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n755", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n326", "id": "e3087", "label": "SUBCLASS_OF", "to": "n7", "width": 1}, {"associationSlot": 0.295789239067691, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n327", "id": "e80", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n328", "width": 1}, {"associationSlot": 0.292209917458, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n327", "id": "e81", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.457741853243809, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n327", "id": "e82", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.457741853243724, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n327", "id": "e83", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067671, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n327", "id": "e84", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243825, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n327", "id": "e85", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067669, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n327", "id": "e86", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n327", "id": "e2259", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n778", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n327", "id": "e2792", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n540", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n327", "id": "e3088", "label": "SUBCLASS_OF", "to": "n8", "width": 1}, {"associationSlot": 0.277071009394596, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n328", "id": "e87", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n329", "width": 1}, {"associationSlot": 0.295789239067644, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n328", "id": "e88", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.295789239067625, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n328", "id": "e89", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067644, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n328", "id": "e90", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n328", "id": "e2260", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n747", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n328", "id": "e2261", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n778", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n328", "id": "e2793", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n540", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n328", "id": "e3089", "label": "SUBCLASS_OF", "to": "n8", "width": 1}, {"associationSlot": 0.292209917457992, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n329", "id": "e91", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n330", "width": 1}, {"associationSlot": 0.292209917457969, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n329", "id": "e92", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.277071009394593, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n329", "id": "e93", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.292209917457968, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n329", "id": "e94", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.277071009394581, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n329", "id": "e95", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n329", "id": "e2313", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n751", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n329", "id": "e2831", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n540", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n329", "id": "e2832", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n544", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n329", "id": "e3033", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n576", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n329", "id": "e3034", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n580", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n329", "id": "e3035", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n573", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n329", "id": "e3036", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n618", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n329", "id": "e3090", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"associationSlot": 0.4577418532438, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n330", "id": "e96", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n331", "width": 1}, {"associationSlot": 0.295789239067675, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n330", "id": "e97", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243756, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n330", "id": "e98", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067677, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n330", "id": "e99", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n330", "id": "e3091", "label": "SUBCLASS_OF", "to": "n10", "width": 1}, {"associationSlot": 0.295789239067646, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n331", "id": "e100", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n332", "width": 1}, {"associationSlot": 0.457741853243697, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n331", "id": "e101", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"associationSlot": 0.295789239067618, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n331", "id": "e102", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n331", "id": "e2212", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n748", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n331", "id": "e3092", "label": "SUBCLASS_OF", "to": "n11", "width": 1}, {"associationSlot": 0.295789239067635, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n332", "id": "e103", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n333", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n332", "id": "e2319", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n753", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n332", "id": "e2320", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n755", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n332", "id": "e2838", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n553", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n332", "id": "e3093", "label": "SUBCLASS_OF", "to": "n9", "width": 1}, {"associationSlot": 0.295789239067637, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n333", "id": "e104", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n334", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n333", "id": "e3094", "label": "SUBCLASS_OF", "to": "n12", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2224", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n761", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2778", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n553", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2946", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n580", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2947", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n573", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2948", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n597", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2949", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n609", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2950", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n560", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2951", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n591", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n334", "id": "e2952", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n610", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n334", "id": "e3095", "label": "SUBCLASS_OF", "to": "n13", "width": 1}]);

                  nodeColors = {};
                  allNodes = nodes.get({ returnType: "Object" });
                  for (nodeId in allNodes) {
                    nodeColors[nodeId] = allNodes[nodeId].color;
                  }
                  allEdges = edges.get({ returnType: "Object" });
                  // adding nodes and edges to the graph
                  data = {nodes: nodes, edges: edges};

                  var options = {
    "configure": {
        "enabled": false
    },
    "edges": {
        "color": {
            "inherit": true
        },
        "smooth": {
            "enabled": true,
            "type": "dynamic"
        }
    },
    "interaction": {
        "dragNodes": true,
        "hideEdgesOnDrag": false,
        "hideNodesOnDrag": false
    },
    "physics": {
        "enabled": true,
        "forceAtlas2Based": {
            "avoidOverlap": 0,
            "centralGravity": 0.005,
            "damping": 0.4,
            "gravitationalConstant": -15,
            "springConstant": 0.1,
            "springLength": 100
        },
        "solver": "forceAtlas2Based",
        "stabilization": {
            "enabled": true,
            "fit": true,
            "iterations": 1000,
            "onlyDynamicEdges": false,
            "updateInterval": 50
        }
    }
};

                  


                  

                  network = new vis.Network(container, data, options);

                  

                  

                  


                  

                  return network;

              }
              drawGraph();
        </script>
    </body>
</html>