<html>
    <head>
        <meta charset="utf-8">
        
            <script src="lib/bindings/utils.js"></script>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/dist/vis-network.min.css" integrity="sha512-WgxfT5LWjfszlPHXRmBWHkV2eceiWTOBvrKCNbdgDYTHrT2AeLCGbF4sZlZw3UMN3WtL0tGUoIAKsu8mllg/XA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
            <script src="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/vis-network.min.js" integrity="sha512-LnvoEWDFrqGHlHmDD2101OrLcbsfkrzoSpvtSQtxK3RMnRV0eOkhhBN2dXHKRrUU8p2DGRTk35n4O8nWSVe1mQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
            
        
<center>
<h1></h1>
</center>

<!-- <link rel="stylesheet" href="../node_modules/vis/dist/vis.min.css" type="text/css" />
<script type="text/javascript" src="../node_modules/vis/dist/vis.js"> </script>-->
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
          crossorigin="anonymous"
        />
        <script
          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js"
          integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf"
          crossorigin="anonymous"
        ></script>


        <center>
          <h1></h1>
        </center>
        <style type="text/css">

             #mynetwork {
                 width: 100%;
                 height: 600px;
                 background-color: white;
                 border: 1px solid lightgray;
                 position: relative;
                 float: left;
             }

             

             

             
        </style>
    </head>


    <body>
        <div class="card" style="width: 100%">
            
            
            <div id="mynetwork" class="card-body"></div>
        </div>

        
        

        <script type="text/javascript">

              // initialize global variables.
              var edges;
              var nodes;
              var allNodes;
              var allEdges;
              var nodeColors;
              var originalNodes;
              var network;
              var container;
              var options, data;
              var filter = {
                  item : '',
                  property : '',
                  value : []
              };

              

              

              // This method is responsible for drawing the graph, returns the drawn network
              function drawGraph() {
                  var container = document.getElementById('mynetwork');

                  

                  // parsing and collecting nodes and edges from the python
                  nodes = new vis.DataSet([{"betweenness_centrality": 1903.4328227086241, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24615384615384617, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n26", "label": "Enterococcus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Enterococcus", "pagerank_centrality": 0.9958400445140728, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:8427810\u0027, \u0027PMID:9103648\u0027]"}, {"betweenness_centrality": 11335.333778289832, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.2689075630252101, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 82.0, "description": "A trophic process during which a living entity acquires food and energy by consuming another organism or organism part, living or dead.", "font": {"size": 12}, "id": "n556", "iri": "http://purl.obolibrary.org/obo/ECOCORE_00000011", "label": "Heterotrophy", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Heterotrophy", "pagerank_centrality": 4.810868420146474, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 11, "synonym": "", "xref": ""}, {"betweenness_centrality": 19402.53922432753, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.275777162203755, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 100.0, "description": "The chemical reactions and pathways involving carbohydrates, any of a group of organic compounds based of the general formula Cx(H2O)y. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n938", "iri": "http://purl.obolibrary.org/obo/GO_0005975 ", "label": "Carbohydrate metabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Carbohydrate metabolic process", "pagerank_centrality": 6.027438888431428, "providedBy": "MiDAS", "shape": "dot", "size": 14, "synonym": "carbohydrate metabolism", "xref": ""}, {"betweenness_centrality": 49411.49917747123, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.29797140006651146, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 108.0, "description": "The anaerobic enzymatic conversion of organic compounds, especially carbohydrates, coupling the oxidation and reduction of NAD/H and the generation of adenosine triphosphate (ATP). During fermentation, organic compounds act as both the electron acceptor and donor. Organic molecules, such as sugars and amino acids, are utilized as the energy and carbon source. In the absence of, or the inability to utilize, external electron acceptors, organic molecules act as the electron acceptor to balance the redox of the cell. Common by-products include volatile fatty acids, alcohols, hydrogen and CO2.", "font": {"size": 12}, "id": "n557", "iri": "http://purl.obolibrary.org/obo/GO_0006113", "label": "Fermentation", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Fermentation", "pagerank_centrality": 5.8841757772761865, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 24, "synonym": "", "xref": "MetaCyc:FERMENTATION-PWY|Wikipedia:Fermentation_(biochemistry)"}, {"betweenness_centrality": 3905.6939534506137, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.2501395868230039, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 54.0, "description": "The chemical reactions and pathways involving amino acids, carboxylic acids containing one or more amino groups. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n939", "iri": "http://purl.obolibrary.org/obo/GO_0006520 ", "label": "Amino acid metabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Amino acid metabolic process", "pagerank_centrality": 3.1025410816790253, "providedBy": "MiDAS", "shape": "dot", "size": 9, "synonym": "cellular amino acid metabolic process|cellular amino acid metabolism|amino acid and derivative metabolism|cellular amino acid and derivative metabolic process", "xref": ""}, {"betweenness_centrality": 2701.631327657949, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2540402608449107, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n27", "label": "Streptococcus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Streptococcus", "pagerank_centrality": 1.261621730958828, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10555340\u0027, \u0027PMID:14657115\u0027, \u0027PMID:1720654\u0027, \u0027PMID:19620365\u0027, \u0027PMID:19880633\u0027, \u0027PMID:7537076\u0027, \u0027PMID:8995803\u0027]"}, {"betweenness_centrality": 2574.552967481923, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2571018651362984, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n33", "label": "Desulforhabdus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Desulforhabdus", "pagerank_centrality": 0.9807093875054327, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10939678\u0027, \u0027PMID:8588734\u0027, \u0027PMID:9336942\u0027]"}, {"betweenness_centrality": 3027.708642482462, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.24434142350695393, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 26.0, "description": "The pathways by which inorganic sulfate is processed and incorporated into sulfated compounds. These organisms utilize sulfate as a terminal electron acceptor producing hydrogen sulfide. A variety of sulfate reducers have been described, being able to degrade a range of organic compounds to acetate or completely to CO2. A variety of alternate electron acceptors have also been demonstrated, including Fe(III), nitrate and organics (fermentation). Therefore, a high abundance does not always reflect high levels of sulfate reduction. They are thought not to degrade complex polymers and rely on the activities of the hydrolysers and fermenters for a supply of carbon. The abundance and activity of these organisms is not favorable in wastewater treatment and anaerobic digestion as they compete with the methanogens for substrate, giving a lower methane yield, and hydrogen sulfide is toxic, odorous and associated with bio-corrosion problems in sewer systems and treatment facilities.", "font": {"size": 12}, "id": "n541", "iri": "http://purl.obolibrary.org/obo/GO_0000103", "label": "Sulfate assimilation", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Sulfate assimilation", "pagerank_centrality": 1.3140086272151603, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 8, "synonym": "sulphate assimilation|sulfate assimilation|phosphoadenylyl sulfate reduction by an oxidoreductase|acting on sulfur group of donors|NAD or NADP as acceptor|sulphate assimilation|phosphoadenylyl sulphate reduction by an oxidoreductase|acting on sulphur group of donors|NAD or NADP as acceptor", "xref": ""}, {"betweenness_centrality": 1567.7230980608372, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24561403508771928, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n37", "label": "Serratia", "labels": ":Genus", "metabolic_cluster": 114, "name": "Serratia", "pagerank_centrality": 0.7888122281190322, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Serratia\" EXACT scientific_name []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 3114.3282886051315, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.26345192590414584, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n39", "label": "Pseudomonas", "labels": ":Genus", "metabolic_cluster": 114, "name": "Pseudomonas", "pagerank_centrality": 1.1712054406630654, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"\\\"Chlorobacterium\\\" Guillebeau 1890, nom. rejic. Opin. 6 (not \\\"Chlorobacterium\\\" Lauterborn 1916)\" RELATED synonym []\u0027, \u0027\"Liquidomonas\" RELATED synonym []\u0027, \u0027\"Loefflerella\" RELATED synonym []\u0027, \u0027\"RNA similarity group I\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10758879\u0027, \u0027PMID:10939664\u0027, \u0027PMID:15950132\u0027, \u0027PMID:18048745\u0027, \u0027PMID:23918787\u0027, \u0027PMID:7727274\u0027, \u0027PMID:9103607\u0027]"}, {"betweenness_centrality": 17721.11500321186, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.29176164115923153, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 70.0, "description": "The reduction of nitrate to dinitrogen by four reactions; each intermediate is transformed to the next lower oxidation state; also part of cellular bioenergetics; the nitrogen compounds can serve as terminal acceptors for electron transport phosphorylation in place of oxygen. The denitrifiers are usually facultative-anaerobic heterotrophic organisms utilizing nitrate and/or nitrite as electron acceptors in the absence of oxygen. Denitrification is the final step of nitrogen removal from wastewater and generally occurs under anaerobic conditions. Nitrate is available from the oxidation of ammonia by the aerobic activities of the nitrifiers (AOB and NOB). Denitrification involves the sequential reduction of nitrate to inert dinitrogen gas, via several intermediates (nitrite, nitric oxide (gas) and nitrous oxide (gas)). Not all organisms involved in denitrification possess all the genes for reduction of nitrate all the way to dinitrogen gas. As such, for MiDAS profiles, the nitrite reducing organisms are selected to represent the denitrifiers; given denitrification is defined as \u201cthe dissimilatory transformation of nitrate or nitrite to a gas species concomitant with energy conservation\u201d and not all organisms capable of nitrate reduction can reduce nitrite.", "font": {"size": 12}, "id": "n540", "iri": "http://purl.obolibrary.org/obo/GO_0019333", "label": "Denitrification pathway", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Denitrification pathway", "pagerank_centrality": 2.5029258657669415, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 13, "synonym": "", "xref": ""}, {"betweenness_centrality": 1707.5318008945312, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24730886006072317, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 10.0, "font": {"size": 12}, "id": "n46", "label": "Bacteroides", "labels": ":Genus", "metabolic_cluster": 114, "name": "Bacteroides", "pagerank_centrality": 0.6925466171890364, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Capsularis\" RELATED synonym []\u0027, \u0027\"Ristella\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 2392.150227877559, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24854368932038834, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n49", "label": "Ruminococcus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Ruminococcus", "pagerank_centrality": 1.4013607614173558, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 217.86533206001948, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.23104693140794225, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 6.0, "description": "The chemical reactions and pathways resulting in the formation of acetate, the anion of acetic acid. Acetogenesis is routinely applied to broadly describe the biological synthesis of acetate. However, this definition is somewhat misleading as several metabolic strategies result in acetate production. MiDAS therefore applies the strict microbiological definition. True acetogens employ the reductive acetyl-CoA pathway for the synthesis of acetyl-CoA that can be converted to acetate or utilized as a source of carbon for biomass production. In this pathway, hydrogen and CO2 are utilized as an electron donor and acceptor, respectively. Acetogens often utilize alternative donors and acceptors to supplement or replace the use of the pathway for energy or carbon supply depending on growth conditions. Commonly, the pathway operates in conjunction with fermentative pathways, operating to replace generated reducing equivalents in the reduction of CO2. Importantly, organisms possessing the reductive acetyl-CoA pathway can reverse the pathway to oxidize acetate to CO2\u00a0and H2 in concert with hydrogenotrophic methanogens. This is known as syntrophic acetate oxidation (SAO) as it requires the methanogens to remove H2 to make the pathway thermodynamically favorable.", "font": {"size": 12}, "id": "n547", "iri": "http://purl.obolibrary.org/obo/GO_0019413", "label": "Acetate biosynthetic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Acetate biosynthetic process", "pagerank_centrality": 0.4647342854582864, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 8, "synonym": "acetate anabolism|acetate biosynthesis|acetate formation|acetate synthesis", "xref": ""}, {"betweenness_centrality": 115.30617143765645, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.2216176106851348, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 8.0, "description": "The process in which a multicellular organism, a unicellular organism or a group of unicellular organisms grow in a threadlike, filamentous shape. Excessive growth of organisms with a filamentous morphology has long been associated with the operational problem of bulking. In this condition, interfloc bridging by the filaments impedes adequate settling of the biomass, resulting in loss of solids with the treated effluent", "font": {"size": 12}, "id": "n940", "iri": "http://purl.obolibrary.org/obo/GO_0030447 ", "label": "Filamentous growth", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Filamentous growth", "pagerank_centrality": 0.6268255335585204, "providedBy": "MiDAS", "shape": "dot", "size": 8, "synonym": "", "xref": ""}, {"betweenness_centrality": 2139.044728921496, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23886963476406292, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 10.0, "font": {"size": 12}, "id": "n54", "label": "Eubacterium", "labels": ":Genus", "metabolic_cluster": 114, "name": "Eubacterium", "pagerank_centrality": 0.7230023081480631, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:8863439\u0027]"}, {"betweenness_centrality": 1994.0307614450326, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2401500938086304, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n75", "label": "Bowdeniella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Bowdeniella", "pagerank_centrality": 0.7022856267157686, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Actinocladothrix\" RELATED synonym []\u0027, \u0027\"Discomyces\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:30186281\u0027]"}, {"betweenness_centrality": 2126.73886484537, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.25211029825548675, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 14.0, "font": {"size": 12}, "id": "n79", "label": "Prevotella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Prevotella", "pagerank_centrality": 0.9307112539421857, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16957124\u0027, \u0027PMID:22199207\u0027, \u0027PMID:2223612\u0027, \u0027PMID:7547307\u0027, \u0027PMID:9542075\u0027]"}, {"betweenness_centrality": 2024.4191967612485, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23880597014925373, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n80", "label": "Anaerocella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Anaerocella", "pagerank_centrality": 0.8940173021794391, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:23337575\u0027]"}, {"betweenness_centrality": 1828.168302608383, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2501395868230039, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n90", "label": "Lactobacillus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Lactobacillus", "pagerank_centrality": 0.976588622085282, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Paralactobacillus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10826783\u0027, \u0027PMID:18048724\u0027, \u0027PMID:19060088\u0027, \u0027PMID:19244449\u0027, \u0027PMID:21296926\u0027, \u0027PMID:21724957\u0027, \u0027PMID:29915113\u0027]"}, {"betweenness_centrality": 1828.7860473990372, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2451436388508892, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n117", "label": "Syntrophobacter", "labels": ":Genus", "metabolic_cluster": 114, "name": "Syntrophobacter", "pagerank_centrality": 0.8052797228147336, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 3354.9997008526298, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2553434026788259, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n130", "label": "Thermanaerovibrio", "labels": ":Genus", "metabolic_cluster": 114, "name": "Thermanaerovibrio", "pagerank_centrality": 1.0507437376232758, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10425752\u0027, \u0027PMID:10843074\u0027, \u0027PMID:24501645\u0027]"}, {"betweenness_centrality": 2903.9308787663413, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24034334763948498, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n131", "label": "Alteracholeplasma", "labels": ":Genus", "metabolic_cluster": 114, "name": "Alteracholeplasma", "pagerank_centrality": 0.7091306226665816, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Sapromyces\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:15653928\u0027, \u0027PMID:16350067\u0027, \u0027PMID:5487182\u0027]"}, {"betweenness_centrality": 2731.6602571078874, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24150943396226415, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 10.0, "font": {"size": 12}, "id": "n183", "label": "Exilispira", "labels": ":Genus", "metabolic_cluster": 114, "name": "Exilispira", "pagerank_centrality": 0.7440185040395001, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:18842837\u0027]"}, {"betweenness_centrality": 1362.6382307328474, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2260343087790111, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n341", "label": "Anaerophaga thermohalophila", "labels": ":Species", "metabolic_cluster": 114, "name": "Anaerophaga thermohalophila", "pagerank_centrality": 1.0154618038216137, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11837300\u0027]"}, {"betweenness_centrality": 2642.4227943200917, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2606166375799884, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n343", "label": "Alicyclobacillus acidocaldarius", "labels": ":Species", "metabolic_cluster": 114, "name": "Alicyclobacillus acidocaldarius", "pagerank_centrality": 1.3046473948122235, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Bacillus acidocaldarius\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:1374624\u0027, \u0027PMID:17220431\u0027]"}, {"betweenness_centrality": 8093.664634033452, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.27921470863197256, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 50.0, "font": {"size": 12}, "id": "n344", "label": "Bacillus cereus", "labels": ":Species", "metabolic_cluster": 114, "name": "Bacillus cereus", "pagerank_centrality": 2.383316910827646, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 10, "synonym": "[\u0027\"Bacillus cereus Frankland and Frankland 1887 (Approved Lists 1980)\" RELATED synonym []\u0027, \u0027\"Bacillus endorhythmos\" RELATED synonym []\u0027, \u0027\"Bacillus medusa\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:9542081\u0027]"}, {"betweenness_centrality": 1892.7212514214614, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2557807593491293, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 28.0, "font": {"size": 12}, "id": "n345", "label": "Bacillus coahuilensis", "labels": ":Species", "metabolic_cluster": 114, "name": "Bacillus coahuilensis", "pagerank_centrality": 1.5021766335828848, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:18398195\u0027]"}, {"betweenness_centrality": 1732.7479849090787, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.25440090857467346, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n346", "label": "Priestia flexa", "labels": ":Species", "metabolic_cluster": 114, "name": "Priestia flexa", "pagerank_centrality": 1.2827834602918515, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Bacillus flexus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:3246588\u0027]"}, {"betweenness_centrality": 1510.795021269975, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24053691275167785, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n354", "label": "Solobacterium moorei", "labels": ":Species", "metabolic_cluster": 114, "name": "Solobacterium moorei", "pagerank_centrality": 1.2536404342317329, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"unclassified Clostridium group RCA59\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:38014937\u0027]"}, {"betweenness_centrality": 1135.985261668016, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23284823284823286, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 20.0, "font": {"size": 12}, "id": "n355", "label": "Desulforhabdus amnigena", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulforhabdus amnigena", "pagerank_centrality": 1.2171383554333943, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Desulforhabdus amnigenus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10939678\u0027, \u0027PMID:8588734\u0027, \u0027PMID:9336942\u0027]"}, {"betweenness_centrality": 1800.2315034593069, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23218450375745012, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n356", "label": "Desulfovirga adipica", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulfovirga adipica", "pagerank_centrality": 1.405572332513615, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10758870\u0027]"}, {"betweenness_centrality": 561.5210636905871, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.18764397905759161, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n20", "label": "Anaerophaga", "labels": ":Genus", "metabolic_cluster": 114, "name": "Anaerophaga", "pagerank_centrality": 0.48566511095139914, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11837300\u0027]"}, {"betweenness_centrality": 704.0290390512633, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2101313320825516, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n22", "label": "Alicyclobacillus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Alicyclobacillus", "pagerank_centrality": 0.4671540247185428, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13130044\u0027, \u0027PMID:1374624\u0027, \u0027PMID:15774689\u0027]"}, {"betweenness_centrality": 11.455952173444613, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.2248995983935743, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 6.0, "description": "The chemical reactions and pathways resulting in the breakdown of chitin, a linear polysaccharide consisting of beta-(1-\u003e4)-linked N-acetyl-D-glucosamine residues.", "font": {"size": 12}, "id": "n535", "iri": "http://purl.obolibrary.org/obo/GO_0006032", "label": "Chitin catabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Chitin catabolic process", "pagerank_centrality": 0.3794219075837636, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "beta-1,4-linked N-acetylglucosamine catabolic process|beta-1,4-linked N-acetylglucosamine catabolism|chitin breakdown|chitin catabolism|chitin degradation", "xref": ""}, {"betweenness_centrality": 599.6859764658478, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2229965156794425, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 6.0, "font": {"size": 12}, "id": "n23", "label": "Bacillus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Bacillus", "pagerank_centrality": 0.4806942584717906, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Bacillus\" EXACT scientific_name []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 2064.962823437401, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.25175611126720987, "cluster": "1", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 14.0, "font": {"size": 12}, "id": "n350", "label": "Clostridium tyrobutyricum", "labels": ":Species", "metabolic_cluster": 114, "name": "Clostridium tyrobutyricum", "pagerank_centrality": 0.8540411014893602, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 120.61433875507183, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2065944201060641, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n24", "label": "Priestia", "labels": ":Genus", "metabolic_cluster": 114, "name": "Priestia", "pagerank_centrality": 0.3999757141267944, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 2298.85196008492, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2545454545454545, "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 14.0, "font": {"size": 12}, "id": "n348", "label": "Enterococcus cecorum", "labels": ":Species", "metabolic_cluster": 114, "name": "Enterococcus cecorum", "pagerank_centrality": 0.8054952327800111, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Streptococcus cecorum\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 1557.0078248771733, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2631424375917768, "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n424", "label": "Streptococcus intermedius", "labels": ":Species", "metabolic_cluster": 114, "name": "Streptococcus intermedius", "pagerank_centrality": 1.180750954457315, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:1995029\u0027, \u0027PMID:9542082\u0027]"}, {"betweenness_centrality": 2731.887189670189, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.26987951807228916, "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 20.0, "font": {"size": 12}, "id": "n427", "label": "Clostridium saccharoperbutylacetonicum", "labels": ":Species", "metabolic_cluster": 114, "name": "Clostridium saccharoperbutylacetonicum", "pagerank_centrality": 1.1365514925207012, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:11760952\u0027, \u0027PMID:7547288\u0027, \u0027PMID:9103631\u0027]"}, {"betweenness_centrality": 1958.6907248059242, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2066420664206642, "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n497", "label": "Prototheca moriformis", "labels": ":Species", "metabolic_cluster": 114, "name": "Prototheca moriformis", "pagerank_centrality": 0.7468016118755109, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Prototheca ulmea\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 1830.4301071609418, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23339411305027352, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n28", "label": "Clostridium", "labels": ":Genus", "metabolic_cluster": 114, "name": "Clostridium", "pagerank_centrality": 0.9637791446669762, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Anaerobacter\" RELATED synonym []\u0027, \u0027\"Anaerobacter Duda et al. 1996\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:26643615\u0027, \u0027PMID:27488356\u0027]"}, {"betweenness_centrality": 210.50770737054708, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.1977052074139453, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n32", "label": "Solobacterium", "labels": ":Genus", "metabolic_cluster": 114, "name": "Solobacterium", "pagerank_centrality": 0.44506900331367144, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Clostridium cluster XVI\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 82.25291745947908, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19210977701543738, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n34", "label": "Desulfovirga", "labels": ":Genus", "metabolic_cluster": 114, "name": "Desulfovirga", "pagerank_centrality": 0.42839546716212246, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10758870\u0027]"}, {"betweenness_centrality": 2629.4914541000994, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.26619132501485443, "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 34.0, "font": {"size": 12}, "id": "n359", "label": "Serratia marcescens", "labels": ":Species", "metabolic_cluster": 114, "name": "Serratia marcescens", "pagerank_centrality": 1.5871928777845599, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Bacillus marcescens\" RELATED synonym []\u0027, \u0027\"Serratia marcescens subsp. marcescens\" RELATED synonym []\u0027, \u0027\"Serratia marcescens subsp. sakuensis\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:12656181\u0027, \u0027PMID:31992497\u0027]"}, {"betweenness_centrality": 346.17002477222724, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.25993617638526256, "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n365", "label": "Pseudomonas viridiflava", "labels": ":Species", "metabolic_cluster": 114, "name": "Pseudomonas viridiflava", "pagerank_centrality": 0.7118545686448832, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Phytomonas viridiflava\" RELATED synonym []\u0027, \u0027\"Pseudomonas syringae group genomosp. 6\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10319466\u0027]"}, {"betweenness_centrality": 2326.640074486439, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2658753709198813, "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n372", "label": "Bacteroides uniformis", "labels": ":Species", "metabolic_cluster": 114, "name": "Bacteroides uniformis", "pagerank_centrality": 1.1479104393861146, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16559622\u0027, \u0027PMID:28066339\u0027]"}, {"betweenness_centrality": 159.95444829478828, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23566543924250394, "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 16.0, "font": {"size": 12}, "id": "n384", "label": "Eubacterium cylindroides", "labels": ":Species", "metabolic_cluster": 114, "name": "Eubacterium cylindroides", "pagerank_centrality": 0.793410277415161, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Bacteroides cylindroides\" RELATED synonym []\u0027, \u0027\"Eubacterium cylindroides (Rocchi 1908) Holdeman and Moore 1970\" RELATED synonym []\u0027, \u0027\"Pseudobacterium cylindroides\" RELATED synonym []\u0027, \u0027\"Ristella cylindroides\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:25180093\u0027]"}, {"betweenness_centrality": 404.11126704006745, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.25189766657295476, "cluster": "4", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n376", "label": "Clostridium gasigenes", "labels": ":Species", "metabolic_cluster": 114, "name": "Clostridium gasigenes", "pagerank_centrality": 0.6024565929157929, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10826793\u0027]"}, {"betweenness_centrality": 2310.272059884944, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24347826086956523, "cluster": "2", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n386", "label": "Alcaligenes faecalis", "labels": ":Species", "metabolic_cluster": 114, "name": "Alcaligenes faecalis", "pagerank_centrality": 1.213345482952376, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Bacillus faecalis alcaligenes\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16094869\u0027, \u0027PMID:4074238\u0027]"}, {"betweenness_centrality": 214.8605217790144, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20053715308863027, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n56", "label": "Alcaligenes", "labels": ":Genus", "metabolic_cluster": 114, "name": "Alcaligenes", "pagerank_centrality": 0.4167387842440145, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 942.7066570692366, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.238107892638852, "cluster": "6", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 10.0, "font": {"size": 12}, "id": "n407", "label": "Bowdeniella nasicola", "labels": ":Species", "metabolic_cluster": 114, "name": "Bowdeniella nasicola", "pagerank_centrality": 0.7173012380846049, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Actinomyces nasicola\" RELATED synonym []\u0027, \u0027\"Bowdeniella nasicola corrig. (Hall et al. 2003) Nouioui et al. 2018\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13130031\u0027]"}, {"betweenness_centrality": 5994.180262571959, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24785615491009683, "cluster": "6", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n444", "label": "Advenella mimigardefordensis", "labels": ":Species", "metabolic_cluster": 114, "name": "Advenella mimigardefordensis", "pagerank_centrality": 1.6704483464923943, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Tetrathiobacter mimigardefordensis\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:16738107\u0027, \u0027PMID:19567588\u0027]"}, {"betweenness_centrality": 3807.1731678540805, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22025565388397247, "cluster": "6", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n491", "label": "Mononchus truncatus", "labels": ":Species", "metabolic_cluster": 114, "name": "Mononchus truncatus", "pagerank_centrality": 1.0963807100208116, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 933.7648234723944, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.17582417582417584, "cluster": "7", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 6.0, "font": {"size": 12}, "id": "n408", "label": "Flexivirga lutea", "labels": ":Species", "metabolic_cluster": 114, "name": "Flexivirga lutea", "pagerank_centrality": 0.7965832434798575, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:27295948\u0027]"}, {"betweenness_centrality": 2030.2847938275302, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20621403912543154, "cluster": "7", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 10.0, "font": {"size": 12}, "id": "n412", "label": "Anaerocella delicata", "labels": ":Species", "metabolic_cluster": 114, "name": "Anaerocella delicata", "pagerank_centrality": 1.126924189242675, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:23337575\u0027]"}, {"betweenness_centrality": 4678.158272216349, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20550458715596331, "cluster": "7", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n472", "label": "Paratrimastix pyriformis", "labels": ":Species", "metabolic_cluster": 114, "name": "Paratrimastix pyriformis", "pagerank_centrality": 0.8849451968171875, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Trimastix pyriformis\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:1\u0027, \u0027PMID:26312987\u0027]"}, {"betweenness_centrality": 190.6549247411158, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.15368782161234992, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n76", "label": "Flexivirga", "labels": ":Genus", "metabolic_cluster": 114, "name": "Flexivirga", "pagerank_centrality": 0.634510820108589, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:21811262\u0027, \u0027PMID:27295948\u0027, \u0027PMID:27902309\u0027]"}, {"betweenness_centrality": 3695.367908723486, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2250690781210751, "cluster": "7", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n409", "label": "Haematomicrobium sanguinis", "labels": ":Species", "metabolic_cluster": 114, "name": "Haematomicrobium sanguinis", "pagerank_centrality": 0.8170880137056613, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Arthrobacter sanguinis\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:18650355\u0027, \u0027PMID:28032539\u0027]"}, {"betweenness_centrality": 168.2155478643431, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19264674263599227, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n77", "label": "Haematomicrobium", "labels": ":Genus", "metabolic_cluster": 114, "name": "Haematomicrobium", "pagerank_centrality": 0.5168407359935868, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:28032539\u0027]"}, {"betweenness_centrality": 230.9172575872855, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2183767974652693, "cluster": "6", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 6.0, "font": {"size": 12}, "id": "n411", "label": "Prevotella ihumii", "labels": ":Species", "metabolic_cluster": 114, "name": "Prevotella ihumii", "pagerank_centrality": 0.5344277970375858, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 5398.191347259074, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2356034709439916, "cluster": "7", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 14.0, "font": {"size": 12}, "id": "n428", "label": "Oxobacter pfennigii", "labels": ":Species", "metabolic_cluster": 114, "name": "Oxobacter pfennigii", "pagerank_centrality": 1.362743136569698, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Clostridium pfennigii\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:7981107\u0027]"}, {"betweenness_centrality": 3779.404011436226, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22216712124969007, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n418", "label": "Thermomicrobium carboxidum", "labels": ":Species", "metabolic_cluster": 114, "name": "Thermomicrobium carboxidum", "pagerank_centrality": 1.5834109415167759, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:24814334\u0027, \u0027PMID:26374291\u0027]"}, {"betweenness_centrality": 2603.4254722089745, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23944414751469803, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n429", "label": "Serpentinicella alkaliphila", "labels": ":Species", "metabolic_cluster": 114, "name": "Serpentinicella alkaliphila", "pagerank_centrality": 1.4134205369814292, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:27499124\u0027]"}, {"betweenness_centrality": 1656.1768160858676, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22810590631364563, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n451", "label": "Desulfonatronum lacustre", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulfonatronum lacustre", "pagerank_centrality": 1.5939744523142547, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 1838.9094679827294, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23925233644859814, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n452", "label": "Desulfonatronum thiodismutans", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulfonatronum thiodismutans", "pagerank_centrality": 1.367254769956517, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13130014\u0027]"}, {"betweenness_centrality": 10654.04824675038, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2732540408661177, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 36.0, "font": {"size": 12}, "id": "n453", "label": "Desulfocurvus vexinensis", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulfocurvus vexinensis", "pagerank_centrality": 2.0536238724145552, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19643880\u0027]"}, {"betweenness_centrality": 1791.6124003540388, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23272727272727273, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 26.0, "font": {"size": 12}, "id": "n456", "label": "Syntrophobacter pfennigii", "labels": ":Species", "metabolic_cluster": 114, "name": "Syntrophobacter pfennigii", "pagerank_centrality": 1.6488247681482495, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Syntrophobacterium pfennigii\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 5268.103168000534, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2557077625570776, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 26.0, "font": {"size": 12}, "id": "n461", "label": "Rickettsiella grylli", "labels": ":Species", "metabolic_cluster": 114, "name": "Rickettsiella grylli", "pagerank_centrality": 1.541707232372107, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:9336939\u0027]"}, {"betweenness_centrality": 3542.9578532116616, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24262117519631735, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 26.0, "font": {"size": 12}, "id": "n463", "label": "Methylococcus capsulatus", "labels": ":Species", "metabolic_cluster": 114, "name": "Methylococcus capsulatus", "pagerank_centrality": 1.7242965392940846, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:5937247\u0027]"}, {"betweenness_centrality": 473.8638810958381, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22416812609457093, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 20.0, "font": {"size": 12}, "id": "n471", "label": "Alteracholeplasma palmae", "labels": ":Species", "metabolic_cluster": 114, "name": "Alteracholeplasma palmae", "pagerank_centrality": 1.2708717087085106, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Acholeplasma palmae\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:7981098\u0027]"}, {"betweenness_centrality": 5385.287288399086, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.223888055972014, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n475", "label": "Pseudotrichomonas keilini", "labels": ":Species", "metabolic_cluster": 114, "name": "Pseudotrichomonas keilini", "pagerank_centrality": 1.4182258464129358, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 2059.446734966266, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.1838703057664683, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n86", "label": "Thermomicrobium", "labels": ":Genus", "metabolic_cluster": 114, "name": "Thermomicrobium", "pagerank_centrality": 0.5829483402537748, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 4939.714066944847, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.27159745377387084, "cluster": "6", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 20.0, "font": {"size": 12}, "id": "n422", "label": "Lactobacillus casei", "labels": ":Species", "metabolic_cluster": 114, "name": "Lactobacillus casei", "pagerank_centrality": 1.065339066664582, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "[\u0027\"Bacillus a\" RELATED synonym []\u0027, \u0027\"Bacillus casei a\" RELATED synonym []\u0027, \u0027\"Bacterium casei a\" RELATED synonym []\u0027, \u0027\"Caseobacterium vulgare\" RELATED synonym []\u0027, \u0027\"Lactobacillus casei\" RELATED synonym []\u0027, \u0027\"Lactobacillus casei subsp. casei\" RELATED synonym []\u0027, \u0027\"Lactobacterium casei\" RELATED synonym []\u0027, \u0027\"Streptobacterium casei\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10758849\u0027, \u0027PMID:11760954\u0027, \u0027PMID:11837314\u0027, \u0027PMID:18599731\u0027, \u0027PMID:8573516\u0027, \u0027PMID:8995801\u0027]"}, {"betweenness_centrality": 381.27368809033936, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2236645032451323, "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 10.0, "font": {"size": 12}, "id": "n426", "label": "Clostridium longisporum", "labels": ":Species", "metabolic_cluster": 114, "name": "Clostridium longisporum", "pagerank_centrality": 0.7078900670162239, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"\u0027Clostridium longisporum\u0027\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:2774798\u0027]"}, {"betweenness_centrality": 480.47326640815237, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22422422422422422, "cluster": "3", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n470", "label": "Thermanaerovibrio acidaminovorans", "labels": ":Species", "metabolic_cluster": 114, "name": "Thermanaerovibrio acidaminovorans", "pagerank_centrality": 0.5977414254568137, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Selenomonas acidaminovorans\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:10425752\u0027, \u0027PMID:24501645\u0027]"}, {"betweenness_centrality": 56.044197430637, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.20954162768942938, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 6.0, "description": "The chemical reactions and pathways resulting in the breakdown of cellulose, a linear beta1-4 glucan of molecular mass 50-400 kDa with the pyranose units in the -4C1 conformation.", "font": {"size": 12}, "id": "n537", "iri": "http://purl.obolibrary.org/obo/GO_0030245", "label": "Cellulose catabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Cellulose catabolic process", "pagerank_centrality": 0.4233820395055269, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "cellulose breakdown|cellulose catabolism|cellulose degradation", "xref": ""}, {"betweenness_centrality": 221.15689463273222, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19402338674750974, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n91", "label": "Oxobacter", "labels": ":Genus", "metabolic_cluster": 114, "name": "Oxobacter", "pagerank_centrality": 0.5004563603015034, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:7981107\u0027]"}, {"betweenness_centrality": 1025.83976815532, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19627601314348303, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n92", "label": "Serpentinicella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Serpentinicella", "pagerank_centrality": 0.520447600034578, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:27499124\u0027]"}, {"betweenness_centrality": 2911.614570804651, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20053715308863027, "cluster": "6", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 6.0, "font": {"size": 12}, "id": "n485", "label": "Cyniclomyces guttulatus", "labels": ":Species", "metabolic_cluster": 114, "name": "Cyniclomyces guttulatus", "pagerank_centrality": 0.7109595494856538, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Cryptococcus guttulatus\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 3359.7382069695536, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.26810293237582283, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 22.0, "description": "", "font": {"size": 12}, "id": "n589", "iri": "http://purl.obolibrary.org/obo/CHEBI_22599", "label": "arabinose", "labels": ":ChemicalEntity", "metabolic_cluster": 114, "name": "arabinose", "pagerank_centrality": 1.0236235921807553, "providedBy": "KG-Microbe", "shape": "dot", "size": 9, "synonym": "Ara|arabino-pentose|arabinose", "xref": "CAS:147-81-9|HMDB:HMDB0029942|Wikipedia:Arabinose"}, {"betweenness_centrality": 349.58325279246355, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20299048482102403, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n107", "label": "Advenella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Advenella", "pagerank_centrality": 0.44125208658569115, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Tetrathiobacter\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:15653883\u0027, \u0027PMID:16166666\u0027, \u0027PMID:19567588\u0027]"}, {"betweenness_centrality": 849.1807882902475, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19640508548882069, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 6.0, "font": {"size": 12}, "id": "n113", "label": "Desulfonatronum", "labels": ":Genus", "metabolic_cluster": 114, "name": "Desulfonatronum", "pagerank_centrality": 0.6051422024374148, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 370.1309786982396, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2177929022848809, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n114", "label": "Desulfocurvus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Desulfocurvus", "pagerank_centrality": 0.43338394749007736, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19643880\u0027]"}, {"betweenness_centrality": 762.3984207759405, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.20842056292160968, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n122", "label": "Rickettsiella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Rickettsiella", "pagerank_centrality": 0.497571754175585, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Diplorickettsia\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13316619\u0027]"}, {"betweenness_centrality": 0.0, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.1952920662598082, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 2.0, "description": "The cellular metabolic process in which organic chemical compounds are synthesized from carbon-containing molecules and nutrients using energy obtained from the oxidation of inorganic compounds or methane.", "font": {"size": 12}, "id": "n543", "iri": "http://purl.obolibrary.org/obo/GO_0062097", "label": "Chemosynthesis", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Chemosynthesis", "pagerank_centrality": 0.2619679432118007, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "", "xref": ""}, {"betweenness_centrality": 887.6231404538909, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2004474272930649, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n123", "label": "Methylococcus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Methylococcus", "pagerank_centrality": 0.5292936599300598, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:5937247\u0027]"}, {"betweenness_centrality": 3387.826217153478, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.1751368256450352, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n132", "label": "Paratrimastix", "labels": ":Genus", "metabolic_cluster": 114, "name": "Paratrimastix", "pagerank_centrality": 0.8182766470850708, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027, \u0027PMID:26312987\u0027]"}, {"betweenness_centrality": 1902.9722590487481, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.18831441782261454, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 10.0, "font": {"size": 12}, "id": "n135", "label": "Pseudotrichomonas", "labels": ":Genus", "metabolic_cluster": 114, "name": "Pseudotrichomonas", "pagerank_centrality": 0.8847004134484601, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 2080.759091963841, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.16976127320954906, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n145", "label": "Cyniclomyces", "labels": ":Genus", "metabolic_cluster": 114, "name": "Cyniclomyces", "pagerank_centrality": 0.6821982693296392, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 2218.5558136793284, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.18353133961491191, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n150", "label": "Mononchus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Mononchus", "pagerank_centrality": 0.6193909612228679, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 1150.7743283136288, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.17462482946793997, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n156", "label": "Prototheca", "labels": ":Genus", "metabolic_cluster": 114, "name": "Prototheca", "pagerank_centrality": 0.5991573661320767, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:1\u0027]"}, {"betweenness_centrality": 487.32939279269533, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24574876577070762, "cluster": "4", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n524", "label": "Ruminococcus albus", "labels": ":Species", "metabolic_cluster": 114, "name": "Ruminococcus albus", "pagerank_centrality": 0.6973760332799261, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 0.5865145441463665, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.20832364566379913, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 4.0, "description": "The chemical reactions and pathways resulting in the breakdown of xylan, a polymer containing a beta-1,4-linked D-xylose backbone.", "font": {"size": 12}, "id": "n534", "iri": "http://purl.obolibrary.org/obo/GO_0045493", "label": "Xylan catabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Xylan catabolic process", "pagerank_centrality": 0.3037241306431739, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "xylan breakdown|xylan catabolism|xylan degradation", "xref": ""}, {"betweenness_centrality": 3293.3673211212263, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.27800186161960905, "cluster": "4", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 88.0, "font": {"size": 12}, "id": "n533", "label": "Exilispira thermophila", "labels": ":Species", "metabolic_cluster": 114, "name": "Exilispira thermophila", "pagerank_centrality": 2.5966869614090444, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:18842837\u0027]"}]);
                  edges = new vis.DataSet([{"color": "#11449cff", "font": {"size": 6}, "from": "n26", "hasattribute": "Positive", "id": "e2337", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n26", "hasattribute": "Positive", "id": "e2519", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n26", "hasattribute": "Positive", "id": "e2673", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n26", "hasattribute": "Positive", "id": "e2712", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n27", "hasattribute": "Positive", "id": "e2334", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n27", "hasattribute": "Positive", "id": "e2516", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n27", "hasattribute": "Positive", "id": "e2672", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_situ\u0027, \u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n27", "hasattribute": "Positive", "id": "e2709", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "hasattribute": "Positive", "id": "e2377", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n541", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "hasattribute": "Negative", "id": "e2544", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "hasattribute": "Negative", "id": "e2689", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n33", "hasattribute": "Negative", "id": "e2726", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n37", "hasattribute": "Positive", "id": "e2355", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n37", "hasattribute": "Positive", "id": "e2545", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n37", "hasattribute": "Positive", "id": "e2690", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n37", "hasattribute": "Positive", "id": "e2727", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "hasattribute": "Positive", "id": "e2345", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "hasattribute": "Positive", "id": "e2483", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n540", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "hasattribute": "Positive", "id": "e2529", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n39", "hasattribute": "Positive", "id": "e2718", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n46", "hasattribute": "Positive", "id": "e2542", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n46", "hasattribute": "Positive", "id": "e2687", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "hasattribute": "Negative", "id": "e2332", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "hasattribute": "Positive", "id": "e2485", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n547", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "hasattribute": "Negative", "id": "e2487", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n940", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "hasattribute": "Positive", "id": "e2514", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "hasattribute": "Positive", "id": "e2671", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n49", "hasattribute": "Negative", "id": "e2708", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n54", "hasattribute": "Positive", "id": "e2533", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n54", "hasattribute": "Positive", "id": "e2680", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n54", "hasattribute": "Positive", "id": "e2721", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n75", "hasattribute": "Positive", "id": "e2536", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n75", "hasattribute": "Positive", "id": "e2683", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n79", "hasattribute": "Negative", "id": "e2346", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n79", "hasattribute": "Positive", "id": "e2531", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n79", "hasattribute": "Positive", "id": "e2678", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n79", "hasattribute": "Positive", "id": "e2719", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n80", "hasattribute": "Negative", "id": "e2365", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n80", "hasattribute": "Negative", "id": "e2555", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n80", "hasattribute": "Positive", "id": "e2698", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n80", "hasattribute": "Positive", "id": "e2731", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n90", "hasattribute": "Positive", "id": "e2348", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n90", "hasattribute": "Positive", "id": "e2535", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n90", "hasattribute": "Positive", "id": "e2682", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n90", "hasattribute": "Positive", "id": "e2722", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n117", "hasattribute": "Negative", "id": "e2338", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n117", "hasattribute": "Positive", "id": "e2376", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n541", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n117", "hasattribute": "Negative", "id": "e2520", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n117", "hasattribute": "Positive", "id": "e2674", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n130", "hasattribute": "Negative", "id": "e2354", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n556", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n130", "hasattribute": "Positive", "id": "e2543", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n130", "hasattribute": "Positive", "id": "e2688", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n130", "hasattribute": "Positive", "id": "e2725", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n131", "hasattribute": "Positive", "id": "e2534", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n131", "hasattribute": "Positive", "id": "e2681", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n183", "hasattribute": "Negative", "id": "e2526", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n183", "hasattribute": "Negative", "id": "e2716", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n939", "width": 1}, {"associationSlot": 0.255066956698569, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n341", "id": "e118", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n343", "width": 1}, {"associationSlot": 0.255066956698428, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n341", "id": "e119", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n344", "width": 1}, {"associationSlot": 0.27082957409849, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n341", "id": "e120", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n345", "width": 1}, {"associationSlot": 0.255066956698406, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n341", "id": "e121", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n346", "width": 1}, {"associationSlot": 0.25506695669848, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n341", "id": "e122", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n354", "width": 1}, {"associationSlot": 0.255066956698562, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n341", "id": "e123", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n355", "width": 1}, {"associationSlot": 0.25506695669837, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n341", "id": "e124", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n356", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n341", "id": "e3102", "label": "SUBCLASS_OF", "to": "n20", "width": 1}, {"associationSlot": 0.255066956698482, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n343", "id": "e147", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n344", "width": 1}, {"associationSlot": 0.270829574098495, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n343", "id": "e148", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n345", "width": 1}, {"associationSlot": 0.255066956698536, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n343", "id": "e149", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n346", "width": 1}, {"associationSlot": 0.25506695669845, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n343", "id": "e150", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n354", "width": 1}, {"associationSlot": 0.255066956698415, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n343", "id": "e151", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n355", "width": 1}, {"associationSlot": 0.255066956698466, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n343", "id": "e152", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n356", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n343", "id": "e3104", "label": "SUBCLASS_OF", "to": "n22", "width": 1}, {"associationSlot": 0.270829574098487, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n344", "id": "e153", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n345", "width": 1}, {"associationSlot": 0.25506695669851, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n344", "id": "e154", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n346", "width": 1}, {"associationSlot": 0.255066956698371, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n344", "id": "e155", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n354", "width": 1}, {"associationSlot": 0.255066956698452, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n344", "id": "e156", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n355", "width": 1}, {"associationSlot": 0.255066956698424, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n344", "id": "e157", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n356", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n344", "id": "e2752", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n344", "id": "e2753", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n540", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n344", "id": "e2754", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n535", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n344", "id": "e3105", "label": "SUBCLASS_OF", "to": "n23", "width": 1}, {"associationSlot": 0.270829574098476, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n345", "id": "e158", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n346", "width": 1}, {"associationSlot": 0.388631952458032, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n345", "id": "e160", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n350", "width": 1}, {"associationSlot": 0.270829574098467, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n345", "id": "e161", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n354", "width": 1}, {"associationSlot": 0.270829574098503, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n345", "id": "e162", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n355", "width": 1}, {"associationSlot": 0.270829574098469, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n345", "id": "e163", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n356", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n345", "id": "e3106", "label": "SUBCLASS_OF", "to": "n23", "width": 1}, {"associationSlot": 0.255066956698362, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n346", "id": "e164", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n354", "width": 1}, {"associationSlot": 0.255066956698542, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n346", "id": "e165", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n355", "width": 1}, {"associationSlot": 0.25506695669841, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n346", "id": "e166", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n356", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n346", "id": "e3107", "label": "SUBCLASS_OF", "to": "n24", "width": 1}, {"associationSlot": 0.333408129812786, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n348", "id": "e198", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n424", "width": 1}, {"associationSlot": 0.333408129812779, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n348", "id": "e199", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n427", "width": 1}, {"associationSlot": 0.333408129812823, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n348", "id": "e200", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n497", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n348", "id": "e3109", "label": "SUBCLASS_OF", "to": "n26", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n350", "id": "e2758", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n350", "id": "e3111", "label": "SUBCLASS_OF", "to": "n28", "width": 1}, {"associationSlot": 0.255066956698497, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n354", "id": "e280", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n355", "width": 1}, {"associationSlot": 0.255066956698439, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n354", "id": "e281", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n356", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n354", "id": "e3115", "label": "SUBCLASS_OF", "to": "n32", "width": 1}, {"associationSlot": 0.255066956698497, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n355", "id": "e282", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n356", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n355", "id": "e2804", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n355", "id": "e3116", "label": "SUBCLASS_OF", "to": "n33", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n356", "id": "e2757", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n356", "id": "e3117", "label": "SUBCLASS_OF", "to": "n34", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n359", "id": "e2825", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n540", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n359", "id": "e2826", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n535", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n359", "id": "e3120", "label": "SUBCLASS_OF", "to": "n37", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n365", "id": "e3126", "label": "SUBCLASS_OF", "to": "n39", "width": 1}, {"associationSlot": 0.312632438904919, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n372", "id": "e491", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n384", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n372", "id": "e3133", "label": "SUBCLASS_OF", "to": "n46", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n376", "id": "e2847", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n376", "id": "e3137", "label": "SUBCLASS_OF", "to": "n28", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n384", "id": "e3145", "label": "SUBCLASS_OF", "to": "n54", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n386", "id": "e2816", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n540", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n386", "id": "e3147", "label": "SUBCLASS_OF", "to": "n56", "width": 1}, {"associationSlot": 0.269843778093374, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n407", "id": "e762", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n444", "width": 1}, {"associationSlot": 0.284094237060953, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n407", "id": "e763", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n491", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n407", "id": "e3168", "label": "SUBCLASS_OF", "to": "n75", "width": 1}, {"associationSlot": 0.260190287097582, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n408", "id": "e764", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n412", "width": 1}, {"associationSlot": 0.449115621858808, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n408", "id": "e765", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n472", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n408", "id": "e3169", "label": "SUBCLASS_OF", "to": "n76", "width": 1}, {"associationSlot": 0.260190287097567, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n409", "id": "e766", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n412", "width": 1}, {"associationSlot": 0.449115621858783, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n409", "id": "e767", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n472", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n409", "id": "e3170", "label": "SUBCLASS_OF", "to": "n77", "width": 1}, {"associationSlot": 0.269843778093355, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n411", "id": "e787", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n444", "width": 1}, {"associationSlot": 0.284094237060938, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n411", "id": "e788", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n491", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n411", "id": "e3172", "label": "SUBCLASS_OF", "to": "n79", "width": 1}, {"associationSlot": 0.26019028709757, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n412", "id": "e789", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n428", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n412", "id": "e3173", "label": "SUBCLASS_OF", "to": "n80", "width": 1}, {"associationSlot": 0.39262092506519, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e866", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n429", "width": 1}, {"associationSlot": 0.316597927717648, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e867", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n451", "width": 1}, {"associationSlot": 0.316597927717653, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e868", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n452", "width": 1}, {"associationSlot": 0.316597927717727, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e869", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.316597927717767, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e870", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.316597927717664, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e871", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n461", "width": 1}, {"associationSlot": 0.316597927717688, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e872", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717722, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e873", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.316597927717559, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e874", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n418", "id": "e3179", "label": "SUBCLASS_OF", "to": "n86", "width": 1}, {"associationSlot": 0.269843778093375, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n422", "id": "e915", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n444", "width": 1}, {"associationSlot": 0.284094237060954, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n422", "id": "e916", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n491", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n422", "id": "e2762", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n422", "id": "e3183", "label": "SUBCLASS_OF", "to": "n90", "width": 1}, {"associationSlot": 0.260828394674529, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n424", "id": "e922", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n426", "width": 1}, {"associationSlot": 0.269249203560562, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n424", "id": "e923", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n470", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n424", "id": "e3185", "label": "SUBCLASS_OF", "to": "n27", "width": 1}, {"associationSlot": 0.260828394674527, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n426", "id": "e926", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n427", "width": 1}, {"associationSlot": 0.260828394674602, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n426", "id": "e927", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n497", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n426", "id": "e2759", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n537", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n426", "id": "e3187", "label": "SUBCLASS_OF", "to": "n28", "width": 1}, {"associationSlot": 0.269249203560559, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n427", "id": "e928", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n470", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n427", "id": "e2796", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n427", "id": "e3188", "label": "SUBCLASS_OF", "to": "n28", "width": 1}, {"associationSlot": 0.449115621858786, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n428", "id": "e929", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n472", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n428", "id": "e2797", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n428", "id": "e2798", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n547", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n428", "id": "e3189", "label": "SUBCLASS_OF", "to": "n91", "width": 1}, {"associationSlot": 0.392620925065179, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e930", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n451", "width": 1}, {"associationSlot": 0.392620925065165, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e931", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n452", "width": 1}, {"associationSlot": 0.392620925065205, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e932", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.392620925065177, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e933", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.39262092506516, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e934", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n461", "width": 1}, {"associationSlot": 0.392620925065166, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e935", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.392620925065183, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e936", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.392620925065181, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e937", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n429", "id": "e3190", "label": "SUBCLASS_OF", "to": "n92", "width": 1}, {"associationSlot": 0.269843778093379, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n444", "id": "e1092", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n485", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n444", "id": "e2962", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n589", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n444", "id": "e3205", "label": "SUBCLASS_OF", "to": "n107", "width": 1}, {"associationSlot": 0.316597927717671, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1127", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n452", "width": 1}, {"associationSlot": 0.316597927717641, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1128", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.316597927717688, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1129", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.316597927717664, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1130", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n461", "width": 1}, {"associationSlot": 0.316597927717771, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1131", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717691, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1132", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.316597927717771, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1133", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n451", "id": "e2834", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n451", "id": "e3212", "label": "SUBCLASS_OF", "to": "n113", "width": 1}, {"associationSlot": 0.31659792771775, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1134", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.316597927717649, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1135", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.316597927717706, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1136", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n461", "width": 1}, {"associationSlot": 0.316597927717556, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1137", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717673, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1138", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.31659792771773, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1139", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n452", "id": "e3213", "label": "SUBCLASS_OF", "to": "n113", "width": 1}, {"associationSlot": 0.316597927717722, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1140", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.316597927717758, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1141", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n461", "width": 1}, {"associationSlot": 0.31659792771772, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1142", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.31659792771774, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1143", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.316597927717759, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1144", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2801", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2802", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n453", "id": "e3214", "label": "SUBCLASS_OF", "to": "n114", "width": 1}, {"associationSlot": 0.31659792771765, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n456", "id": "e1151", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n461", "width": 1}, {"associationSlot": 0.316597927717662, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n456", "id": "e1152", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717713, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n456", "id": "e1153", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.316597927717683, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n456", "id": "e1154", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n456", "id": "e2740", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n456", "id": "e3217", "label": "SUBCLASS_OF", "to": "n117", "width": 1}, {"associationSlot": 0.316597927717539, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n461", "id": "e1196", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717675, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n461", "id": "e1197", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.316597927717729, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n461", "id": "e1198", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n461", "id": "e3222", "label": "SUBCLASS_OF", "to": "n122", "width": 1}, {"associationSlot": 0.316597927717745, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n463", "id": "e1202", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"associationSlot": 0.316597927717748, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n463", "id": "e1203", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n463", "id": "e2807", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n543", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n463", "id": "e3224", "label": "SUBCLASS_OF", "to": "n123", "width": 1}, {"associationSlot": 0.269249203560633, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n470", "id": "e1223", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n497", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n470", "id": "e3231", "label": "SUBCLASS_OF", "to": "n130", "width": 1}, {"associationSlot": 0.316597927717736, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n471", "id": "e1224", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n475", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n471", "id": "e3232", "label": "SUBCLASS_OF", "to": "n131", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n472", "id": "e3233", "label": "SUBCLASS_OF", "to": "n132", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n475", "id": "e3237", "label": "SUBCLASS_OF", "to": "n135", "width": 1}, {"associationSlot": 0.28409423706096, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n485", "id": "e1323", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n491", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n485", "id": "e3248", "label": "SUBCLASS_OF", "to": "n145", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n491", "id": "e3254", "label": "SUBCLASS_OF", "to": "n150", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n497", "id": "e3260", "label": "SUBCLASS_OF", "to": "n156", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n524", "id": "e2742", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n524", "id": "e2743", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n537", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n524", "id": "e2744", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n534", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n524", "id": "e3287", "label": "SUBCLASS_OF", "to": "n49", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n533", "id": "e3296", "label": "SUBCLASS_OF", "to": "n183", "width": 1}]);

                  nodeColors = {};
                  allNodes = nodes.get({ returnType: "Object" });
                  for (nodeId in allNodes) {
                    nodeColors[nodeId] = allNodes[nodeId].color;
                  }
                  allEdges = edges.get({ returnType: "Object" });
                  // adding nodes and edges to the graph
                  data = {nodes: nodes, edges: edges};

                  var options = {
    "configure": {
        "enabled": false
    },
    "edges": {
        "color": {
            "inherit": true
        },
        "smooth": {
            "enabled": true,
            "type": "dynamic"
        }
    },
    "interaction": {
        "dragNodes": true,
        "hideEdgesOnDrag": false,
        "hideNodesOnDrag": false
    },
    "physics": {
        "enabled": true,
        "forceAtlas2Based": {
            "avoidOverlap": 0,
            "centralGravity": 0.005,
            "damping": 0.4,
            "gravitationalConstant": -15,
            "springConstant": 0.1,
            "springLength": 100
        },
        "solver": "forceAtlas2Based",
        "stabilization": {
            "enabled": true,
            "fit": true,
            "iterations": 1000,
            "onlyDynamicEdges": false,
            "updateInterval": 50
        }
    }
};

                  


                  

                  network = new vis.Network(container, data, options);

                  

                  

                  


                  

                  return network;

              }
              drawGraph();
        </script>
    </body>
</html>