<html>
    <head>
        <meta charset="utf-8">
        
            <script src="lib/bindings/utils.js"></script>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/dist/vis-network.min.css" integrity="sha512-WgxfT5LWjfszlPHXRmBWHkV2eceiWTOBvrKCNbdgDYTHrT2AeLCGbF4sZlZw3UMN3WtL0tGUoIAKsu8mllg/XA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
            <script src="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/vis-network.min.js" integrity="sha512-LnvoEWDFrqGHlHmDD2101OrLcbsfkrzoSpvtSQtxK3RMnRV0eOkhhBN2dXHKRrUU8p2DGRTk35n4O8nWSVe1mQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
            
        
<center>
<h1></h1>
</center>

<!-- <link rel="stylesheet" href="../node_modules/vis/dist/vis.min.css" type="text/css" />
<script type="text/javascript" src="../node_modules/vis/dist/vis.js"> </script>-->
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
          crossorigin="anonymous"
        />
        <script
          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js"
          integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf"
          crossorigin="anonymous"
        ></script>


        <center>
          <h1></h1>
        </center>
        <style type="text/css">

             #mynetwork {
                 width: 100%;
                 height: 600px;
                 background-color: white;
                 border: 1px solid lightgray;
                 position: relative;
                 float: left;
             }

             

             

             
        </style>
    </head>


    <body>
        <div class="card" style="width: 100%">
            
            
            <div id="mynetwork" class="card-body"></div>
        </div>

        
        

        <script type="text/javascript">

              // initialize global variables.
              var edges;
              var nodes;
              var allNodes;
              var allEdges;
              var nodeColors;
              var originalNodes;
              var network;
              var container;
              var options, data;
              var filter = {
                  item : '',
                  property : '',
                  value : []
              };

              

              

              // This method is responsible for drawing the graph, returns the drawn network
              function drawGraph() {
                  var container = document.getElementById('mynetwork');

                  

                  // parsing and collecting nodes and edges from the python
                  nodes = new vis.DataSet([{"betweenness_centrality": 1828.7860473990372, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2451436388508892, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 12.0, "font": {"size": 12}, "id": "n117", "label": "Syntrophobacter", "labels": ":Genus", "metabolic_cluster": 114, "name": "Syntrophobacter", "pagerank_centrality": 0.8052797228147336, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 3027.708642482462, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.24434142350695393, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 26.0, "description": "The pathways by which inorganic sulfate is processed and incorporated into sulfated compounds. These organisms utilize sulfate as a terminal electron acceptor producing hydrogen sulfide. A variety of sulfate reducers have been described, being able to degrade a range of organic compounds to acetate or completely to CO2. A variety of alternate electron acceptors have also been demonstrated, including Fe(III), nitrate and organics (fermentation). Therefore, a high abundance does not always reflect high levels of sulfate reduction. They are thought not to degrade complex polymers and rely on the activities of the hydrolysers and fermenters for a supply of carbon. The abundance and activity of these organisms is not favorable in wastewater treatment and anaerobic digestion as they compete with the methanogens for substrate, giving a lower methane yield, and hydrogen sulfide is toxic, odorous and associated with bio-corrosion problems in sewer systems and treatment facilities.", "font": {"size": 12}, "id": "n541", "iri": "http://purl.obolibrary.org/obo/GO_0000103", "label": "Sulfate assimilation", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Sulfate assimilation", "pagerank_centrality": 1.3140086272151603, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 8, "synonym": "sulphate assimilation|sulfate assimilation|phosphoadenylyl sulfate reduction by an oxidoreductase|acting on sulfur group of donors|NAD or NADP as acceptor|sulphate assimilation|phosphoadenylyl sulphate reduction by an oxidoreductase|acting on sulphur group of donors|NAD or NADP as acceptor", "xref": ""}, {"betweenness_centrality": 49411.49917747123, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.29797140006651146, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 108.0, "description": "The anaerobic enzymatic conversion of organic compounds, especially carbohydrates, coupling the oxidation and reduction of NAD/H and the generation of adenosine triphosphate (ATP). During fermentation, organic compounds act as both the electron acceptor and donor. Organic molecules, such as sugars and amino acids, are utilized as the energy and carbon source. In the absence of, or the inability to utilize, external electron acceptors, organic molecules act as the electron acceptor to balance the redox of the cell. Common by-products include volatile fatty acids, alcohols, hydrogen and CO2.", "font": {"size": 12}, "id": "n557", "iri": "http://purl.obolibrary.org/obo/GO_0006113", "label": "Fermentation", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Fermentation", "pagerank_centrality": 5.8841757772761865, "providedBy": "MiDAS|KG-Microbe", "shape": "dot", "size": 24, "synonym": "", "xref": "MetaCyc:FERMENTATION-PWY|Wikipedia:Fermentation_(biochemistry)"}, {"betweenness_centrality": 2903.9308787663413, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24034334763948498, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 8.0, "font": {"size": 12}, "id": "n131", "label": "Alteracholeplasma", "labels": ":Genus", "metabolic_cluster": 114, "name": "Alteracholeplasma", "pagerank_centrality": 0.7091306226665816, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Sapromyces\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:15653928\u0027, \u0027PMID:16350067\u0027, \u0027PMID:5487182\u0027]"}, {"betweenness_centrality": 19402.53922432753, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.275777162203755, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 100.0, "description": "The chemical reactions and pathways involving carbohydrates, any of a group of organic compounds based of the general formula Cx(H2O)y. The three main classes of organics fed into wastewater treatment systems are represented here: Lipids/fatty acids; carbohydrates/sugars; proteins/amino acids. In this MiDAS field guide there is no differentiation between those able to use complex organics and those only able to utilize simple compounds released during hydrolysis. Where known, this is covered in the genus description. Organisms only need to reportedly assimilate one compound for the class to be designated as positive for this field.", "font": {"size": 12}, "id": "n938", "iri": "http://purl.obolibrary.org/obo/GO_0005975 ", "label": "Carbohydrate metabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Carbohydrate metabolic process", "pagerank_centrality": 6.027438888431428, "providedBy": "MiDAS", "shape": "dot", "size": 14, "synonym": "carbohydrate metabolism", "xref": ""}, {"betweenness_centrality": 3779.404011436226, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22216712124969007, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n418", "label": "Thermomicrobium carboxidum", "labels": ":Species", "metabolic_cluster": 114, "name": "Thermomicrobium carboxidum", "pagerank_centrality": 1.5834109415167759, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:24814334\u0027, \u0027PMID:26374291\u0027]"}, {"betweenness_centrality": 2603.4254722089745, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23944414751469803, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n429", "label": "Serpentinicella alkaliphila", "labels": ":Species", "metabolic_cluster": 114, "name": "Serpentinicella alkaliphila", "pagerank_centrality": 1.4134205369814292, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:27499124\u0027]"}, {"betweenness_centrality": 1656.1768160858676, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22810590631364563, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 24.0, "font": {"size": 12}, "id": "n451", "label": "Desulfonatronum lacustre", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulfonatronum lacustre", "pagerank_centrality": 1.5939744523142547, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 1838.9094679827294, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23925233644859814, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 22.0, "font": {"size": 12}, "id": "n452", "label": "Desulfonatronum thiodismutans", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulfonatronum thiodismutans", "pagerank_centrality": 1.367254769956517, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:13130014\u0027]"}, {"betweenness_centrality": 10654.04824675038, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2732540408661177, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 36.0, "font": {"size": 12}, "id": "n453", "label": "Desulfocurvus vexinensis", "labels": ":Species", "metabolic_cluster": 114, "name": "Desulfocurvus vexinensis", "pagerank_centrality": 2.0536238724145552, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 11, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19643880\u0027]"}, {"betweenness_centrality": 1791.6124003540388, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.23272727272727273, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 26.0, "font": {"size": 12}, "id": "n456", "label": "Syntrophobacter pfennigii", "labels": ":Species", "metabolic_cluster": 114, "name": "Syntrophobacter pfennigii", "pagerank_centrality": 1.6488247681482495, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Syntrophobacterium pfennigii\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 3542.9578532116616, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.24262117519631735, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 26.0, "font": {"size": 12}, "id": "n463", "label": "Methylococcus capsulatus", "labels": ":Species", "metabolic_cluster": 114, "name": "Methylococcus capsulatus", "pagerank_centrality": 1.7242965392940846, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 9, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:5937247\u0027]"}, {"betweenness_centrality": 473.8638810958381, "border-color": "#a48f23ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.22416812609457093, "cluster": "11", "color": {"background": "#e3cd72ff", "border": "#a48f23ff"}, "degree_centrality": 20.0, "font": {"size": 12}, "id": "n471", "label": "Alteracholeplasma palmae", "labels": ":Species", "metabolic_cluster": 114, "name": "Alteracholeplasma palmae", "pagerank_centrality": 1.2708717087085106, "propertyValue": "[\u0027has_rank NCBITaxon:species\u0027]", "shape": "dot", "size": 8, "synonym": "[\u0027\"Acholeplasma palmae\" RELATED synonym []\u0027]", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:7981098\u0027]"}, {"betweenness_centrality": 2059.446734966266, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.1838703057664683, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n86", "label": "Thermomicrobium", "labels": ":Genus", "metabolic_cluster": 114, "name": "Thermomicrobium", "pagerank_centrality": 0.5829483402537748, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 1025.83976815532, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19627601314348303, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n92", "label": "Serpentinicella", "labels": ":Genus", "metabolic_cluster": 114, "name": "Serpentinicella", "pagerank_centrality": 0.520447600034578, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:27499124\u0027]"}, {"betweenness_centrality": 849.1807882902475, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.19640508548882069, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 6.0, "font": {"size": 12}, "id": "n113", "label": "Desulfonatronum", "labels": ":Genus", "metabolic_cluster": 114, "name": "Desulfonatronum", "pagerank_centrality": 0.6051422024374148, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027]"}, {"betweenness_centrality": 181.25747770518288, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.2330299089726918, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 6.0, "description": "The simplest carboxylic acid, containing a single carbon. Occurs naturally in various sources including the venom of bee and ant stings, and is a useful organic synthetic reagent. Principally used as a preservative and antibacterial agent in livestock feed. Induces severe metabolic acidosis and ocular injury in human subjects.", "font": {"size": 12}, "id": "n611", "iri": "http://purl.obolibrary.org/obo/CHEBI_30751", "label": "formic acid", "labels": ":ChemicalEntity", "metabolic_cluster": 462, "name": "formic acid", "pagerank_centrality": 0.43357319757759044, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "Acide formique|Ameisensaeure|FORMIC ACID|Formic acid|H-COOH|HCO2H|HCOOH|Methanoic acid|aminic acid|bilorin|formic acid|formylic acid|hydrogen carboxylic acid|methoic acid", "xref": "BPDB:1749|Beilstein:1209246|CAS:64-18-6|DrugBank:DB01942|Gmelin:1008|HMDB:HMDB0000142|KEGG:C00058|KNApSAcK:C00001182|LIPID_MAPS_instance:LMFA01010040|MetaCyc:FORMATE|PDBeChem:FMT|PMID:12591956|PMID:14637377|PMID:15811469|PMID:16120414|PMID:16185830|PMID:16222862|PMID:16230297|PMID:16445901|PMID:16465784|PMID:18034701|PMID:18397576|PMID:22080171|PMID:22280475|PMID:22304812|PMID:22385261|PMID:22447125|PMID:22483350|PMID:22499553|PMID:22540994|PMID:22606986|PMID:22622393|PMID:3946945|PMID:7361809|Patent:CN101481304|Reaxys:1209246|Wikipedia:Formic_acid"}, {"betweenness_centrality": 1100.246894938073, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.2398929049531459, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 12.0, "description": "A monocarboxylic acid anion resulting from the removal of a proton from the carboxy group of acetic acid.", "font": {"size": 12}, "id": "n610", "iri": "http://purl.obolibrary.org/obo/CHEBI_30089", "label": "acetate", "labels": ":ChemicalEntity", "metabolic_cluster": 78, "name": "acetate", "pagerank_centrality": 0.7450895488639689, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "ACETATE ION|Azetat|CH3-COO(-)|Ethanoat|MeCO2 anion|acetate|acetic acid, ion(1-)|ethanoate", "xref": "Beilstein:1901470|CAS:71-50-1|DrugBank:DB03166|Gmelin:1379|KEGG:C00033|MetaCyc:ACET|PDBeChem:ACT|PMID:17190852|PMID:22211106|PMID:22371380|Reaxys:1901470|UM-BBD_compID:c0050|Wikipedia:Acetate"}, {"betweenness_centrality": 338.8967944081467, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.24209673061334774, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 8.0, "description": "A hydroxy monocarboxylic acid anion that is the conjugate base of lactic acid, arising from deprotonation of the carboxy group.", "font": {"size": 12}, "id": "n592", "iri": "http://purl.obolibrary.org/obo/CHEBI_24996", "label": "lactate", "labels": ":ChemicalEntity", "metabolic_cluster": 445, "name": "lactate", "pagerank_centrality": 0.5256562643508425, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "2-hydroxypropanoate|2-hydroxypropanoic acid, ion(1-)|2-hydroxypropionate|MeCH(OH)CO2 anion|b-lactate|beta-lactate|lactate", "xref": "Beilstein:3587719|CAS:113-21-3|Gmelin:240074|KEGG:C01432|MetaCyc:Lactate"}, {"betweenness_centrality": 1032.3158040563278, "border-color": "#d65c5cff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "ChemicalEntity", "closeness_centrality": 0.25963488843813387, "color": {"background": "#e9aaaaff", "border": "#d65c5cff"}, "degree_centrality": 16.0, "description": "A 2-oxo monocarboxylic acid anion that is the conjugate base of pyruvic acid, arising from deprotonation of the carboxy group.", "font": {"size": 12}, "id": "n560", "iri": "http://purl.obolibrary.org/obo/CHEBI_15361", "label": "pyruvate", "labels": ":ChemicalEntity", "metabolic_cluster": 424, "name": "pyruvate", "pagerank_centrality": 0.8122519763571289, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "2-oxopropanoate|2-oxopropanoic acid, ion(1-)|pyruvate", "xref": "Beilstein:3587721|CAS:57-60-3|Gmelin:2502|KEGG:C00022|PMID:17190852|PMID:21603897|PMID:21823181|PMID:21854850|PMID:22006570|PMID:22016370|PMID:22215378|PMID:22311625|PMID:22451307|PMID:22458763|Reaxys:3587721|UM-BBD_compID:c0159"}, {"betweenness_centrality": 370.1309786982396, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2177929022848809, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n114", "label": "Desulfocurvus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Desulfocurvus", "pagerank_centrality": 0.43338394749007736, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:19643880\u0027]"}, {"betweenness_centrality": 367.3908283400976, "border-color": "#d761d6ff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "EnvironmentalFeature", "closeness_centrality": 0.22422422422422422, "color": {"background": "#ecb5ebff", "border": "#d761d6ff"}, "degree_centrality": 8.0, "description": "Water that has been adversely affected in quality by anthropogenic influence.", "font": {"size": 12}, "id": "n752", "iri": "http://purl.obolibrary.org/obo/ENVO_00002001", "label": "waste water", "labels": ":EnvironmentalFeature", "name": "waste water", "pagerank_centrality": 0.5272683795377303, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "", "xref": "EcoLexicon:waste_water|SWEETRealm:Wastewater|https://en.wikipedia.org/wiki/Waste_water"}, {"betweenness_centrality": 594.4206746289348, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.22915601023017904, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 12.0, "description": "The chemical reactions and pathways resulting in the breakdown of a hydrocarbon, a compound consisting of carbon and hydrogen only.", "font": {"size": 12}, "id": "n549", "iri": "http://purl.obolibrary.org/obo/GO_0120253", "label": "Hydrocarbon catabolic process", "labels": ":BiologicalProcess", "metabolic_cluster": 57, "name": "Hydrocarbon catabolic process", "pagerank_centrality": 0.6848254328164265, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "hydrocarbon breakdown|hydrocarbon catabolism|hydrocarbon degradation", "xref": ""}, {"betweenness_centrality": 0.0, "border-color": "#76d30aff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "BiologicalProcess", "closeness_centrality": 0.1952920662598082, "color": {"background": "#cdfb98ff", "border": "#76d30aff"}, "degree_centrality": 2.0, "description": "The cellular metabolic process in which organic chemical compounds are synthesized from carbon-containing molecules and nutrients using energy obtained from the oxidation of inorganic compounds or methane.", "font": {"size": 12}, "id": "n543", "iri": "http://purl.obolibrary.org/obo/GO_0062097", "label": "Chemosynthesis", "labels": ":BiologicalProcess", "metabolic_cluster": 114, "name": "Chemosynthesis", "pagerank_centrality": 0.2619679432118007, "providedBy": "KG-Microbe", "shape": "dot", "size": 8, "synonym": "", "xref": ""}, {"betweenness_centrality": 887.6231404538909, "border-color": "#009bedff", "borderWidth": 1.5, "borderWidthSelected": 2, "category": "OrganismTaxon", "closeness_centrality": 0.2004474272930649, "color": {"background": "#7ad1ffff", "border": "#009bedff"}, "degree_centrality": 4.0, "font": {"size": 12}, "id": "n123", "label": "Methylococcus", "labels": ":Genus", "metabolic_cluster": 114, "name": "Methylococcus", "pagerank_centrality": 0.5292936599300598, "propertyValue": "[\u0027has_rank NCBITaxon:genus\u0027]", "shape": "dot", "size": 8, "synonym": "Unknown", "xref": "[\u0027GC_ID:11\u0027, \u0027PMID:5937247\u0027]"}]);
                  edges = new vis.DataSet([{"color": "#11449cff", "font": {"size": 6}, "from": "n117", "hasattribute": "Positive", "id": "e2376", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n541", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n117", "hasattribute": "Positive", "id": "e2674", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n131", "hasattribute": "Positive", "id": "e2534", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n938", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n131", "hasattribute": "Positive", "id": "e2681", "label": "PARTICIPATES_IN", "providedBy": "MiDAS", "supportingStudyContext": "[\u0027In_vitro_Genomic\u0027]", "to": "n557", "width": 1}, {"associationSlot": 0.39262092506519, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e866", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n429", "width": 1}, {"associationSlot": 0.316597927717648, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e867", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n451", "width": 1}, {"associationSlot": 0.316597927717653, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e868", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n452", "width": 1}, {"associationSlot": 0.316597927717727, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e869", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.316597927717767, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e870", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.316597927717688, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e872", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717722, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n418", "id": "e873", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n418", "id": "e3179", "label": "SUBCLASS_OF", "to": "n86", "width": 1}, {"associationSlot": 0.392620925065179, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e930", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n451", "width": 1}, {"associationSlot": 0.392620925065165, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e931", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n452", "width": 1}, {"associationSlot": 0.392620925065205, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e932", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.392620925065177, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e933", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.392620925065166, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e935", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.392620925065183, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n429", "id": "e936", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n429", "id": "e3190", "label": "SUBCLASS_OF", "to": "n92", "width": 1}, {"associationSlot": 0.316597927717671, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1127", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n452", "width": 1}, {"associationSlot": 0.316597927717641, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1128", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.316597927717688, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1129", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.316597927717771, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1131", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717691, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n451", "id": "e1132", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n451", "id": "e2834", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n451", "id": "e3212", "label": "SUBCLASS_OF", "to": "n113", "width": 1}, {"associationSlot": 0.31659792771775, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1134", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n453", "width": 1}, {"associationSlot": 0.316597927717649, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1135", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.316597927717556, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1137", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717673, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n452", "id": "e1138", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n452", "id": "e3213", "label": "SUBCLASS_OF", "to": "n113", "width": 1}, {"associationSlot": 0.316597927717722, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1140", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n456", "width": 1}, {"associationSlot": 0.31659792771772, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1142", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.31659792771774, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n453", "id": "e1143", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2801", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2802", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n557", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2965", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n611", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2966", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n610", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2967", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n592", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n453", "id": "e2968", "label": "PRODUCES", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n560", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n453", "id": "e3214", "label": "SUBCLASS_OF", "to": "n114", "width": 1}, {"associationSlot": 0.316597927717662, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n456", "id": "e1152", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n463", "width": 1}, {"associationSlot": 0.316597927717713, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n456", "id": "e1153", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n456", "id": "e2172", "label": "LOCATED_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0001015", "to": "n752", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n456", "id": "e2740", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n541", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n456", "id": "e3217", "label": "SUBCLASS_OF", "to": "n117", "width": 1}, {"associationSlot": 0.316597927717745, "biome": "ENVO_00002001", "biomeName": "wastewater", "color": "#B53C44FF", "font": {"size": 6}, "from": "n463", "id": "e1202", "label": "ASSOCIATED_WITH", "supportingDataSource": "cclasso microbial association network", "supportingStudyMethodType": "metagenomics", "to": "n471", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n463", "id": "e2806", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n549", "width": 1}, {"color": "#11449cff", "font": {"size": 6}, "from": "n463", "id": "e2807", "label": "PARTICIPATES_IN", "primaryKnowledgeSource": "", "providedBy": "KG-Microbe", "relation": "RO:0002215", "to": "n543", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n463", "id": "e3224", "label": "SUBCLASS_OF", "to": "n123", "width": 1}, {"color": "#a5abb6ff", "font": {"size": 6}, "from": "n471", "id": "e3232", "label": "SUBCLASS_OF", "to": "n131", "width": 1}]);

                  nodeColors = {};
                  allNodes = nodes.get({ returnType: "Object" });
                  for (nodeId in allNodes) {
                    nodeColors[nodeId] = allNodes[nodeId].color;
                  }
                  allEdges = edges.get({ returnType: "Object" });
                  // adding nodes and edges to the graph
                  data = {nodes: nodes, edges: edges};

                  var options = {
    "configure": {
        "enabled": false
    },
    "edges": {
        "color": {
            "inherit": true
        },
        "smooth": {
            "enabled": true,
            "type": "dynamic"
        }
    },
    "interaction": {
        "dragNodes": true,
        "hideEdgesOnDrag": false,
        "hideNodesOnDrag": false
    },
    "physics": {
        "enabled": true,
        "forceAtlas2Based": {
            "avoidOverlap": 0,
            "centralGravity": 0.005,
            "damping": 0.4,
            "gravitationalConstant": -15,
            "springConstant": 0.1,
            "springLength": 100
        },
        "solver": "forceAtlas2Based",
        "stabilization": {
            "enabled": true,
            "fit": true,
            "iterations": 1000,
            "onlyDynamicEdges": false,
            "updateInterval": 50
        }
    }
};

                  


                  

                  network = new vis.Network(container, data, options);

                  

                  

                  


                  

                  return network;

              }
              drawGraph();
        </script>
    </body>
</html>